# Scientific discovery in the age of artificial intelligence

**来源**: [https://www.nature.com/articles/s41586-023-06221-2](https://www.nature.com/articles/s41586-023-06221-2)

---

# Scientific discovery in the age of artificial intelligence | Nature

原文链接: https://www.nature.com/articles/s41586-023-06221-2

* Review
* Published: 02 August 2023

# Scientific discovery in the age of artificial intelligence

* [Hanchen Wang](#auth-Hanchen-Wang-Aff1-Aff2-Aff37-Aff38) 
  [ORCID: orcid.org/0000-0002-1691-024X](https://orcid.org/0000-0002-1691-024X)[1](#Aff1),[2](#Aff2)[na1](#na1)[nAff37](#nAff37)[nAff38](#nAff38),
* [Tianfan Fu](#auth-Tianfan-Fu-Aff3)[3](#Aff3)[na1](#na1),
* [Yuanqi Du](#auth-Yuanqi-Du-Aff4)[4](#Aff4)[na1](#na1),
* [Wenhao Gao](#auth-Wenhao-Gao-Aff5)[5](#Aff5),
* [Kexin Huang](#auth-Kexin-Huang-Aff6)[6](#Aff6),
* [Ziming Liu](#auth-Ziming-Liu-Aff7)[7](#Aff7),
* [Payal Chandak](#auth-Payal-Chandak-Aff8) 
  [ORCID: orcid.org/0000-0003-1097-803X](https://orcid.org/0000-0003-1097-803X)[8](#Aff8),
* [Shengchao Liu](#auth-Shengchao-Liu-Aff9-Aff10) 
  [ORCID: orcid.org/0000-0003-2030-2367](https://orcid.org/0000-0003-2030-2367)[9](#Aff9),[10](#Aff10),
* [Peter Van Katwyk](#auth-Peter-Katwyk-Aff11-Aff12) 
  [ORCID: orcid.org/0000-0002-3512-0665](https://orcid.org/0000-0002-3512-0665)[11](#Aff11),[12](#Aff12),
* [Andreea Deac](#auth-Andreea-Deac-Aff9-Aff10)[9](#Aff9),[10](#Aff10),
* [Anima Anandkumar](#auth-Anima-Anandkumar-Aff2-Aff13)[2](#Aff2),[13](#Aff13),
* [Karianne Bergen](#auth-Karianne-Bergen-Aff11-Aff12)[11](#Aff11),[12](#Aff12),
* [Carla P. Gomes](#auth-Carla_P_-Gomes-Aff4) 
  [ORCID: orcid.org/0000-0002-4441-7225](https://orcid.org/0000-0002-4441-7225)[4](#Aff4),
* [Shirley Ho](#auth-Shirley-Ho-Aff14-Aff15-Aff16-Aff17)[14](#Aff14),[15](#Aff15),[16](#Aff16),[17](#Aff17),
* [Pushmeet Kohli](#auth-Pushmeet-Kohli-Aff18) 
  [ORCID: orcid.org/0000-0002-7466-7997](https://orcid.org/0000-0002-7466-7997)[18](#Aff18),
* [Joan Lasenby](#auth-Joan-Lasenby-Aff1)[1](#Aff1),
* [Jure Leskovec](#auth-Jure-Leskovec-Aff6) 
  [ORCID: orcid.org/0000-0002-5411-923X](https://orcid.org/0000-0002-5411-923X)[6](#Aff6),
* [Tie-Yan Liu](#auth-Tie_Yan-Liu-Aff19)[19](#Aff19),
* [Arjun Manrai](#auth-Arjun-Manrai-Aff20)[20](#Aff20),
* [Debora Marks](#auth-Debora-Marks-Aff21-Aff22) 
  [ORCID: orcid.org/0000-0001-9388-2281](https://orcid.org/0000-0001-9388-2281)[21](#Aff21),[22](#Aff22),
* [Bharath Ramsundar](#auth-Bharath-Ramsundar-Aff23)[23](#Aff23),
* [Le Song](#auth-Le-Song-Aff24-Aff25)[24](#Aff24),[25](#Aff25),
* [Jimeng Sun](#auth-Jimeng-Sun-Aff26)[26](#Aff26),
* [Jian Tang](#auth-Jian-Tang-Aff9-Aff27-Aff28)[9](#Aff9),[27](#Aff27),[28](#Aff28),
* [Petar Veličković](#auth-Petar-Veli_kovi_-Aff18-Aff29)[18](#Aff18),[29](#Aff29),
* [Max Welling](#auth-Max-Welling-Aff30-Aff31)[30](#Aff30),[31](#Aff31),
* [Linfeng Zhang](#auth-Linfeng-Zhang-Aff32-Aff33)[32](#Aff32),[33](#Aff33),
* [Connor W. Coley](#auth-Connor_W_-Coley-Aff5-Aff34) 
  [ORCID: orcid.org/0000-0002-8271-8723](https://orcid.org/0000-0002-8271-8723)[5](#Aff5),[34](#Aff34),
* [Yoshua Bengio](#auth-Yoshua-Bengio-Aff9-Aff10) 
  [ORCID: orcid.org/0000-0002-9322-3515](https://orcid.org/0000-0002-9322-3515)[9](#Aff9),[10](#Aff10) &
* …
* [Marinka Zitnik](#auth-Marinka-Zitnik-Aff20-Aff22-Aff35-Aff36) 
  [ORCID: orcid.org/0000-0001-8530-7228](https://orcid.org/0000-0001-8530-7228)[20](#Aff20),[22](#Aff22),[35](#Aff35),[36](#Aff36)

Show authors

[*Nature*](/)
**volume 620**, pages 47–60 (2023)[Cite this article](#citeas)

* 184k Accesses
* 1255 Citations
* 755 Altmetric
* [Metrics details](/articles/s41586-023-06221-2/metrics)

### Subjects

* [Computer science](/subjects/computer-science)
* [Machine learning](/subjects/machine-learning)
* [Scientific community](/subjects/scientific-community)
* [Statistics](/subjects/statistics)

A [Publisher Correction](https://doi.org/10.1038/s41586-023-06559-7) to this article was published on 30 August 2023

This article has been [updated](#change-history)

## Abstract

Artificial intelligence (AI) is being increasingly integrated into scientific discovery to augment and accelerate research, helping scientists to generate hypotheses, design experiments, collect and interpret large datasets, and gain insights that might not have been possible using traditional scientific methods alone. Here we examine breakthroughs over the past decade that include self-supervised learning, which allows models to be trained on vast amounts of unlabelled data, and geometric deep learning, which leverages knowledge about the structure of scientific data to enhance model accuracy and efficiency. Generative AI methods can create designs, such as small-molecule drugs and proteins, by analysing diverse data modalities, including images and sequences. We discuss how these methods can help scientists throughout the scientific process and the central issues that remain despite such advances. Both developers and users of AI tools need a better understanding of when such approaches need improvement, and challenges posed by poor data quality and stewardship remain. These issues cut across scientific disciplines and require developing foundational algorithmic approaches that can contribute to scientific understanding or acquire it autonomously, making them critical areas of focus for AI innovation.

[Access through your institution](https://wayf.springernature.com?redirect_uri=https%3A%2F%2Fwww.nature.com%2Farticles%2Fs41586-023-06221-2)

[Buy or subscribe](#access-options)

This is a preview of subscription content, [access via your institution](https://wayf.springernature.com?redirect_uri=https%3A%2F%2Fwww.nature.com%2Farticles%2Fs41586-023-06221-2)

## Access options

[Access through your institution](https://wayf.springernature.com?redirect_uri=https%3A%2F%2Fwww.nature.com%2Farticles%2Fs41586-023-06221-2)

Access Nature and 54 other Nature Portfolio journals

Get Nature+, our best-value online-access subscription

$32.99 / 30 days

cancel any time

[Learn more](https://shop-amers.nature.com/products/plus/?region=US)

Subscribe to this journal

Receive 51 print issues and online access

$199.00 per year

only $3.90 per issue

[Learn more](/nature/subscribe)

Buy this article

* Purchase on SpringerLink
* Instant access to the full article PDF.

USD 39.95

Prices may be subject to local taxes which are calculated during checkout

**Fig. 1: Science in the age of artificial intelligence.**

![](ai-leaders-insights/技术路径与方法论/深度研究流派/科学突破驱动/images/5533848d738ea1b1d7049a5b8d420b1e.png)

**Fig. 2: Learning meaningful representations of scientific data.**

![](ai-leaders-insights/技术路径与方法论/深度研究流派/科学突破驱动/images/3bc578b8eaee63d406db5f25823cc5c0.png)

**Fig. 3: AI-guided generation of scientific hypotheses.**

![](ai-leaders-insights/技术路径与方法论/深度研究流派/科学突破驱动/images/1282955b57a80a93f2d0dc121e00ae22.png)

**Fig. 4: Integration of AI with scientific experiments and simulation.**

![](ai-leaders-insights/技术路径与方法论/深度研究流派/科学突破驱动/images/11bdce997735954da92130966d8ea662.png)

### Similar content being viewed by others

![](ai-leaders-insights/技术路径与方法论/深度研究流派/科学突破驱动/images/75812bd3fbc9c147f5ad1ccb1f475e76.png)

### [Quantifying the use and potential benefits of artificial intelligence in scientific research](https://www.nature.com/articles/s41562-024-02020-5?fromPaywallRec=true)

Article
11 October 2024

![](ai-leaders-insights/技术路径与方法论/深度研究流派/科学突破驱动/images/66dc17c55dd38a445d1db230c2eb9381.png)

### [Accelerating science with human-aware artificial intelligence](https://www.nature.com/articles/s41562-023-01648-z?fromPaywallRec=true)

Article
13 July 2023

![](ai-leaders-insights/技术路径与方法论/深度研究流派/科学突破驱动/images/6e39c8b78a607d6695b6149a12023bb5.png)

### [Generative AI lacks the human creativity to achieve scientific discovery from scratch](https://www.nature.com/articles/s41598-025-93794-9?fromPaywallRec=true)

Article
Open access
20 March 2025

## Change history

* ### 30 August 2023

  A Correction to this paper has been published: <https://doi.org/10.1038/s41586-023-06559-7>

## References

1. LeCun, Y., Bengio, Y. & Hinton, G. Deep learning. *Nature* **521**, 436–444 (2015). **This survey summarizes key elements of deep learning and its development in speech recognition, computer vision and and natural language processing**.

   [Article](https://doi.org/10.1038%2Fnature14539) 
   [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2015Natur.521..436L) 
   [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2MXht1WlurzP) 
   [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26017442) 
   [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Deep%20learning&journal=Nature&doi=10.1038%2Fnature14539&volume=521&pages=436-444&publication_year=2015&author=LeCun%2CY&author=Bengio%2CY&author=Hinton%2CG)
2. de Regt, H. W. Understanding, values, and the aims of science. *Phil. Sci.* **87**, 921–932 (2020).

   [Article](https://doi.org/10.1086%2F710520) 
   [MathSciNet](http://www.ams.org/mathscinet-getitem?mr=4190891) 
   [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Understanding%2C%20values%2C%20and%20the%20aims%20of%20science&journal=Phil.%20Sci.&doi=10.1086%2F710520&volume=87&pages=921-932&publication_year=2020&author=Regt%2CHW)
3. Pickstone, J. V. *Ways of Knowing: A New History of Science, Technology, and Medicine* (Univ. Chicago Press, 2001).
4. Han, J. et al. Deep potential: a general representation of a many-body potential energy surface. *Commun. Comput. Phys.* **23**, 629–639 (2018). **This paper introduced a deep neural network architecture that learns the potential energy surface of many-body systems while respecting the underlying symmetries of the system by incorporating group theory.**
5. Akiyama, K. et al. First M87 Event Horizon Telescope results. IV. Imaging the central supermassive black hole. *Astrophys. J. Lett.* **875**, L4 (2019).

   [Article](https://doi.org/10.3847%2F2041-8213%2Fab0e85) 
   [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019ApJ...875L...4E) 
   [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlWltbjI) 
   [Google Scholar](http://scholar.google.com/scholar_lookup?&title=First%20M87%20Event%20Horizon%20Telescope%20results.%20IV.%20Imaging%20the%20central%20supermassive%20black%20hole&journal=Astrophys.%20J.%20Lett.&doi=10.3847%2F2041-8213%2Fab0e85&volume=875&publication_year=2019&author=Akiyama%2CK)
6. Wagner, A. Z. Constructions in combinatorics via neural networks. Preprint at <https://arxiv.org/abs/2104.14516> (2021).
7. Coley, C. W. et al. A robotic platform for flow synthesis of organic compounds informed by AI planning. *Science* **365**, eaax1566 (2019).

   [Article](https://doi.org/10.1126%2Fscience.aax1566) 
   [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsFCisb3O) 
   [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31395756) 
   [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20robotic%20platform%20for%20flow%20synthesis%20of%20organic%20compounds%20informed%20by%20AI%20planning&journal=Science&doi=10.1126%2Fscience.aax1566&volume=365&publication_year=2019&author=Coley%2CCW)
8. Bommasani, R. et al. On the opportunities and risks of foundation models. Preprint at <https://arxiv.org/abs/2108.07258> (2021).
9. Davies, A. et al. Advancing mathematics by guiding human intuition with AI. *Nature* **600**, 70–74 (2021). **This paper explores how AI can aid the development of pure mathematics by guiding mathematical intuition.**

   [Article](https://doi.org/10.1038%2Fs41586-021-04086-x) 
   [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021Natur.600...70D) 
   [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXis1eqsbbJ) 
   [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34853458) 
   [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8636249) 
   [MATH](http://www.emis.de/MATH-item?1505.57001) 
   [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Advancing%20mathematics%20by%20guiding%20human%20intuition%20with%20AI&journal=Nature&doi=10.1038%2Fs41586-021-04086-x&volume=600&pages=70-74&publication_year=2021&author=Davies%2CA)
10. Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. *Nature* **596**, 583–589 (2021).**This study was the first to demonstrate the ability to predict protein folding structures using AI methods with a high degree of accuracy, achieving results that are at or near the experimental resolution. This accomplishment is particularly noteworthy, as predicting protein folding has been a grand challenge in the field of molecular biology for over 50 years.**

    [Article](https://doi.org/10.1038%2Fs41586-021-03819-2) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021Natur.596..583J) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvVaktrrL) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34265844) 
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8371605) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold&journal=Nature&doi=10.1038%2Fs41586-021-03819-2&volume=596&pages=583-589&publication_year=2021&author=Jumper%2CJ)
11. Stokes, J. M. et al. A deep learning approach to antibiotic discovery. *Cell* **180**, 688–702 (2020).

    [Article](https://doi.org/10.1016%2Fj.cell.2020.01.021) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXnvFelur8%3D) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32084340) 
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8349178) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20deep%20learning%20approach%20to%20antibiotic%20discovery&journal=Cell&doi=10.1016%2Fj.cell.2020.01.021&volume=180&pages=688-702&publication_year=2020&author=Stokes%2CJM)
12. Bohacek, R. S., McMartin, C. & Guida, W. C. The art and practice of structure-based drug design: a molecular modeling perspective. *Med. Res. Rev.* **16**, 3–50 (1996).

    [Article](https://doi.org/10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6) 
    [CAS](/articles/cas-redirect/1:CAS:528:DyaK28XhtFyls78%3D) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8788213) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20art%20and%20practice%20of%20structure-based%20drug%20design%3A%20a%20molecular%20modeling%20perspective&journal=Med.%20Res.%20Rev.&doi=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6&volume=16&pages=3-50&publication_year=1996&author=Bohacek%2CRS&author=McMartin%2CC&author=Guida%2CWC)
13. Bileschi, M. L. et al. Using deep learning to annotate the protein universe. *Nat. Biotechnol.* **40**, 932–937 (2022).
14. Bellemare, M. G. et al. Autonomous navigation of stratospheric balloons using reinforcement learning. *Nature* **588**, 77–82 (2020). **This paper describes a reinforcement-learning algorithm for navigating a super-pressure balloon in the stratosphere, making real-time decisions in the changing environment.**

    [Article](https://doi.org/10.1038%2Fs41586-020-2939-8) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020Natur.588...77B) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXisVOks7rO) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33268863) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Autonomous%20navigation%20of%20stratospheric%20balloons%20using%20reinforcement%20learning&journal=Nature&doi=10.1038%2Fs41586-020-2939-8&volume=588&pages=77-82&publication_year=2020&author=Bellemare%2CMG)
15. Tshitoyan, V. et al. Unsupervised word embeddings capture latent knowledge from materials science literature. *Nature* **571**, 95–98 (2019).

    [Article](https://doi.org/10.1038%2Fs41586-019-1335-8) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019Natur.571...95T) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlamurrK) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31270483) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Unsupervised%20word%20embeddings%20capture%20latent%20knowledge%20from%20materials%20science%20literature&journal=Nature&doi=10.1038%2Fs41586-019-1335-8&volume=571&pages=95-98&publication_year=2019&author=Tshitoyan%2CV)
16. Zhang, L. et al. Deep potential molecular dynamics: a scalable model with the accuracy of quantum mechanics. *Phys. Rev. Lett.* **120**, 143001 (2018).

    [Article](https://doi.org/10.1103%2FPhysRevLett.120.143001) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2018PhRvL.120n3001Z) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXltFSksrg%3D) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29694129) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Deep%20potential%20molecular%20dynamics%3A%20a%20scalable%20model%20with%20the%20accuracy%20of%20quantum%20mechanics&journal=Phys.%20Rev.%20Lett.&doi=10.1103%2FPhysRevLett.120.143001&volume=120&publication_year=2018&author=Zhang%2CL)
17. Deiana, A. M. et al. Applications and techniques for fast machine learning in science. *Front. Big Data* **5**, 787421 (2022).
18. Karagiorgi, G. et al. Machine learning in the search for new fundamental physics. *Nat. Rev. Phys.* **4**, 399–412 (2022).
19. Zhou, C. & Paffenroth, R. C. Anomaly detection with robust deep autoencoders. In *ACM SIGKDD International Conference on Knowledge Discovery and Data Mining* 665–674 (2017).
20. Hinton, G. E. & Salakhutdinov, R. R. Reducing the dimensionality of data with neural networks. *Science* **313**, 504–507 (2006).

    [Article](https://doi.org/10.1126%2Fscience.1127647) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2006Sci...313..504H) 
    [MathSciNet](http://www.ams.org/mathscinet-getitem?mr=2242509) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD28Xnt1KntrY%3D) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16873662) 
    [MATH](http://www.emis.de/MATH-item?1226.68083) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reducing%20the%20dimensionality%20of%20data%20with%20neural%20networks&journal=Science&doi=10.1126%2Fscience.1127647&volume=313&pages=504-507&publication_year=2006&author=Hinton%2CGE&author=Salakhutdinov%2CRR)
21. Kasieczka, G. et al. The LHC Olympics 2020 a community challenge for anomaly detection in high energy physics. *Rep. Prog. Phys.* **84**, 124201 (2021).

    [Article](https://doi.org/10.1088%2F1361-6633%2Fac36b9) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021RPPh...84l4201K) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB38XhtFeitrw%3D) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20LHC%20Olympics%202020%20a%20community%20challenge%20for%20anomaly%20detection%20in%20high%20energy%20physics&journal=Rep.%20Prog.%20Phys.&doi=10.1088%2F1361-6633%2Fac36b9&volume=84&publication_year=2021&author=Kasieczka%2CG)
22. Govorkova, E. et al. Autoencoders on field-programmable gate arrays for real-time, unsupervised new physics detection at 40 MHz at the Large Hadron Collider. *Nat. Mach. Intell.* **4**, 154–161 (2022).

    [Article](https://doi.org/10.1038%2Fs42256-022-00441-3) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Autoencoders%20on%20field-programmable%20gate%20arrays%20for%20real-time%2C%20unsupervised%20new%20physics%20detection%20at%2040%20MHz%20at%20the%20Large%20Hadron%20Collider&journal=Nat.%20Mach.%20Intell.&doi=10.1038%2Fs42256-022-00441-3&volume=4&pages=154-161&publication_year=2022&author=Govorkova%2CE)
23. Chamberland, M. et al. Detecting microstructural deviations in individuals with deep diffusion MRI tractometry. *Nat. Comput. Sci.* **1**, 598–606 (2021).

    [Article](https://doi.org/10.1038%2Fs43588-021-00126-8) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35865756) 
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7613101) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Detecting%20microstructural%20deviations%20in%20individuals%20with%20deep%20diffusion%20MRI%20tractometry&journal=Nat.%20Comput.%20Sci.&doi=10.1038%2Fs43588-021-00126-8&volume=1&pages=598-606&publication_year=2021&author=Chamberland%2CM)
24. Rafique, M. et al. Delegated regressor, a robust approach for automated anomaly detection in the soil radon time series data. *Sci. Rep.* **10**, 3004 (2020).

    [Article](https://doi.org/10.1038%2Fs41598-020-59881-9) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020NatSR..10.3004R) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXlvVKlu7w%3D) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32080258) 
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7033208) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Delegated%20regressor%2C%20a%20robust%20approach%20for%20automated%20anomaly%20detection%20in%20the%20soil%20radon%20time%20series%20data&journal=Sci.%20Rep.&doi=10.1038%2Fs41598-020-59881-9&volume=10&publication_year=2020&author=Rafique%2CM)
25. Pastore, V. P. et al. Annotation-free learning of plankton for classification and anomaly detection. *Sci. Rep.* **10**, 12142 (2020).

    [Article](https://doi.org/10.1038%2Fs41598-020-68662-3) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020NatSR..1012142P) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVKls73F) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32699302) 
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7376023) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Annotation-free%20learning%20of%20plankton%20for%20classification%20and%20anomaly%20detection&journal=Sci.%20Rep.&doi=10.1038%2Fs41598-020-68662-3&volume=10&publication_year=2020&author=Pastore%2CVP)
26. Naul, B. et al. A recurrent neural network for classification of unevenly sampled variable stars. *Nat. Astron.* **2**, 151–155 (2018).

    [Article](https://doi.org/10.1038%2Fs41550-017-0321-z) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2018NatAs...2..151N) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20recurrent%20neural%20network%20for%20classification%20of%20unevenly%20sampled%20variable%20stars&journal=Nat.%20Astron.&doi=10.1038%2Fs41550-017-0321-z&volume=2&pages=151-155&publication_year=2018&author=Naul%2CB)
27. Lee, D.-H. et al. Pseudo-label: the simple and efficient semi-supervised learning method for deep neural networks. In *ICML Workshop on Challenges in Representation Learning* (2013).
28. Zhou, D. et al. Learning with local and global consistency. In *Advances in Neural Information Processing Systems* **16**, 321–328 (2003).
29. Radivojac, P. et al. A large-scale evaluation of computational protein function prediction. *Nat. Methods* **10**, 221–227 (2013).

    [Article](https://doi.org/10.1038%2Fnmeth.2340) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsVaqtbs%3D) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23353650) 
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3584181) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20large-scale%20evaluation%20of%20computational%20protein%20function%20prediction&journal=Nat.%20Methods&doi=10.1038%2Fnmeth.2340&volume=10&pages=221-227&publication_year=2013&author=Radivojac%2CP)
30. Barkas, N. et al. Joint analysis of heterogeneous single-cell RNA-seq dataset collections. *Nat. Methods* **16**, 695–698 (2019).

    [Article](https://doi.org/10.1038%2Fs41592-019-0466-z) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlOlsb7J) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31308548) 
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6684315) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Joint%20analysis%20of%20heterogeneous%20single-cell%20RNA-seq%20dataset%20collections&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0466-z&volume=16&pages=695-698&publication_year=2019&author=Barkas%2CN)
31. Tran, K. & Ulissi, Z. W. Active learning across intermetallics to guide discovery of electrocatalysts for CO2 reduction and H2 evolution. *Nat. Catal.* **1**, 696–703 (2018).

    [Article](https://doi.org/10.1038%2Fs41929-018-0142-1) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtFGisb%2FK) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Active%20learning%20across%20intermetallics%20to%20guide%20discovery%20of%20electrocatalysts%20for%20CO2%20reduction%20and%20H2%20evolution&journal=Nat.%20Catal.&doi=10.1038%2Fs41929-018-0142-1&volume=1&pages=696-703&publication_year=2018&author=Tran%2CK&author=Ulissi%2CZW)
32. Jablonka, K. M. et al. Bias free multiobjective active learning for materials design and discovery. *Nat. Commun.* **12**, 2312 (2021).

    [Article](https://doi.org/10.1038%2Fs41467-021-22437-0) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatCo..12.2312J) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXpsVKrsrs%3D) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33875649) 
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8055971) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Bias%20free%20multiobjective%20active%20learning%20for%20materials%20design%20and%20discovery&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-22437-0&volume=12&publication_year=2021&author=Jablonka%2CKM)
33. Roussel, R. et al. Turn-key constrained parameter space exploration for particle accelerators using Bayesian active learning. *Nat. Commun.* **12**, 5612 (2021).

    [Article](https://doi.org/10.1038%2Fs41467-021-25757-3) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatCo..12.5612R) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXitFSmtLbJ) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34556642) 
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8460822) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Turn-key%20constrained%20parameter%20space%20exploration%20for%20particle%20accelerators%20using%20Bayesian%20active%20learning&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-25757-3&volume=12&publication_year=2021&author=Roussel%2CR)
34. Ratner, A. J. et al. Data programming: creating large training sets, quickly. In *Advances in Neural Information Processing Systems* **29**, 3567–3575 (2016).
35. Ratner, A. et al. Snorkel: rapid training data creation with weak supervision. In *International Conference on Very Large Data Bases* **11**, 269–282 (2017). **This paper presents a weakly-supervised AI system designed to annotate massive amounts of data using labeling functions.**
36. Butter, A. et al. GANplifying event samples. *SciPost Phys.* **10**, 139 (2021).

    [Article](https://doi.org/10.21468%2FSciPostPhys.10.6.139) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021ScPP...10..139B) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GANplifying%20event%20samples&journal=SciPost%20Phys.&doi=10.21468%2FSciPostPhys.10.6.139&volume=10&publication_year=2021&author=Butter%2CA)
37. Brown, T. et al. Language models are few-shot learners. In *Advances in Neural Information Processing Systems* **33**, 1877–1901 (2020).
38. Ramesh, A. et al. Zero-shot text-to-image generation. In *International Conference on Machine Learning* **139**, 8821–8831 (2021).
39. Littman, M. L. Reinforcement learning improves behaviour from evaluative feedback. *Nature* **521**, 445–451 (2015).

    [Article](https://doi.org/10.1038%2Fnature14540) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2015Natur.521..445L) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2MXht1WlurvI) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26017443) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reinforcement%20learning%20improves%20behaviour%20from%20evaluative%20feedback&journal=Nature&doi=10.1038%2Fnature14540&volume=521&pages=445-451&publication_year=2015&author=Littman%2CML)
40. Cubuk, E. D. et al. Autoaugment: learning augmentation strategies from data. In *IEEE Conference on Computer Vision and Pattern Recognition* 113–123 (2019).
41. Reed, C. J. et al. Selfaugment: automatic augmentation policies for self-supervised learning. In *IEEE Conference on Computer Vision and Pattern Recognition* 2674–2683 (2021).
42. ATLAS Collaboration et al. Deep generative models for fast photon shower simulation in ATLAS. Preprint at <https://arxiv.org/abs/2210.06204> (2022).
43. Mahmood, F. et al. Deep adversarial training for multi-organ nuclei segmentation in histopathology images. *IEEE Trans. Med. Imaging* **39**, 3257–3267 (2019).

    [Article](https://doi.org/10.1109%2FTMI.2019.2927182) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Deep%20adversarial%20training%20for%20multi-organ%20nuclei%20segmentation%20in%20histopathology%20images&journal=IEEE%20Trans.%20Med.%20Imaging&doi=10.1109%2FTMI.2019.2927182&volume=39&pages=3257-3267&publication_year=2019&author=Mahmood%2CF)
44. Teixeira, B. et al. Generating synthetic X-ray images of a person from the surface geometry. In *IEEE Conference on Computer Vision and Pattern Recognition* 9059–9067 (2018).
45. Lee, D., Moon, W.-J. & Ye, J. C. Assessing the importance of magnetic resonance contrasts using collaborative generative adversarial networks. *Nat. Mach. Intell.* **2**, 34–42 (2020).

    [Article](https://doi.org/10.1038%2Fs42256-019-0137-x) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Assessing%20the%20importance%20of%20magnetic%20resonance%20contrasts%20using%20collaborative%20generative%20adversarial%20networks&journal=Nat.%20Mach.%20Intell.&doi=10.1038%2Fs42256-019-0137-x&volume=2&pages=34-42&publication_year=2020&author=Lee%2CD&author=Moon%2CW-J&author=Ye%2CJC)
46. Kench, S. & Cooper, S. J. Generating three-dimensional structures from a two-dimensional slice with generative adversarial network-based dimensionality expansion. *Nat. Mach. Intell.* **3**, 299–305 (2021).

    [Article](https://doi.org/10.1038%2Fs42256-021-00322-1) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Generating%20three-dimensional%20structures%20from%20a%20two-dimensional%20slice%20with%20generative%20adversarial%20network-based%20dimensionality%20expansion&journal=Nat.%20Mach.%20Intell.&doi=10.1038%2Fs42256-021-00322-1&volume=3&pages=299-305&publication_year=2021&author=Kench%2CS&author=Cooper%2CSJ)
47. Wan, C. & Jones, D. T. Protein function prediction is improved by creating synthetic feature samples with generative adversarial networks. *Nat. Mach. Intell.* **2**, 540–550 (2020).

    [Article](https://doi.org/10.1038%2Fs42256-020-0222-1) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Protein%20function%20prediction%20is%20improved%20by%20creating%20synthetic%20feature%20samples%20with%20generative%20adversarial%20networks&journal=Nat.%20Mach.%20Intell.&doi=10.1038%2Fs42256-020-0222-1&volume=2&pages=540-550&publication_year=2020&author=Wan%2CC&author=Jones%2CDT)
48. Repecka, D. et al. Expanding functional protein sequence spaces using generative adversarial networks. *Nat. Mach. Intell.* **3**, 324–333 (2021).

    [Article](https://doi.org/10.1038%2Fs42256-021-00310-5) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Expanding%20functional%20protein%20sequence%20spaces%20using%20generative%20adversarial%20networks&journal=Nat.%20Mach.%20Intell.&doi=10.1038%2Fs42256-021-00310-5&volume=3&pages=324-333&publication_year=2021&author=Repecka%2CD)
49. Marouf, M. et al. Realistic in silico generation and augmentation of single-cell RNA-seq data using generative adversarial networks. *Nat. Commun.* **11**, 166 (2020).

    [Article](https://doi.org/10.1038%2Fs41467-019-14018-z) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020NatCo..11..166M) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXpt1yltw%3D%3D) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31919373) 
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6952370) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Realistic%20in%20silico%20generation%20and%20augmentation%20of%20single-cell%20RNA-seq%20data%20using%20generative%20adversarial%20networks&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-019-14018-z&volume=11&publication_year=2020&author=Marouf%2CM)
50. Ghahramani, Z. Probabilistic machine learning and artificial intelligence. *Nature* **521**, 452–459 (2015).**This survey provides an introduction to probabilistic machine learning, which involves the representation and manipulation of uncertainty in models and predictions, playing a central role in scientific data analysis.**

    [Article](https://doi.org/10.1038%2Fnature14541) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2015Natur.521..452G) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2MXht1WlurzE) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26017444) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Probabilistic%20machine%20learning%20and%20artificial%20intelligence&journal=Nature&doi=10.1038%2Fnature14541&volume=521&pages=452-459&publication_year=2015&author=Ghahramani%2CZ)
51. Cogan, J. et al. Jet-images: computer vision inspired techniques for jet tagging. *J. High Energy Phys.* **2015**, 118 (2015).

    [Article](https://link.springer.com/doi/10.1007/JHEP02(2015)118) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Jet-images%3A%20computer%20vision%20inspired%20techniques%20for%20jet%20tagging&journal=J.%20High%20Energy%20Phys.&doi=10.1007%2FJHEP02%282015%29118&volume=2015&publication_year=2015&author=Cogan%2CJ)
52. Zhao, W. et al. Sparse deconvolution improves the resolution of live-cell super-resolution fluorescence microscopy. *Nat. Biotechnol.* **40**, 606–617 (2022).

    [Article](https://doi.org/10.1038%2Fs41587-021-01092-2) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXisV2lt7bM) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34782739) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Sparse%20deconvolution%20improves%20the%20resolution%20of%20live-cell%20super-resolution%20fluorescence%20microscopy&journal=Nat.%20Biotechnol.&doi=10.1038%2Fs41587-021-01092-2&volume=40&pages=606-617&publication_year=2022&author=Zhao%2CW)
53. Brbić, M. et al. MARS: discovering novel cell types across heterogeneous single-cell experiments. *Nat. Methods* **17**, 1200–1206 (2020).

    [Article](https://doi.org/10.1038%2Fs41592-020-00979-3) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33077966) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=MARS%3A%20discovering%20novel%20cell%20types%20across%20heterogeneous%20single-cell%20experiments&journal=Nat.%20Methods&doi=10.1038%2Fs41592-020-00979-3&volume=17&pages=1200-1206&publication_year=2020&author=Brbi%C4%87%2CM)
54. Qiao, C. et al. Evaluation and development of deep neural networks for image super-resolution in optical microscopy. *Nat. Methods* **18**, 194–202 (2021).

    [Article](https://doi.org/10.1038%2Fs41592-020-01048-5) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvFeitL0%3D) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33479522) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evaluation%20and%20development%20of%20deep%20neural%20networks%20for%20image%20super-resolution%20in%20optical%20microscopy&journal=Nat.%20Methods&doi=10.1038%2Fs41592-020-01048-5&volume=18&pages=194-202&publication_year=2021&author=Qiao%2CC)
55. Andreassen, A. et al. OmniFold: a method to simultaneously unfold all observables. *Phys. Rev. Lett.* **124**, 182001 (2020).

    [Article](https://doi.org/10.1103%2FPhysRevLett.124.182001) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020PhRvL.124r2001A) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVertbjF) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32441957) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=OmniFold%3A%20a%20method%20to%20simultaneously%20unfold%20all%20observables&journal=Phys.%20Rev.%20Lett.&doi=10.1103%2FPhysRevLett.124.182001&volume=124&publication_year=2020&author=Andreassen%2CA)
56. Bergenstråhle, L. et al. Super-resolved spatial transcriptomics by deep data fusion. *Nat. Biotechnol.* **40**, 476–479 (2021).
57. Vincent, P. et al. Extracting and composing robust features with denoising autoencoders. In *International Conference on Machine Learning* 1096–1103 (2008).
58. Kingma, D. P. & Welling, M. Auto-encoding variational Bayes. In *International Conference on Learning Representations* (2014).
59. Eraslan, G. et al. Single-cell RNA-seq denoising using a deep count autoencoder. *Nat. Commun.* **10**, 390 (2019).

    [Article](https://doi.org/10.1038%2Fs41467-018-07931-2) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019NatCo..10..390E) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXntFektb8%3D) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30674886) 
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6344535) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Single-cell%20RNA-seq%20denoising%20using%20a%20deep%20count%20autoencoder&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-018-07931-2&volume=10&publication_year=2019&author=Eraslan%2CG)
60. Goodfellow, I., Bengio, Y. & Courville, A. *Deep Learning* (MIT Press, 2016).
61. Olshausen, B. A. & Field, D. J. Emergence of simple-cell receptive field properties by learning a sparse code for natural images. *Nature* **381**, 607–609 (1996).

    [Article](https://doi.org/10.1038%2F381607a0) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=1996Natur.381..607O) 
    [CAS](/articles/cas-redirect/1:CAS:528:DyaK28XjsFylur8%3D) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8637596) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Emergence%20of%20simple-cell%20receptive%20field%20properties%20by%20learning%20a%20sparse%20code%20for%20natural%20images&journal=Nature&doi=10.1038%2F381607a0&volume=381&pages=607-609&publication_year=1996&author=Olshausen%2CBA&author=Field%2CDJ)
62. Bengio, Y. Deep learning of representations for unsupervised and transfer learning. In *ICML Workshop on Unsupervised and Transfer Learning* (2012).
63. Detlefsen, N. S., Hauberg, S. & Boomsma, W. Learning meaningful representations of protein sequences. *Nat. Commun.* **13**, 1914 (2022).

    [Article](https://doi.org/10.1038%2Fs41467-022-29443-w) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2022NatCo..13.1914D) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB38Xps1CktbY%3D) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35395843) 
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8993921) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Learning%20meaningful%20representations%20of%20protein%20sequences&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-022-29443-w&volume=13&publication_year=2022&author=Detlefsen%2CNS&author=Hauberg%2CS&author=Boomsma%2CW)
64. Becht, E. et al. Dimensionality reduction for visualizing single-cell data using UMAP. *Nat. Biotechnol.* **37**, 38–44 (2019).

    [Article](https://doi.org/10.1038%2Fnbt.4314) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXitl2isbrN) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dimensionality%20reduction%20for%20visualizing%20single-cell%20data%20using%20UMAP&journal=Nat.%20Biotechnol.&doi=10.1038%2Fnbt.4314&volume=37&pages=38-44&publication_year=2019&author=Becht%2CE)
65. Bronstein, M. M. et al. Geometric deep learning: going beyond euclidean data. *IEEE Signal Process Mag.* **34**, 18–42 (2017).

    [Article](https://doi.org/10.1109%2FMSP.2017.2693418) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2017ISPM...34...18B) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Geometric%20deep%20learning%3A%20going%20beyond%20euclidean%20data&journal=IEEE%20Signal%20Process%20Mag.&doi=10.1109%2FMSP.2017.2693418&volume=34&pages=18-42&publication_year=2017&author=Bronstein%2CMM)
66. Anderson, P. W. More is different: broken symmetry and the nature of the hierarchical structure of science. *Science* **177**, 393–396 (1972).

    [Article](https://doi.org/10.1126%2Fscience.177.4047.393) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=1972Sci...177..393A) 
    [CAS](/articles/cas-redirect/1:CAS:528:DyaE38XltVGlu7s%3D) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17796623) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=More%20is%20different%3A%20broken%20symmetry%20and%20the%20nature%20of%20the%20hierarchical%20structure%20of%20science&journal=Science&doi=10.1126%2Fscience.177.4047.393&volume=177&pages=393-396&publication_year=1972&author=Anderson%2CPW)
67. Qiao, Z. et al. Informing geometric deep learning with electronic interactions to accelerate quantum chemistry. *Proc. Natl Acad. Sci. USA* **119**, e2205221119 (2022).
68. Bogatskiy, A. et al. Symmetry group equivariant architectures for physics. Preprint at <https://arxiv.org/abs/2203.06153> (2022).
69. Bronstein, M. M. et al. Geometric deep learning: grids, groups, graphs, geodesics, and gauges. Preprint at <https://arxiv.org/abs/2104.13478> (2021).
70. Townshend, R. J. L. et al. Geometric deep learning of RNA structure. *Science* **373**, 1047–1051 (2021).

    [Article](https://doi.org/10.1126%2Fscience.abe5650) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021Sci...373.1047T) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvFWis7fI) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34446608) 
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9829186) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Geometric%20deep%20learning%20of%20RNA%20structure&journal=Science&doi=10.1126%2Fscience.abe5650&volume=373&pages=1047-1051&publication_year=2021&author=Townshend%2CRJL)
71. Wicky, B. I. M. et al. Hallucinating symmetric protein assemblies. *Science* **378**, 56–61 (2022).
72. Kipf, T. N. & Welling, M. Semi-supervised classification with graph convolutional networks. In *International Conference on Learning Representations* (2017).
73. Veličković, P. et al. Graph attention networks. In *International Conference on Learning Representations* (2018).
74. Hamilton, W. L., Ying, Z. & Leskovec, J. Inductive representation learning on large graphs. In *Advances in Neural Information Processing Systems* **30**, 1024–1034 (2017).
75. Gilmer, J. et al. Neural message passing for quantum chemistry. In *International Conference on Machine Learning* 1263–1272 (2017).
76. Li, M. M., Huang, K. & Zitnik, M. Graph representation learning in biomedicine and healthcare. *Nat. Biomed. Eng.* **6**, 1353–1369 (2022).
77. Satorras, V. G., Hoogeboom, E. & Welling, M. E(*n*) equivariant graph neural networks. In *International Conference on Machine Learning* 9323–9332 (2021). **This study incorporates principles of physics into the design of neural models, advancing the field of equivariant machine learning**.
78. Thomas, N. et al. Tensor field networks: rotation-and translation-equivariant neural networks for 3D point clouds. Preprint at <https://arxiv.org/abs/1802.08219> (2018).
79. Finzi, M. et al. Generalizing convolutional neural networks for equivariance to lie groups on arbitrary continuous data. In *International Conference on Machine Learning* 3165–3176 (2020).
80. Fuchs, F. et al. SE(3)-transformers: 3D roto-translation equivariant attention networks. In *Advances in Neural Information Processing Systems* **33**, 1970-1981 (2020).
81. Zaheer, M. et al. Deep sets. In *Advances in Neural Information Processing Systems* **30**, 3391–3401 (2017). **This paper is an early study that explores the use of deep neural architectures on set data, which consists of an unordered list of elements.**
82. Cohen, T. S. et al. Spherical CNNs. In *International Conference on Learning Representations* (2018).
83. Gordon, J. et al. Permutation equivariant models for compositional generalization in language. In *International Conference on Learning Representations* (2019).
84. Finzi, M., Welling, M. & Wilson, A. G. A practical method for constructing equivariant multilayer perceptrons for arbitrary matrix groups. In *International Conference on Machine Learning* 3318–3328 (2021).
85. Dijk, D. V. et al. Recovering gene interactions from single-cell data using data diffusion. *Cell* **174**, 716–729 (2018).

    [Article](https://doi.org/10.1016%2Fj.cell.2018.05.061) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29961576) 
    [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6771278) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Recovering%20gene%20interactions%20from%20single-cell%20data%20using%20data%20diffusion&journal=Cell&doi=10.1016%2Fj.cell.2018.05.061&volume=174&pages=716-729&publication_year=2018&author=Dijk%2CDV)
86. Gainza, P. et al. Deciphering interaction fingerprints from protein molecular surfaces using geometric deep learning. *Nat. Methods* **17**, 184–192 (2020).

    [Article](https://doi.org/10.1038%2Fs41592-019-0666-6) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXitlCht7%2FI) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31819266) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Deciphering%20interaction%20fingerprints%20from%20protein%20molecular%20surfaces%20using%20geometric%20deep%20learning&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0666-6&volume=17&pages=184-192&publication_year=2020&author=Gainza%2CP)
87. Hatfield, P. W. et al. The data-driven future of high-energy-density physics. *Nature* **593**, 351–361 (2021).

    [Article](https://doi.org/10.1038%2Fs41586-021-03382-w) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021Natur.593..351H) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtFanurzJ) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34012079) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20data-driven%20future%20of%20high-energy-density%20physics&journal=Nature&doi=10.1038%2Fs41586-021-03382-w&volume=593&pages=351-361&publication_year=2021&author=Hatfield%2CPW)
88. Bapst, V. et al. Unveiling the predictive power of static structure in glassy systems. *Nat. Phys.* **16**, 448–454 (2020).

    [Article](https://doi.org/10.1038%2Fs41567-020-0842-8) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXmsFSnsL4%3D) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Unveiling%20the%20predictive%20power%20of%20static%20structure%20in%20glassy%20systems&journal=Nat.%20Phys.&doi=10.1038%2Fs41567-020-0842-8&volume=16&pages=448-454&publication_year=2020&author=Bapst%2CV)
89. Zhang, R., Zhou, T. & Ma, J. Multiscale and integrative single-cell Hi-C analysis with Higashi. *Nat. Biotechnol.* **40**, 254–261 (2022).

    [Article](https://doi.org/10.1038%2Fs41587-021-01034-y) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1Wqu7%2FL) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34635838) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multiscale%20and%20integrative%20single-cell%20Hi-C%20analysis%20with%20Higashi&journal=Nat.%20Biotechnol.&doi=10.1038%2Fs41587-021-01034-y&volume=40&pages=254-261&publication_year=2022&author=Zhang%2CR&author=Zhou%2CT&author=Ma%2CJ)
90. Sammut, S.-J. et al. Multi-omic machine learning predictor of breast cancer therapy response. *Nature* **601**, 623–629 (2022).

    [Article](https://doi.org/10.1038%2Fs41586-021-04278-5) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2022Natur.601..623S) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB38Xht1Kqtbs%3D) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34875674) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Multi-omic%20machine%20learning%20predictor%20of%20breast%20cancer%20therapy%20response&journal=Nature&doi=10.1038%2Fs41586-021-04278-5&volume=601&pages=623-629&publication_year=2022&author=Sammut%2CS-J)
91. DeZoort, G. et al. Graph neural networks at the Large Hadron Collider. *Nat. Rev. Phys*. **5**, 281–303 (2023).
92. Liu, S. et al. Pre-training molecular graph representation with 3D geometry. In *International Conference on Learning Representations* (2022).
93. The LIGO Scientific Collaboration. et al. A gravitational-wave standard siren measurement of the Hubble constant. *Nature* **551**, 85–88 (2017).

    [Article](https://doi.org/10.1038%2Fnature24471) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20gravitational-wave%20standard%20siren%20measurement%20of%20the%20Hubble%20constant&journal=Nature&doi=10.1038%2Fnature24471&volume=551&pages=85-88&publication_year=2017)
94. Reichstein, M. et al. Deep learning and process understanding for data-driven Earth system science. *Nature* **566**, 195–204 (2019).

    [Article](https://doi.org/10.1038%2Fs41586-019-0912-1) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019Natur.566..195R) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXnsVCjtrw%3D) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30760912) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Deep%20learning%20and%20process%20understanding%20for%20data-driven%20Earth%20system%20science&journal=Nature&doi=10.1038%2Fs41586-019-0912-1&volume=566&pages=195-204&publication_year=2019&author=Reichstein%2CM)
95. Goenka, S. D. et al. Accelerated identification of disease-causing variants with ultra-rapid nanopore genome sequencing. *Nat. Biotechnol.* **40**, 1035–1041 (2022).
96. Bengio, Y. et al. Greedy layer-wise training of deep networks. In *Advances in Neural Information Processing Systems* **19**, 153–160 (2006).
97. Hinton, G. E., Osindero, S. & Teh, Y.-W. A fast learning algorithm for deep belief nets. *Neural Comput.* **18**, 1527–1554 (2006).

    [Article](https://doi.org/10.1162%2Fneco.2006.18.7.1527) 
    [MathSciNet](http://www.ams.org/mathscinet-getitem?mr=2224485) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16764513) 
    [MATH](http://www.emis.de/MATH-item?1106.68094) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20fast%20learning%20algorithm%20for%20deep%20belief%20nets&journal=Neural%20Comput.&doi=10.1162%2Fneco.2006.18.7.1527&volume=18&pages=1527-1554&publication_year=2006&author=Hinton%2CGE&author=Osindero%2CS&author=Teh%2CY-W)
98. Jordan, M. I. & Mitchell, T. M. Machine learning: trends, perspectives, and prospects. *Science* **349**, 255–260 (2015).

    [Article](https://doi.org/10.1126%2Fscience.aaa8415) 
    [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2015Sci...349..255J) 
    [MathSciNet](http://www.ams.org/mathscinet-getitem?mr=3382217) 
    [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtFKktL%2FM) 
    [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26185243) 
    [MATH](http://www.emis.de/MATH-item?1355.68227) 
    [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Machine%20learning%3A%20trends%2C%20perspectives%2C%20and%20prospects&journal=Science&doi=10.1126%2Fscience.aaa8415&volume=349&pages=255-260&publication_year=2015&author=Jordan%2CMI&author=Mitchell%2CTM)
99. Devlin, J. et al. BERT: pre-training of deep bidirectional transformers for language understanding. In *North American Chapter of the Association for Computational Linguistics* 4171–4186 (2019).
100. Rives, A. et al. Biological structure and function emerge from scaling unsupervised learning to 250 million protein sequences. *Proc. Natl Acad. Sci. USA* **118**, e2016239118 (2021).
101. Elnaggar, A. et al. ProtTrans: rowards cracking the language of lifes code through self-supervised deep learning and high performance computing. In *IEEE Transactions on Pattern Analysis and Machine Intelligence* (2021).
102. Hie, B. et al. Learning the language of viral evolution and escape. *Science* **371**, 284–288 (2021).**This paper modeled viral escape with machine learning algorithms originally developed for human natural language.**

     [Article](https://doi.org/10.1126%2Fscience.abd7331) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021Sci...371..284H) 
     [MathSciNet](http://www.ams.org/mathscinet-getitem?mr=4269294) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsVaitbs%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33446556) 
     [MATH](http://www.emis.de/MATH-item?1491.68156) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Learning%20the%20language%20of%20viral%20evolution%20and%20escape&journal=Science&doi=10.1126%2Fscience.abd7331&volume=371&pages=284-288&publication_year=2021&author=Hie%2CB)
103. Biswas, S. et al. Low-*N* protein engineering with data-efficient deep learning. *Nat. Methods* **18**, 389–396 (2021).

     [Article](https://doi.org/10.1038%2Fs41592-021-01100-y) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXosVCltrg%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33828272) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Low-N%20protein%20engineering%20with%20data-efficient%20deep%20learning&journal=Nat.%20Methods&doi=10.1038%2Fs41592-021-01100-y&volume=18&pages=389-396&publication_year=2021&author=Biswas%2CS)
104. Ferruz, N. & Höcker, B. Controllable protein design with language models. *Nat. Mach. Intell.* **4**, 521–532 (2022).
105. Hsu, C. et al. Learning inverse folding from millions of predicted structures. In *International Conference on Machine Learning* 8946–8970 (2022).
106. Baek, M. et al. Accurate prediction of protein structures and interactions using a three-track neural network. *Science* **373**, 871–876 (2021). **Inspired by AlphaFold2, this study reported RoseTTAFold, a novel three-track neural module capable of simultaneously processing protein’s sequence, distance and coordinates.**

     [Article](https://doi.org/10.1126%2Fscience.abj8754) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021Sci...373..871B) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvVCku7zM) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34282049) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7612213) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Accurate%20prediction%20of%20protein%20structures%20and%20interactions%20using%20a%20three-track%20neural%20network&journal=Science&doi=10.1126%2Fscience.abj8754&volume=373&pages=871-876&publication_year=2021&author=Baek%2CM)
107. Weininger, D. SMILES, a chemical language and information system. 1. Introduction to methodology and encoding rules. *J. Chem. Inf. Comput. Sci.* **28**, 31–36 (1988).

     [Article](https://doi.org/10.1021%2Fci00057a005) 
     [CAS](/articles/cas-redirect/1:CAS:528:DyaL1cXnsVeqsA%3D%3D) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=SMILES%2C%20a%20chemical%20language%20and%20information%20system.%201.%20Introduction%20to%20methodology%20and%20encoding%20rules&journal=J.%20Chem.%20Inf.%20Comput.%20Sci.&doi=10.1021%2Fci00057a005&volume=28&pages=31-36&publication_year=1988&author=Weininger%2CD)
108. Lin, T.-S. et al. BigSMILES: a structurally-based line notation for describing macromolecules. *ACS Cent. Sci.* **5**, 1523–1531 (2019).

     [Article](https://doi.org/10.1021%2Facscentsci.9b00476) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019pso..book.....L) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhslCntb7E) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31572779) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6764162) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=BigSMILES%3A%20a%20structurally-based%20line%20notation%20for%20describing%20macromolecules&journal=ACS%20Cent.%20Sci.&doi=10.1021%2Facscentsci.9b00476&volume=5&pages=1523-1531&publication_year=2019&author=Lin%2CT-S)
109. Krenn, M. et al. SELFIES and the future of molecular string representations. *Patterns* **3**, 100588 (2022).
110. Flam-Shepherd, D., Zhu, K. & Aspuru-Guzik, A. Language models can learn complex molecular distributions. *Nat. Commun.* **13**, 3293 (2022).

     [Article](https://doi.org/10.1038%2Fs41467-022-30839-x) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2022NatCo..13.3293F) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB38XhsF2mtr7I) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35672310) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9174447) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Language%20models%20can%20learn%20complex%20molecular%20distributions&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-022-30839-x&volume=13&publication_year=2022&author=Flam-Shepherd%2CD&author=Zhu%2CK&author=Aspuru-Guzik%2CA)
111. Skinnider, M. A. et al. Chemical language models enable navigation in sparsely populated chemical space. *Nat. Mach. Intell.* **3**, 759–770 (2021).

     [Article](https://doi.org/10.1038%2Fs42256-021-00368-1) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Chemical%20language%20models%20enable%20navigation%20in%20sparsely%20populated%20chemical%20space&journal=Nat.%20Mach.%20Intell.&doi=10.1038%2Fs42256-021-00368-1&volume=3&pages=759-770&publication_year=2021&author=Skinnider%2CMA)
112. Chithrananda, S., Grand, G. & Ramsundar, B. ChemBERTa: large-scale self-supervised pretraining for molecular property prediction. In *Machine Learning for Molecules Workshop at NeurIPS* (2020).
113. Schwaller, P. et al. Predicting retrosynthetic pathways using transformer-based models and a hyper-graph exploration strategy. *Chem. Sci.* **11**, 3316–3325 (2020).

     [Article](https://doi.org/10.1039%2FC9SC05704H) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXktFWgtbs%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34122839) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8152799) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Predicting%20retrosynthetic%20pathways%20using%20transformer-based%20models%20and%20a%20hyper-graph%20exploration%20strategy&journal=Chem.%20Sci.&doi=10.1039%2FC9SC05704H&volume=11&pages=3316-3325&publication_year=2020&author=Schwaller%2CP)
114. Tetko, I. V. et al. State-of-the-art augmented NLP transformer models for direct and single-step retrosynthesis. *Nat. Commun.* **11**, 5575 (2020).

     [Article](https://doi.org/10.1038%2Fs41467-020-19266-y) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020NatCo..11.5575T) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXit1OqtLzN) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33149154) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7643129) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=State-of-the-art%20augmented%20NLP%20transformer%20models%20for%20direct%20and%20single-step%20retrosynthesis&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-020-19266-y&volume=11&publication_year=2020&author=Tetko%2CIV)
115. Schwaller, P. et al. Mapping the space of chemical reactions using attention-based neural networks. *Nat. Mach. Intell.* **3**, 144–152 (2021).

     [Article](https://doi.org/10.1038%2Fs42256-020-00284-w) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mapping%20the%20space%20of%20chemical%20reactions%20using%20attention-based%20neural%20networks&journal=Nat.%20Mach.%20Intell.&doi=10.1038%2Fs42256-020-00284-w&volume=3&pages=144-152&publication_year=2021&author=Schwaller%2CP)
116. Kovács, D. P., McCorkindale, W. & Lee, A. A. Quantitative interpretation explains machine learning models for chemical reaction prediction and uncovers bias. *Nat. Commun.* **12**, 1695 (2021).

     [Article](https://doi.org/10.1038%2Fs41467-021-21895-w) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatCo..12.1695K) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33727552) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7966799) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Quantitative%20interpretation%20explains%20machine%20learning%20models%20for%20chemical%20reaction%20prediction%20and%20uncovers%20bias&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-021-21895-w&volume=12&publication_year=2021&author=Kov%C3%A1cs%2CDP&author=McCorkindale%2CW&author=Lee%2CAA)
117. Pesciullesi, G. et al. Transfer learning enables the molecular transformer to predict regio-and stereoselective reactions on carbohydrates. *Nat. Commun.* **11**, 4874 (2020).

     [Article](https://doi.org/10.1038%2Fs41467-020-18671-7) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020NatCo..11.4874P) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFOhtrbJ) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32978395) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7519051) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Transfer%20learning%20enables%20the%20molecular%20transformer%20to%20predict%20regio-and%20stereoselective%20reactions%20on%20carbohydrates&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-020-18671-7&volume=11&publication_year=2020&author=Pesciullesi%2CG)
118. Vaswani, A. et al. Attention is all you need. In *Advances in Neural Information Processing Systems* **30**, 5998–6008 (2017). **This paper introduced the transformer, a modern neural network architecture that can process sequential data in parallel, revolutionizing natural language processing and sequence modeling.**
119. Mousavi, S. M. et al. Earthquake transformer—an attentive deep-learning model for simultaneous earthquake detection and phase picking. *Nat. Commun.* **11**, 3952 (2020).

     [Article](https://doi.org/10.1038%2Fs41467-020-17591-w) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020NatCo..11.3952M) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsF2is7zJ) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32770023) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7415159) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Earthquake%20transformer%E2%80%94an%20attentive%20deep-learning%20model%20for%20simultaneous%20earthquake%20detection%20and%20phase%20picking&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-020-17591-w&volume=11&publication_year=2020&author=Mousavi%2CSM)
120. Avsec, Ž. et al. Effective gene expression prediction from sequence by integrating long-range interactions. *Nat. Methods* **18**, 1196–1203 (2021).

     [Article](https://doi.org/10.1038%2Fs41592-021-01252-x) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXitFOltLbP) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34608324) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8490152) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Effective%20gene%20expression%20prediction%20from%20sequence%20by%20integrating%20long-range%20interactions&journal=Nat.%20Methods&doi=10.1038%2Fs41592-021-01252-x&volume=18&pages=1196-1203&publication_year=2021&author=Avsec%2C%C5%BD)
121. Meier, J. et al. Language models enable zero-shot prediction of the effects of mutations on protein function. In *Advances in Neural Information Processing Systems* **34**, 29287–29303 (2021).
122. Kamienny, P.-A. et al. End-to-end symbolic regression with transformers. In *Advances in Neural Information Processing Systems* **35**, 10269–10281 (2022).
123. Jaegle, A. et al. Perceiver: general perception with iterative attention. In *International Conference on Machine Learning* 4651–4664 (2021).
124. Chen, L. et al. Decision transformer: reinforcement learning via sequence modeling. In *Advances in Neural Information Processing Systems* **34**, 15084–15097 (2021).
125. Dosovitskiy, A. et al. An image is worth 16x16 words: transformers for image recognition at scale. In *International Conference on Learning Representations* (2020).
126. Choromanski, K. et al. Rethinking attention with performers. In *International Conference on Learning Representations* (2021).
127. Li, Z. et al. Fourier neural operator for parametric partial differential equations. In *International Conference on Learning Representations* (2021).
128. Kovachki, N. et al. Neural operator: learning maps between function spaces. *J. Mach. Learn. Res.* **24**, 1–97 (2023).
129. Russell, J. L. Kepler’s laws of planetary motion: 1609–1666. *Br. J. Hist. Sci.* **2**, 1–24 (1964).

     [Article](https://doi.org/10.1017%2FS0007087400001813) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Kepler%E2%80%99s%20laws%20of%20planetary%20motion%3A%201609%E2%80%931666&journal=Br.%20J.%20Hist.%20Sci.&doi=10.1017%2FS0007087400001813&volume=2&pages=1-24&publication_year=1964&author=Russell%2CJL)
130. Huang, K. et al. Artificial intelligence foundation for therapeutic science. *Nat. Chem. Biol.* **18**, 1033–1036 (2022).
131. Guimerà, R. et al. A Bayesian machine scientist to aid in the solution of challenging scientific problems. *Sci. Adv.* **6**, eaav6971 (2020).

     [Article](https://doi.org/10.1126%2Fsciadv.aav6971) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020SciA....6.6971G) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32064326) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6994216) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20Bayesian%20machine%20scientist%20to%20aid%20in%20the%20solution%20of%20challenging%20scientific%20problems&journal=Sci.%20Adv.&doi=10.1126%2Fsciadv.aav6971&volume=6&publication_year=2020&author=Guimer%C3%A0%2CR)
132. Liu, G. et al. Deep learning-guided discovery of an antibiotic targeting Acinetobacter baumannii. *Nat. Chem. Biol.* <https://doi.org/10.1038/s41589-023-01349-8> (2023).
133. Gómez-Bombarelli, R. et al. Design of efficient molecular organic light-emitting diodes by a high-throughput virtual screening and experimental approach. *Nat. Mater.* **15**, 1120–1127 (2016). **This paper proposes using a black-box AI predictor to accelerate high-throughput screening of molecules in materials science.**

     [Article](https://doi.org/10.1038%2Fnmat4717) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2016NatMa..15.1120G) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27500805) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Design%20of%20efficient%20molecular%20organic%20light-emitting%20diodes%20by%20a%20high-throughput%20virtual%20screening%20and%20experimental%20approach&journal=Nat.%20Mater.&doi=10.1038%2Fnmat4717&volume=15&pages=1120-1127&publication_year=2016&author=G%C3%B3mez-Bombarelli%2CR)
134. Sadybekov, A. A. et al. Synthon-based ligand discovery in virtual libraries of over 11 billion compounds. *Nature* **601**, 452–459 (2022).

     [Article](https://doi.org/10.1038%2Fs41586-021-04220-9) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2022Natur.601..452S) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXislGntbrK) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34912117) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Synthon-based%20ligand%20discovery%20in%20virtual%20libraries%20of%20over%2011%20billion%20compounds&journal=Nature&doi=10.1038%2Fs41586-021-04220-9&volume=601&pages=452-459&publication_year=2022&author=Sadybekov%2CAA)
135. The NNPDF Collaboration Evidence for intrinsic charm quarks in the proton. *Nature* **606**, 483–487 (2022).

     [Article](https://doi.org/10.1038%2Fs41586-022-04998-2) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Evidence%20for%20intrinsic%20charm%20quarks%20in%20the%20proton&journal=Nature&doi=10.1038%2Fs41586-022-04998-2&volume=606&pages=483-487&publication_year=2022)
136. Graff, D. E., Shakhnovich, E. I. & Coley, C. W. Accelerating high-throughput virtual screening through molecular pool-based active learning. *Chem. Sci.* **12**, 7866–7881 (2021).

     [Article](https://doi.org/10.1039%2FD0SC06805E) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXpvVKjsrY%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34168840) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8188596) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Accelerating%20high-throughput%20virtual%20screening%20through%20molecular%20pool-based%20active%20learning&journal=Chem.%20Sci.&doi=10.1039%2FD0SC06805E&volume=12&pages=7866-7881&publication_year=2021&author=Graff%2CDE&author=Shakhnovich%2CEI&author=Coley%2CCW)
137. Janet, J. P. et al. Accurate multiobjective design in a space of millions of transition metal complexes with neural-network-driven efficient global optimization. *ACS Cent. Sci.* **6**, 513–524 (2020).

     [Article](https://doi.org/10.1021%2Facscentsci.0c00026) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXksFOhs7Y%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32342001) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7181321) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Accurate%20multiobjective%20design%20in%20a%20space%20of%20millions%20of%20transition%20metal%20complexes%20with%20neural-network-driven%20efficient%20global%20optimization&journal=ACS%20Cent.%20Sci.&doi=10.1021%2Facscentsci.0c00026&volume=6&pages=513-524&publication_year=2020&author=Janet%2CJP)
138. Bacon, F. *Novum Organon* Vol. 1620 (2000).
139. Schmidt, M. & Lipson, H. Distilling free-form natural laws from experimental data. *Science* **324**, 81–85 (2009).

     [Article](https://doi.org/10.1126%2Fscience.1165893) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2009Sci...324...81S) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1MXjvVajsbw%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19342586) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Distilling%20free-form%20natural%20laws%20from%20experimental%20data&journal=Science&doi=10.1126%2Fscience.1165893&volume=324&pages=81-85&publication_year=2009&author=Schmidt%2CM&author=Lipson%2CH)
140. Petersen, B. K. et al. Deep symbolic regression: recovering mathematical expressions from data via risk-seeking policy gradients. In *International Conference on Learning Representations* (2020).
141. Zhavoronkov, A. et al. Deep learning enables rapid identification of potent DDR1 kinase inhibitors. *Nat. Biotechnol.* **37**, 1038–1040 (2019). **This paper describes a reinforcement-learning algorithm for navigating molecular combinatorial spaces, and it validates generated molecules using wet-lab experiments.**

     [Article](https://doi.org/10.1038%2Fs41587-019-0224-x) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs12gurnM) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31477924) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Deep%20learning%20enables%20rapid%20identification%20of%20potent%20DDR1%20kinase%20inhibitors&journal=Nat.%20Biotechnol.&doi=10.1038%2Fs41587-019-0224-x&volume=37&pages=1038-1040&publication_year=2019&author=Zhavoronkov%2CA)
142. Zhou, Z. et al. Optimization of molecules via deep reinforcement learning. *Sci. Rep.* **9**, 10752 (2019).

     [Article](https://doi.org/10.1038%2Fs41598-019-47148-x) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019NatSR...910752Z) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31341196) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6656766) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Optimization%20of%20molecules%20via%20deep%20reinforcement%20learning&journal=Sci.%20Rep.&doi=10.1038%2Fs41598-019-47148-x&volume=9&publication_year=2019&author=Zhou%2CZ)
143. You, J. et al. Graph convolutional policy network for goal-directed molecular graph generation. In *Advances in Neural Information Processing Systems* **31**, 6412–6422 (2018).
144. Bengio, Y. et al. GFlowNet foundations. Preprint at <https://arxiv.org/abs/2111.09266> (2021). **This paper describes a generative flow network that generates objects by sampling them from a distribution optimized for drug design.**
145. Jain, M. et al. Biological sequence design with GFlowNets. In *International Conference on Machine Learning* 9786–9801 (2022).
146. Malkin, N. et al. Trajectory balance: improved credit assignment in GFlowNets. In *Advances in Neural Information Processing Systems* **35**, 5955–5967 (2022).
147. Borkowski, O. et al. Large scale active-learning-guided exploration for in vitro protein production optimization. *Nat. Commun.* **11**, 1872 (2020). **This study introduced a dynamic programming approach to determine the optimal locations and capacities of hydropower dams in the Amazon Basin, balancing between energy production and environmental impact**.

     [Article](https://doi.org/10.1038%2Fs41467-020-15798-5) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020NatCo..11.1872B) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXnsFSktLc%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32312991) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7170859) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Large%20scale%20active-learning-guided%20exploration%20for%20in%20vitro%20protein%20production%20optimization&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-020-15798-5&volume=11&publication_year=2020&author=Borkowski%2CO)
148. Flecker, A. S. et al. Reducing adverse impacts of Amazon hydropower expansion. *Science* **375**, 753–760 (2022).**This study introduced a dynamic programming approach to determine the optimal locations and capacities of hydropower dams in the Amazon basin, achieving a balance between the benefits of energy production and the potential environmental impacts.**

     [Article](https://doi.org/10.1126%2Fscience.abj4017) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2022Sci...375..753F) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB38XkvFyhurs%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35175810) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Reducing%20adverse%20impacts%20of%20Amazon%20hydropower%20expansion&journal=Science&doi=10.1126%2Fscience.abj4017&volume=375&pages=753-760&publication_year=2022&author=Flecker%2CAS)
149. Pion-Tonachini, L. et al. Learning from learning machines: a new generation of AI technology to meet the needs of science. Preprint at <https://arxiv.org/abs/2111.13786> (2021).
150. Kusner, M. J., Paige, B. & Hernández-Lobato, J. M. Grammar variational autoencoder. In *International Conference on Machine Learning* 1945–1954 (2017). **This paper describes a grammar variational autoencoder that generates novel symbolic laws and drug molecules.**
151. Brunton, S. L., Proctor, J. L. & Kutz, J. N. Discovering governing equations from data by sparse identification of nonlinear dynamical systems. *Proc. Natl Acad. Sci. USA* **113**, 3932–3937 (2016).

     [Article](https://doi.org/10.1073%2Fpnas.1517384113) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2016PNAS..113.3932B) 
     [MathSciNet](http://www.ams.org/mathscinet-getitem?mr=3494081) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC28XkvFGis7w%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27035946) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4839439) 
     [MATH](http://www.emis.de/MATH-item?1355.94013) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Discovering%20governing%20equations%20from%20data%20by%20sparse%20identification%20of%20nonlinear%20dynamical%20systems&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1517384113&volume=113&pages=3932-3937&publication_year=2016&author=Brunton%2CSL&author=Proctor%2CJL&author=Kutz%2CJN)
152. Liu, Z. & Tegmark, M. Machine learning hidden symmetries. *Phys. Rev. Lett.* **128**, 180201 (2022).

     [Article](https://doi.org/10.1103%2FPhysRevLett.128.180201) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2022PhRvL.128r0201L) 
     [MathSciNet](http://www.ams.org/mathscinet-getitem?mr=4428706) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB38Xhtlagt7zL) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35594106) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Machine%20learning%20hidden%20symmetries&journal=Phys.%20Rev.%20Lett.&doi=10.1103%2FPhysRevLett.128.180201&volume=128&publication_year=2022&author=Liu%2CZ&author=Tegmark%2CM)
153. Gabbard, H. et al. Bayesian parameter estimation using conditional variational autoencoders for gravitational-wave astronomy. *Nat. Phys.* **18**, 112–117 (2022).

     [Article](https://doi.org/10.1038%2Fs41567-021-01425-7) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXisl2htrfE) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Bayesian%20parameter%20estimation%20using%20conditional%20variational%20autoencoders%20for%20gravitational-wave%20astronomy&journal=Nat.%20Phys.&doi=10.1038%2Fs41567-021-01425-7&volume=18&pages=112-117&publication_year=2022&author=Gabbard%2CH)
154. Chen, D. et al. Automating crystal-structure phase mapping by combining deep learning with constraint reasoning. *Nat. Mach. Intell.* **3**, 812–822 (2021).

     [Article](https://doi.org/10.1038%2Fs42256-021-00384-1) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Automating%20crystal-structure%20phase%20mapping%20by%20combining%20deep%20learning%20with%20constraint%20reasoning&journal=Nat.%20Mach.%20Intell.&doi=10.1038%2Fs42256-021-00384-1&volume=3&pages=812-822&publication_year=2021&author=Chen%2CD)
155. Gómez-Bombarelli, R. et al. Automatic chemical design using a data-driven continuous representation of molecules. *ACS Cent. Sci.* **4**, 268–276 (2018).
156. Anishchenko, I. et al. De novo protein design by deep network hallucination. *Nature* **600**, 547–552 (2021).

     [Article](https://doi.org/10.1038%2Fs41586-021-04184-w) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021Natur.600..547A) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXis1eqsb7I) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34853475) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9293396) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=De%20novo%20protein%20design%20by%20deep%20network%20hallucination&journal=Nature&doi=10.1038%2Fs41586-021-04184-w&volume=600&pages=547-552&publication_year=2021&author=Anishchenko%2CI)
157. Fu, T. et al. Differentiable scaffolding tree for molecular optimization. In *International Conference on Learning Representations* (2021).
158. Sanchez-Lengeling, B. & Aspuru-Guzik, A. Inverse molecular design using machine learning: generative models for matter engineering. *Science* **361**, 360–365 (2018).

     [Article](https://doi.org/10.1126%2Fscience.aat2663) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2018Sci...361..360S) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlyitr3L) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30049875) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Inverse%20molecular%20design%20using%20machine%20learning%3A%20generative%20models%20for%20matter%20engineering&journal=Science&doi=10.1126%2Fscience.aat2663&volume=361&pages=360-365&publication_year=2018&author=Sanchez-Lengeling%2CB&author=Aspuru-Guzik%2CA)
159. Huang, K. et al. Therapeutics Data Commons: machine learning datasets and tasks for drug discovery and development. In *NeurIPS Datasets and Benchmarks* (2021). **This study describes an initiative with open AI models, datasets and education programmes to facilitate advances in therapeutic science across all stages of drug discovery and development.**
160. Dance, A. Lab hazard. *Nature* **458**, 664–665 (2009).

     [Article](https://doi.org/10.1038%2Fnj7238-664a) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1MXptVKnsbc%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19444985) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Lab%20hazard&journal=Nature&doi=10.1038%2Fnj7238-664a&volume=458&pages=664-665&publication_year=2009&author=Dance%2CA)
161. Segler, M. H. S., Preuss, M. & Waller, M. P. Planning chemical syntheses with deep neural networks and symbolic AI. *Nature* **555**, 604–610 (2018). **This paper describes an approach that combines deep neural networks with Monte Carlo tree search to plan chemical synthesis.**

     [Article](https://doi.org/10.1038%2Fnature25978) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2018Natur.555..604S) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXmsVGqt7c%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29595767) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Planning%20chemical%20syntheses%20with%20deep%20neural%20networks%20and%20symbolic%20AI&journal=Nature&doi=10.1038%2Fnature25978&volume=555&pages=604-610&publication_year=2018&author=Segler%2CMHS&author=Preuss%2CM&author=Waller%2CMP)
162. Gao, W., Raghavan, P. & Coley, C. W. Autonomous platforms for data-driven organic synthesis. *Nat. Commun.* **13**, 1075 (2022).

     [Article](https://doi.org/10.1038%2Fs41467-022-28736-4) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2022NatCo..13.1075G) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB38Xlt12ks78%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35228543) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8885738) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Autonomous%20platforms%20for%20data-driven%20organic%20synthesis&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-022-28736-4&volume=13&publication_year=2022&author=Gao%2CW&author=Raghavan%2CP&author=Coley%2CCW)
163. Kusne, A. G. et al. On-the-fly closed-loop materials discovery via Bayesian active learning. *Nat. Commun.* **11**, 5966 (2020).

     [Article](https://doi.org/10.1038%2Fs41467-020-19597-w) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020NatCo..11.5966K) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXisVOnsL3N) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33235197) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7686338) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=On-the-fly%20closed-loop%20materials%20discovery%20via%20Bayesian%20active%20learning&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-020-19597-w&volume=11&publication_year=2020&author=Kusne%2CAG)
164. Gormley,A. J. & Webb, M. A. Machine learning in combinatorial polymer chemistry. *Nat. Rev. Mater.* **6**, 642–644 (2021).

     [Article](https://doi.org/10.1038%2Fs41578-021-00282-3) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021NatRM...6..642G) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsVynurzK) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34394961) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Machine%20learning%20in%20combinatorial%20polymer%20chemistry&journal=Nat.%20Rev.%20Mater.&doi=10.1038%2Fs41578-021-00282-3&volume=6&pages=642-644&publication_year=2021&author=Gormley%2C%2CAJ&author=Webb%2CMA)
165. Ament, S. et al. Autonomous materials synthesis via hierarchical active learning of nonequilibrium phase diagrams. *Sci. Adv.* **7**, eabg4930 (2021).

     [Article](https://doi.org/10.1126%2Fsciadv.abg4930) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021SciA....7.4930A) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB38XisFSqtLs%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34919429) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8682983) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Autonomous%20materials%20synthesis%20via%20hierarchical%20active%20learning%20of%20nonequilibrium%20phase%20diagrams&journal=Sci.%20Adv.&doi=10.1126%2Fsciadv.abg4930&volume=7&publication_year=2021&author=Ament%2CS)
166. Degrave, J. et al. Magnetic control of tokamak plasmas through deep reinforcement learning. *Nature* **602**, 414–419 (2022).**This paper describes an approach for controlling tokamak plasmas, using a reinforcement-learning agent to command-control coils and satisfy physical and operational constraints.**

     [Article](https://doi.org/10.1038%2Fs41586-021-04301-9) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2022Natur.602..414D) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB38XjvFOkur8%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35173339) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8850200) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Magnetic%20control%20of%20tokamak%20plasmas%20through%20deep%20reinforcement%20learning&journal=Nature&doi=10.1038%2Fs41586-021-04301-9&volume=602&pages=414-419&publication_year=2022&author=Degrave%2CJ)
167. Melnikov, A. A. et al. Active learning machine learns to create new quantum experiments. *Proc. Natl Acad. Sci. USA* **115**, 1221–1226 (2018).

     [Article](https://doi.org/10.1073%2Fpnas.1714936115) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2018PNAS..115.1221M) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlKqtrY%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29348200) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5819408) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Active%20learning%20machine%20learns%20to%20create%20new%20quantum%20experiments&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1714936115&volume=115&pages=1221-1226&publication_year=2018&author=Melnikov%2CAA)
168. Smith, J. S., Isayev, O. & Roitberg, A. E. ANI-1: an extensible neural network potential with DFT accuracy at force field computational cost. *Chem. Sci.* **8**, 3192–3203 (2017).

     [Article](https://doi.org/10.1039%2FC6SC05720A) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2sXitlGnsrs%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28507695) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5414547) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=ANI-1%3A%20an%20extensible%20neural%20network%20potential%20with%20DFT%20accuracy%20at%20force%20field%20computational%20cost&journal=Chem.%20Sci.&doi=10.1039%2FC6SC05720A&volume=8&pages=3192-3203&publication_year=2017&author=Smith%2CJS&author=Isayev%2CO&author=Roitberg%2CAE)
169. Wang, D. et al. Efficient sampling of high-dimensional free energy landscapes using adaptive reinforced dynamics. *Nat. Comput. Sci.* **2**, 20–29 (2022).**This paper describes a neural network for reliable uncertainty estimations in molecular dynamics, enabling efficient sampling of high-dimensional free energy landscapes.**

     [Article](https://doi.org/10.1038%2Fs43588-021-00173-1) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3sXlslWntbs%3D) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Efficient%20sampling%20of%20high-dimensional%20free%20energy%20landscapes%20using%20adaptive%20reinforced%20dynamics&journal=Nat.%20Comput.%20Sci.&doi=10.1038%2Fs43588-021-00173-1&volume=2&pages=20-29&publication_year=2022&author=Wang%2CD)
170. Wang, W. & Gómez-Bombarelli, R. Coarse-graining auto-encoders for molecular dynamics. *npj Comput. Mater.* **5**, 125 (2019).

     [Article](https://doi.org/10.1038%2Fs41524-019-0261-5) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019npjCM...5..125W) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Coarse-graining%20auto-encoders%20for%20molecular%20dynamics&journal=npj%20Comput.%20Mater.&doi=10.1038%2Fs41524-019-0261-5&volume=5&publication_year=2019&author=Wang%2CW&author=G%C3%B3mez-Bombarelli%2CR)
171. Hermann, J., Schätzle, Z. & Noé, F. Deep-neural-network solution of the electronic Schrödinger equation. *Nat. Chem.* **12**, 891–897 (2020).**This paper describes a method to learn the wavefunction of quantum systems using deep neural networks in conjunction with variational quantum Monte Carlo.**

     [Article](https://doi.org/10.1038%2Fs41557-020-0544-y) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFCht77L) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32968231) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Deep-neural-network%20solution%20of%20the%20electronic%20Schr%C3%B6dinger%20equation&journal=Nat.%20Chem.&doi=10.1038%2Fs41557-020-0544-y&volume=12&pages=891-897&publication_year=2020&author=Hermann%2CJ&author=Sch%C3%A4tzle%2CZ&author=No%C3%A9%2CF)
172. Carleo, G. & Troyer, M. Solving the quantum many-body problem with artificial neural networks. *Science* **355**, 602–606 (2017).

     [Article](https://doi.org/10.1126%2Fscience.aag2302) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2017Sci...355..602C) 
     [MathSciNet](http://www.ams.org/mathscinet-getitem?mr=3642415) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2sXit1Okur0%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28183973) 
     [MATH](http://www.emis.de/MATH-item?1404.81313) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Solving%20the%20quantum%20many-body%20problem%20with%20artificial%20neural%20networks&journal=Science&doi=10.1126%2Fscience.aag2302&volume=355&pages=602-606&publication_year=2017&author=Carleo%2CG&author=Troyer%2CM)
173. Em Karniadakis, G. et al. Physics-informed machine learning. *Nat. Rev. Phys.* **3**, 422–440 (2021).

     [Article](https://doi.org/10.1038%2Fs42254-021-00314-5) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Physics-informed%20machine%20learning&journal=Nat.%20Rev.%20Phys.&doi=10.1038%2Fs42254-021-00314-5&volume=3&pages=422-440&publication_year=2021&author=Em%20Karniadakis%2CG)
174. Li, Z. et al. Physics-informed neural operator for learning partial differential equations. Preprint at <https://arxiv.org/abs/2111.03794> (2021).
175. Kochkov, D. et al. Machine learning–accelerated computational fluid dynamics. *Proc. Natl Acad. Sci. USA* **118**, e2101784118 (2021). **This paper describes an approach to accelerating computational fluid dynamics by training a neural network to interpolate from coarse to fine grids and generalize to varying forcing functions and Reynolds numbers.**
176. Ji, W. et al. Stiff-PINN: physics-informed neural network for stiff chemical kinetics. *J. Phys. Chem. A* **125**, 8098–8106 (2021).

     [Article](https://doi.org/10.1021%2Facs.jpca.1c05102) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvFSnu7bM) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34463510) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Stiff-PINN%3A%20physics-informed%20neural%20network%20for%20stiff%20chemical%20kinetics&journal=J.%20Phys.%20Chem.%20A&doi=10.1021%2Facs.jpca.1c05102&volume=125&pages=8098-8106&publication_year=2021&author=Ji%2CW)
177. Smith, J. D., Azizzadenesheli, K. & Ross, Z. E. EikoNet: solving the Eikonal equation with deep neural networks. *IEEE Trans. Geosci. Remote Sens.* **59**, 10685–10696 (2020).

     [Article](https://doi.org/10.1109%2FTGRS.2020.3039165) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021ITGRS..5910685S) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=EikoNet%3A%20solving%20the%20Eikonal%20equation%20with%20deep%20neural%20networks&journal=IEEE%20Trans.%20Geosci.%20Remote%20Sens.&doi=10.1109%2FTGRS.2020.3039165&volume=59&pages=10685-10696&publication_year=2020&author=Smith%2CJD&author=Azizzadenesheli%2CK&author=Ross%2CZE)
178. Waheed, U. B. et al. PINNeik: Eikonal solution using physics-informed neural networks. *Comput. Geosci.* **155**, 104833 (2021).

     [Article](https://doi.org/10.1016%2Fj.cageo.2021.104833) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=PINNeik%3A%20Eikonal%20solution%20using%20physics-informed%20neural%20networks&journal=Comput.%20Geosci.&doi=10.1016%2Fj.cageo.2021.104833&volume=155&publication_year=2021&author=Waheed%2CUB)
179. Chen, R. T. Q. et al. Neural ordinary differential equations. In *Advances in Neural Information Processing Systems* **31**, 6572–6583 (2018). **This paper established a connection between neural networks and differential equations by introducing the adjoint method to learn continuous-time dynamical systems from data, replacing backpropagation.**
180. Raissi, M., Perdikaris, P. & Karniadakis, G. E. Physics-informed neural networks: a deep learning framework for solving forward and inverse problems involving nonlinear partial differential equations. *J. Comput. Phys.* **378**, 686–707 (2019). **This paper describes a deep-learning approach for solving forwards and inverse problems in nonlinear partial differential equations and can find solutions to differential equations from data.**

     [Article](https://doi.org/10.1016%2Fj.jcp.2018.10.045) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019JCoPh.378..686R) 
     [MathSciNet](http://www.ams.org/mathscinet-getitem?mr=3881695) 
     [MATH](http://www.emis.de/MATH-item?1415.68175) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Physics-informed%20neural%20networks%3A%20a%20deep%20learning%20framework%20for%20solving%20forward%20and%20inverse%20problems%20involving%20nonlinear%20partial%20differential%20equations&journal=J.%20Comput.%20Phys.&doi=10.1016%2Fj.jcp.2018.10.045&volume=378&pages=686-707&publication_year=2019&author=Raissi%2CM&author=Perdikaris%2CP&author=Karniadakis%2CGE)
181. Lu, L. et al. Learning nonlinear operators via DeepONet based on the universal approximation theorem of operators. *Nat. Mach. Intell.* **3**, 218–229 (2021).

     [Article](https://doi.org/10.1038%2Fs42256-021-00302-5) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021mwaw.book.....L) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Learning%20nonlinear%20operators%20via%20DeepONet%20based%20on%20the%20universal%20approximation%20theorem%20of%20operators&journal=Nat.%20Mach.%20Intell.&doi=10.1038%2Fs42256-021-00302-5&volume=3&pages=218-229&publication_year=2021&author=Lu%2CL)
182. Brandstetter, J., Worrall, D. & Welling, M. Message passing neural PDE solvers. In *International Conference on Learning Representations* (2022).
183. Noé, F. et al. Boltzmann generators: sampling equilibrium states of many-body systems with deep learning. *Science* **365**, eaaw1147 (2019). **This paper presents an efficient sampling algorithm using normalizing flows to simulate equilibrium states in many-body systems.**
184. Rezende, D. & Mohamed, S. Variational inference with normalizing flows. In *International Conference on Machine Learning* **37**, 1530–1538, (2015).
185. Dinh, L., Sohl-Dickstein, J. & Bengio, S. Density estimation using real NVP. In *International Conference on Learning Representations* (2017).
186. Nicoli, K. A. et al. Estimation of thermodynamic observables in lattice field theories with deep generative models. *Phys. Rev. Lett.* **126**, 032001 (2021).

     [Article](https://doi.org/10.1103%2FPhysRevLett.126.032001) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021PhRvL.126c2001N) 
     [MathSciNet](http://www.ams.org/mathscinet-getitem?mr=4207822) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXjtVOgsLk%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33543982) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Estimation%20of%20thermodynamic%20observables%20in%20lattice%20field%20theories%20with%20deep%20generative%20models&journal=Phys.%20Rev.%20Lett.&doi=10.1103%2FPhysRevLett.126.032001&volume=126&publication_year=2021&author=Nicoli%2CKA)
187. Kanwar, G. et al. Equivariant flow-based sampling for lattice gauge theory. *Phys. Rev. Lett.* **125**, 121601 (2020).

     [Article](https://doi.org/10.1103%2FPhysRevLett.125.121601) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020PhRvL.125l1601K) 
     [MathSciNet](http://www.ams.org/mathscinet-getitem?mr=4155139) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFKqsb7F) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33016765) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Equivariant%20flow-based%20sampling%20for%20lattice%20gauge%20theory&journal=Phys.%20Rev.%20Lett.&doi=10.1103%2FPhysRevLett.125.121601&volume=125&publication_year=2020&author=Kanwar%2CG)
188. Gabrié, M., Rotskoff, G. M. & Vanden-Eijnden, E. Adaptive Monte Carlo augmented with normalizing flows. *Proc. Natl Acad. Sci. USA* **119**, e2109420119 (2022).

     [Article](https://doi.org/10.1073%2Fpnas.2109420119) 
     [MathSciNet](http://www.ams.org/mathscinet-getitem?mr=4417577) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35235453) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8915891) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Adaptive%20Monte%20Carlo%20augmented%20with%20normalizing%20flows&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.2109420119&volume=119&publication_year=2022&author=Gabri%C3%A9%2CM&author=Rotskoff%2CGM&author=Vanden-Eijnden%2CE)
189. Jasra, A., Holmes, C. C. & Stephens, D. A. Markov chain Monte Carlo methods and the label switching problem in Bayesian mixture modeling. *Stat. Sci.* **20**, 50–67 (2005).
190. Bengio, Y. et al. Better mixing via deep representations. In *International Conference on Machine Learning* 552–560 (2013).
191. Pompe, E., Holmes, C. & Łatuszyński, K. A framework for adaptive MCMC targeting multimodal distributions. *Ann. Stat.* **48**, 2930–2952 (2020).

     [Article](https://doi.org/10.1214%2F19-AOS1916) 
     [MathSciNet](http://www.ams.org/mathscinet-getitem?mr=4152629) 
     [MATH](http://www.emis.de/MATH-item?1486.62009) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20framework%20for%20adaptive%20MCMC%20targeting%20multimodal%20distributions&journal=Ann.%20Stat.&doi=10.1214%2F19-AOS1916&volume=48&pages=2930-2952&publication_year=2020&author=Pompe%2CE&author=Holmes%2CC&author=%C5%81atuszy%C5%84ski%2CK)
192. Townshend, R. J. L. et al. ATOM3D: tasks on molecules in three dimensions. In *NeurIPS Datasets and Benchmarks* (2021).
193. Kearnes, S. M. et al. The open reaction database. *J. Am. Chem. Soc.* **143**, 18820–18826 (2021).

     [Article](https://doi.org/10.1021%2Fjacs.1c09820) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXitlOhurzL) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34727496) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20open%20reaction%20database&journal=J.%20Am.%20Chem.%20Soc.&doi=10.1021%2Fjacs.1c09820&volume=143&pages=18820-18826&publication_year=2021&author=Kearnes%2CSM)
194. Chanussot, L. et al. Open Catalyst 2020 (OC20) dataset and community challenges. *ACS Catal.* **11**, 6059–6072 (2021).

     [Article](https://doi.org/10.1021%2Facscatal.0c04525) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtVSksbnJ) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Open%20Catalyst%202020%20%28OC20%29%20dataset%20and%20community%20challenges&journal=ACS%20Catal.&doi=10.1021%2Facscatal.0c04525&volume=11&pages=6059-6072&publication_year=2021&author=Chanussot%2CL)
195. Brown, N. et al. GuacaMol: benchmarking models for de novo molecular design. *J. Chem. Inf. Model.* **59**, 1096–1108 (2019).

     [Article](https://doi.org/10.1021%2Facs.jcim.8b00839) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019ckfh.book.....B) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXltVWrsbY%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30887799) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=GuacaMol%3A%20benchmarking%20models%20for%20de%20novo%20molecular%20design&journal=J.%20Chem.%20Inf.%20Model.&doi=10.1021%2Facs.jcim.8b00839&volume=59&pages=1096-1108&publication_year=2019&author=Brown%2CN)
196. Notin, P. et al. Tranception: protein fitness prediction with autoregressive transformers and inference-time retrieval. In *International Conference on Machine Learning* 16990–17017 (2022).
197. Mitchell, M. et al. Model cards for model reporting. In *Conference on Fairness, Accountability, and Transparency*220–229 (2019).
198. Gebru, T. et al. Datasheets for datasets. *Commun. ACM* **64**, 86–92 (2021).

     [Article](https://doi.org/10.1145%2F3458723) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Datasheets%20for%20datasets&journal=Commun.%20ACM&doi=10.1145%2F3458723&volume=64&pages=86-92&publication_year=2021&author=Gebru%2CT)
199. Bai, X. et al. Advancing COVID-19 diagnosis with privacy-preserving collaboration in artificial intelligence. *Nat. Mach. Intell.* **3**, 1081–1089 (2021).

     [Article](https://doi.org/10.1038%2Fs42256-021-00421-z) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Advancing%20COVID-19%20diagnosis%20with%20privacy-preserving%20collaboration%20in%20artificial%20intelligence&journal=Nat.%20Mach.%20Intell.&doi=10.1038%2Fs42256-021-00421-z&volume=3&pages=1081-1089&publication_year=2021&author=Bai%2CX)
200. Warnat-Herresthal, S. et al. Swarm learning for decentralized and confidential clinical machine learning. *Nature* **594**, 265–270 (2021).

     [Article](https://doi.org/10.1038%2Fs41586-021-03583-3) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021Natur.594..265W) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtFOltbjE) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34040261) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8189907) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Swarm%20learning%20for%20decentralized%20and%20confidential%20clinical%20machine%20learning&journal=Nature&doi=10.1038%2Fs41586-021-03583-3&volume=594&pages=265-270&publication_year=2021&author=Warnat-Herresthal%2CS)
201. Hie, B., Cho, H. & Berger, B. Realizing private and practical pharmacological collaboration. *Science* **362**, 347–350 (2018).

     [Article](https://doi.org/10.1126%2Fscience.aat4807) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2018Sci...362..347H) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvFWksbjJ) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30337410) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6519716) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Realizing%20private%20and%20practical%20pharmacological%20collaboration&journal=Science&doi=10.1126%2Fscience.aat4807&volume=362&pages=347-350&publication_year=2018&author=Hie%2CB&author=Cho%2CH&author=Berger%2CB)
202. Rohrbach, S. et al. Digitization and validation of a chemical synthesis literature database in the ChemPU. *Science* **377**, 172–180 (2022).

     [Article](https://doi.org/10.1126%2Fscience.abo0058) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2022Sci...377..172R) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB38XhslCnsbbK) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35857541) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Digitization%20and%20validation%20of%20a%20chemical%20synthesis%20literature%20database%20in%20the%20ChemPU&journal=Science&doi=10.1126%2Fscience.abo0058&volume=377&pages=172-180&publication_year=2022&author=Rohrbach%2CS)
203. Gysi, D. M. et al. Network medicine framework for identifying drug-repurposing opportunities for COVID-19. *Proc. Natl Acad. Sci. USA* **118**, e2025581118 (2021).

     [Article](https://doi.org/10.1073%2Fpnas.2025581118) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtVyhtbjP) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Network%20medicine%20framework%20for%20identifying%20drug-repurposing%20opportunities%20for%20COVID-19&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.2025581118&volume=118&publication_year=2021&author=Gysi%2CDM)
204. King, R. D. et al. The automation of science. *Science* **324**, 85–89 (2009).

     [Article](https://doi.org/10.1126%2Fscience.1165620) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2009Sci...324...85K) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BD1MXjvVajsbo%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19342587) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20automation%20of%20science&journal=Science&doi=10.1126%2Fscience.1165620&volume=324&pages=85-89&publication_year=2009&author=King%2CRD)
205. Mirdita, M. et al. ColabFold: making protein folding accessible to all. *Nat. Methods* **19**, 679–682 (2022).
206. Doerr, S. et al. TorchMD: a deep learning framework for molecular simulations. *J. Chem. Theory Comput.* **17**, 2355–2363 (2021).

     [Article](https://doi.org/10.1021%2Facs.jctc.0c01343) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXmsFWgs7k%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33729795) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8486166) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=TorchMD%3A%20a%20deep%20learning%20framework%20for%20molecular%20simulations&journal=J.%20Chem.%20Theory%20Comput.&doi=10.1021%2Facs.jctc.0c01343&volume=17&pages=2355-2363&publication_year=2021&author=Doerr%2CS)
207. Schoenholz, S. S. & Cubuk, E. D. JAX MD: a framework for differentiable physics. In *Advances in Neural Information Processing* Systems **33**, 11428–11441 (2020).
208. Peters, J., Janzing, D. & Schölkopf, B. *Elements of Causal Inference: Foundations and Learning Algorithms* (MIT Press, 2017).
209. Bengio, Y. et al. A meta-transfer objective for learning to disentangle causal mechanisms. In *International Conference on Learning Representations* (2020).
210. Schölkopf, B. et al. Toward causal representation learning. *Proc. IEEE* **109**, 612–634 (2021).

     [Article](https://doi.org/10.1109%2FJPROC.2021.3058954) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Toward%20causal%20representation%20learning&journal=Proc.%20IEEE&doi=10.1109%2FJPROC.2021.3058954&volume=109&pages=612-634&publication_year=2021&author=Sch%C3%B6lkopf%2CB)
211. Goyal, A. & Bengio, Y. Inductive biases for deep learning of higher-level cognition. *Proc. R. Soc. A* **478**, 20210068 (2022).
212. Deleu, T. et al. Bayesian structure learning with generative flow networks. In *Conference on Uncertainty in Artificial Intelligence* 518–528 (2022).
213. Geirhos, R. et al. Shortcut learning in deep neural networks. *Nat. Mach. Intell.* **2**, 665–673 (2020).

     [Article](https://doi.org/10.1038%2Fs42256-020-00257-z) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Shortcut%20learning%20in%20deep%20neural%20networks&journal=Nat.%20Mach.%20Intell.&doi=10.1038%2Fs42256-020-00257-z&volume=2&pages=665-673&publication_year=2020&author=Geirhos%2CR)
214. Koh, P. W. et al. WILDS: a benchmark of in-the-wild distribution shifts. In *International Conference on Machine Learning* 5637–5664 (2021).
215. Luo, Z. et al. Label efficient learning of transferable representations across domains and tasks. In *Advances in Neural Information Processing Systems* **30**, 165–177 (2017).
216. Mahmood, R. et al. How much more data do I need? estimating requirements for downstream tasks. In *IEEE Conference on Computer Vision and Pattern Recognition* 275–284 (2022).
217. Coley, C. W., Eyke, N. S. & Jensen, K. F. Autonomous discovery in the chemical sciences part II: outlook. *Angew. Chem. Int. Ed.* **59**, 23414–23436 (2020).

     [Article](https://doi.org/10.1002%2Fanie.201909989) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtFCjtL7M) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Autonomous%20discovery%20in%20the%20chemical%20sciences%20part%20II%3A%20outlook&journal=Angew.%20Chem.%20Int.%20Ed.&doi=10.1002%2Fanie.201909989&volume=59&pages=23414-23436&publication_year=2020&author=Coley%2CCW&author=Eyke%2CNS&author=Jensen%2CKF)
218. Gao, W. & Coley, C. W. The synthesizability of molecules proposed by generative models. *J. Chem. Inf. Model.* **60**, 5714–5723 (2020).

     [Article](https://doi.org/10.1021%2Facs.jcim.0c00174) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXmsFKrurw%3D) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32250616) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=The%20synthesizability%20of%20molecules%20proposed%20by%20generative%20models&journal=J.%20Chem.%20Inf.%20Model.&doi=10.1021%2Facs.jcim.0c00174&volume=60&pages=5714-5723&publication_year=2020&author=Gao%2CW&author=Coley%2CCW)
219. Kogler, R. et al. Jet substructure at the Large Hadron Collider. *Rev. Mod. Phys.* **91**, 045003 (2019).

     [Article](https://doi.org/10.1103%2FRevModPhys.91.045003) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019RvMP...91d5003K) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXjtFOntbs%3D) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Jet%20substructure%20at%20the%20Large%20Hadron%20Collider&journal=Rev.%20Mod.%20Phys.&doi=10.1103%2FRevModPhys.91.045003&volume=91&publication_year=2019&author=Kogler%2CR)
220. Acosta, J. N. et al. Multimodal biomedical AI. *Nat. Med.* **28**, 1773–1784 (2022).
221. Alayrac, J.-B. et al. Flamingo: a visual language model for few-shot learning. In *Advances in Neural Information Processing Systems* **35**, 23716–23736 (2022).
222. Elmarakeby, H. A. et al. Biologically informed deep neural network for prostate cancer discovery. *Nature* **598**, 348–352 (2021).

     [Article](https://doi.org/10.1038%2Fs41586-021-03922-4) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021Natur.598..348E) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXitV2itrzK) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34552244) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8514339) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Biologically%20informed%20deep%20neural%20network%20for%20prostate%20cancer%20discovery&journal=Nature&doi=10.1038%2Fs41586-021-03922-4&volume=598&pages=348-352&publication_year=2021&author=Elmarakeby%2CHA)
223. Qin, Y. et al. A multi-scale map of cell structure fusing protein images and interactions. *Nature* **600**, 536–542 (2021).

     [Article](https://doi.org/10.1038%2Fs41586-021-04115-9) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2021Natur.600..536Q) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXisFyrsbfJ) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34819669) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9053732) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=A%20multi-scale%20map%20of%20cell%20structure%20fusing%20protein%20images%20and%20interactions&journal=Nature&doi=10.1038%2Fs41586-021-04115-9&volume=600&pages=536-542&publication_year=2021&author=Qin%2CY)
224. Schaffer, L. V. & Ideker, T. Mapping the multiscale structure of biological systems. *Cell Systems* **12**, 622–635 (2021).

     [Article](https://doi.org/10.1016%2Fj.cels.2021.05.012) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtlyqtLnO) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34139169) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8245186) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mapping%20the%20multiscale%20structure%20of%20biological%20systems&journal=Cell%20Systems&doi=10.1016%2Fj.cels.2021.05.012&volume=12&pages=622-635&publication_year=2021&author=Schaffer%2CLV&author=Ideker%2CT)
225. Stiglic, G. et al. Interpretability of machine learning-based prediction models in healthcare. *Wiley Interdiscip. Rev. Data Min. Knowl. Discov.* **10**, e1379 (2020).

     [Article](https://doi.org/10.1002%2Fwidm.1379) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Interpretability%20of%20machine%20learning-based%20prediction%20models%20in%20healthcare&journal=Wiley%20Interdiscip.%20Rev.%20Data%20Min.%20Knowl.%20Discov.&doi=10.1002%2Fwidm.1379&volume=10&publication_year=2020&author=Stiglic%2CG)
226. Erion, G. et al. A cost-aware framework for the development of AI models for healthcare applications. *Nat. Biomed. Eng.* **6**, 1384–1398 (2022).
227. Lundberg, S. M. et al. Explainable machine-learning predictions for the prevention of hypoxaemia during surgery. *Nat. Biomed. Eng.* **2**, 749–760 (2018).

     [Article](https://doi.org/10.1038%2Fs41551-018-0304-0) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31001455) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6467492) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Explainable%20machine-learning%20predictions%20for%20the%20prevention%20of%20hypoxaemia%20during%20surgery&journal=Nat.%20Biomed.%20Eng.&doi=10.1038%2Fs41551-018-0304-0&volume=2&pages=749-760&publication_year=2018&author=Lundberg%2CSM)
228. Sanders, L. M. et al. Beyond low Earth orbit: biological research, artificial intelligence, and self-driving labs. Preprint at <https://arxiv.org/abs/2112.12582> (2021).
229. Gagne, D. J. II et al. Interpretable deep learning for spatial analysis of severe hailstorms. *Mon. Weather Rev.* **147**, 2827–2845 (2019).

     [Article](https://doi.org/10.1175%2FMWR-D-18-0316.1) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019MWRv..147.2827G) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Interpretable%20deep%20learning%20for%20spatial%20analysis%20of%20severe%20hailstorms&journal=Mon.%20Weather%20Rev.&doi=10.1175%2FMWR-D-18-0316.1&volume=147&pages=2827-2845&publication_year=2019&author=Gagne%2CDJ)
230. Rudin, C. Stop explaining black box machine learning models for high stakes decisions and use interpretable models instead. *Nat. Mach. Intell.* **1**, 206–215 (2019).

     [Article](https://doi.org/10.1038%2Fs42256-019-0048-x) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35603010) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9122117) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Stop%20explaining%20black%20box%20machine%20learning%20models%20for%20high%20stakes%20decisions%20and%20use%20interpretable%20models%20instead&journal=Nat.%20Mach.%20Intell.&doi=10.1038%2Fs42256-019-0048-x&volume=1&pages=206-215&publication_year=2019&author=Rudin%2CC)
231. Koh, P. W. & Liang, P. Understanding black-box predictions via influence functions. In *International Conference on Machine Learning* 1885–1894 (2017).
232. Mirzasoleiman, B., Bilmes, J. & Leskovec, J. Coresets for data-efficient training of machine learning models. In *International Conference on Machine Learning* 6950–6960 (2020).
233. Kim, B. et al. Interpretability beyond feature attribution: quantitative testing with concept activation vectors (TCAV). In *International Conference on Machine Learning* 2668–2677 (2018).
234. Silver, D. et al. Mastering the game of go without human knowledge. *Nature* **550**, 354–359 (2017).

     [Article](https://doi.org/10.1038%2Fnature24270) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2017Natur.550..354S) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC2sXhs12ltLvM) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29052630) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Mastering%20the%20game%20of%20go%20without%20human%20knowledge&journal=Nature&doi=10.1038%2Fnature24270&volume=550&pages=354-359&publication_year=2017&author=Silver%2CD)
235. Baum, Z. J. et al. Artificial intelligence in chemistry: current trends and future directions. *J. Chem. Inf. Model.* **61**, 3197–3212 (2021).

     [Article](https://doi.org/10.1021%2Facs.jcim.1c00619) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFCit7jN) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=34264069) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Artificial%20intelligence%20in%20chemistry%3A%20current%20trends%20and%20future%20directions&journal=J.%20Chem.%20Inf.%20Model.&doi=10.1021%2Facs.jcim.1c00619&volume=61&pages=3197-3212&publication_year=2021&author=Baum%2CZJ)
236. Finlayson, S. G. et al. Adversarial attacks on medical machine learning. *Science* **363**, 1287–1289 (2019).

     [Article](https://doi.org/10.1126%2Fscience.aaw4399) 
     [ADS](http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019Sci...363.1287F) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtFCisr7M) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30898923) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7657648) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Adversarial%20attacks%20on%20medical%20machine%20learning&journal=Science&doi=10.1126%2Fscience.aaw4399&volume=363&pages=1287-1289&publication_year=2019&author=Finlayson%2CSG)
237. Urbina, F. et al. Dual use of artificial-intelligence-powered drug discovery. *Nat. Mach. Intell.* **4**, 189–191 (2022).

     [Article](https://doi.org/10.1038%2Fs42256-022-00465-9) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36211133) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9544280) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Dual%20use%20of%20artificial-intelligence-powered%20drug%20discovery&journal=Nat.%20Mach.%20Intell.&doi=10.1038%2Fs42256-022-00465-9&volume=4&pages=189-191&publication_year=2022&author=Urbina%2CF)
238. Norgeot, B. et al. Minimum information about clinical artificial intelligence modeling: the MI-CLAIM checklist. *Nat. Med.* **26**, 1320–1324 (2020).

     [Article](https://doi.org/10.1038%2Fs41591-020-1041-y) 
     [CAS](/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvVyqu7%2FJ) 
     [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32908275) 
     [PubMed Central](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7538196) 
     [Google Scholar](http://scholar.google.com/scholar_lookup?&title=Minimum%20information%20about%20clinical%20artificial%20intelligence%20modeling%3A%20the%20MI-CLAIM%20checklist&journal=Nat.%20Med.&doi=10.1038%2Fs41591-020-1041-y&volume=26&pages=1320-1324&publication_year=2020&author=Norgeot%2CB)

[Download references](https://citation-needed.springer.com/v2/references/10.1038/s41586-023-06221-2?format=refman&flavour=references)

## Acknowledgements

M.Z. gratefully acknowledges the support of the National Institutes of Health under R01HD108794, U.S. Air Force under FA8702-15-D-0001, awards from Harvard Data Science Initiative, Amazon Faculty Research, Google Research Scholar Program, Bayer Early Excellence in Science, AstraZeneca Research, Roche Alliance with Distinguished Scientists, and Kempner Institute for the Study of Natural and Artificial Intelligence. C.P.G. and Y.D. acknowledge the support from the U.S. Air Force Office of Scientific Research under Multidisciplinary University Research Initiatives Program (MURI) FA9550-18-1-0136, Defense University Research Instrumentation Program (DURIP) FA9550-21-1-0316, and awards from Scientific Autonomous Reasoning Agent (SARA), and AI for Discovery Assistant (AIDA). Any opinions, findings, conclusions or recommendations expressed in this material are those of the authors and do not necessarily reflect the views of the funders. We thank D. Hassabis, A. Davies, S. Mohamed, Z. Li, K. Ma, Z. Qiao, E. Weinstein, A. V. Weller, Y. Zhong and A. M. Brandt for discussions on the paper.

## Author information

Author notes

1. Hanchen Wang

   Present address: Department of Research and Early Development, Genentech Inc, South San Francisco, CA, USA
2. Hanchen Wang

   Present address: Department of Computer Science, Stanford University, Stanford, CA, USA
3. These authors contributed equally: Hanchen Wang, Tianfan Fu, Yuanqi Du

### Authors and Affiliations

1. Department of Engineering, University of Cambridge, Cambridge, UK

   Hanchen Wang & Joan Lasenby
2. Department of Computing and Mathematical Sciences, California Institute of Technology, Pasadena, CA, USA

   Hanchen Wang & Anima Anandkumar
3. Department of Computational Science and Engineering, Georgia Institute of Technology, Atlanta, GA, USA

   Tianfan Fu
4. Department of Computer Science, Cornell University, Ithaca, NY, USA

   Yuanqi Du & Carla P. Gomes
5. Department of Chemical Engineering, Massachusetts Institute of Technology, Cambridge, MA, USA

   Wenhao Gao & Connor W. Coley
6. Department of Computer Science, Stanford University, Stanford, CA, USA

   Kexin Huang & Jure Leskovec
7. Department of Physics, Massachusetts Institute of Technology, Cambridge, MA, USA

   Ziming Liu
8. Harvard-MIT Program in Health Sciences and Technology, Cambridge, MA, USA

   Payal Chandak
9. Mila – Quebec AI Institute, Montreal, Quebec, Canada

   Shengchao Liu, Andreea Deac, Jian Tang & Yoshua Bengio
10. Université de Montréal, Montreal, Quebec, Canada

    Shengchao Liu, Andreea Deac & Yoshua Bengio
11. Department of Earth, Environmental and Planetary Sciences, Brown University, Providence, RI, USA

    Peter Van Katwyk & Karianne Bergen
12. Data Science Institute, Brown University, Providence, RI, USA

    Peter Van Katwyk & Karianne Bergen
13. NVIDIA, Santa Clara, CA, USA

    Anima Anandkumar
14. Center for Computational Astrophysics, Flatiron Institute, New York, NY, USA

    Shirley Ho
15. Department of Astrophysical Sciences, Princeton University, Princeton, NJ, USA

    Shirley Ho
16. Department of Physics, Carnegie Mellon University, Pittsburgh, PA, USA

    Shirley Ho
17. Department of Physics and Center for Data Science, New York University, New York, NY, USA

    Shirley Ho
18. Google DeepMind, London, UK

    Pushmeet Kohli & Petar Veličković
19. Microsoft Research, Beijing, China

    Tie-Yan Liu
20. Department of Biomedical Informatics, Harvard Medical School, Boston, MA, USA

    Arjun Manrai & Marinka Zitnik
21. Department of Systems Biology, Harvard Medical School, Boston, MA, USA

    Debora Marks
22. Broad Institute of MIT and Harvard, Cambridge, MA, USA

    Debora Marks & Marinka Zitnik
23. Deep Forest Sciences, Palo Alto, CA, USA

    Bharath Ramsundar
24. BioMap, Beijing, China

    Le Song
25. Mohamed bin Zayed University of Artificial Intelligence, Abu Dhabi, United Arab Emirates

    Le Song
26. University of Illinois at Urbana-Champaign, Champaign, IL, USA

    Jimeng Sun
27. HEC Montréal, Montreal, Quebec, Canada

    Jian Tang
28. CIFAR AI Chair, Toronto, Ontario, Canada

    Jian Tang
29. Department of Computer Science and Technology, University of Cambridge, Cambridge, UK

    Petar Veličković
30. University of Amsterdam, Amsterdam, Netherlands

    Max Welling
31. Microsoft Research Amsterdam, Amsterdam, Netherlands

    Max Welling
32. DP Technology, Beijing, China

    Linfeng Zhang
33. AI for Science Institute, Beijing, China

    Linfeng Zhang
34. Department of Electrical Engineering and Computer Science, Massachusetts Institute of Technology, Cambridge, MA, USA

    Connor W. Coley
35. Harvard Data Science Initiative, Cambridge, MA, USA

    Marinka Zitnik
36. Kempner Institute for the Study of Natural and Artificial Intelligence, Harvard University, Cambridge, MA, USA

    Marinka Zitnik

Authors

1. Hanchen Wang

   [View author publications](/search?author=Hanchen%20Wang)

   Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Hanchen%20Wang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Hanchen%20Wang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
2. Tianfan Fu

   [View author publications](/search?author=Tianfan%20Fu)

   Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Tianfan%20Fu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Tianfan%20Fu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
3. Yuanqi Du

   [View author publications](/search?author=Yuanqi%20Du)

   Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yuanqi%20Du) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yuanqi%20Du%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
4. Wenhao Gao

   [View author publications](/search?author=Wenhao%20Gao)

   Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Wenhao%20Gao) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Wenhao%20Gao%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
5. Kexin Huang

   [View author publications](/search?author=Kexin%20Huang)

   Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Kexin%20Huang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Kexin%20Huang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
6. Ziming Liu

   [View author publications](/search?author=Ziming%20Liu)

   Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ziming%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ziming%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
7. Payal Chandak

   [View author publications](/search?author=Payal%20Chandak)

   Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Payal%20Chandak) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Payal%20Chandak%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
8. Shengchao Liu

   [View author publications](/search?author=Shengchao%20Liu)

   Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Shengchao%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Shengchao%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
9. Peter Van Katwyk

   [View author publications](/search?author=Peter%20Van%20Katwyk)

   Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Peter%20Van%20Katwyk) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Peter%20Van%20Katwyk%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
10. Andreea Deac

    [View author publications](/search?author=Andreea%20Deac)

    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Andreea%20Deac) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Andreea%20Deac%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
11. Anima Anandkumar

    [View author publications](/search?author=Anima%20Anandkumar)

    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Anima%20Anandkumar) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Anima%20Anandkumar%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
12. Karianne Bergen

    [View author publications](/search?author=Karianne%20Bergen)

    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Karianne%20Bergen) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Karianne%20Bergen%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
13. Carla P. Gomes

    [View author publications](/search?author=Carla%20P.%20Gomes)

    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Carla%20P.%20Gomes) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Carla%20P.%20Gomes%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
14. Shirley Ho

    [View author publications](/search?author=Shirley%20Ho)

    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Shirley%20Ho) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Shirley%20Ho%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
15. Pushmeet Kohli

    [View author publications](/search?author=Pushmeet%20Kohli)

    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Pushmeet%20Kohli) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Pushmeet%20Kohli%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
16. Joan Lasenby

    [View author publications](/search?author=Joan%20Lasenby)

    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Joan%20Lasenby) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Joan%20Lasenby%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
17. Jure Leskovec

    [View author publications](/search?author=Jure%20Leskovec)

    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jure%20Leskovec) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jure%20Leskovec%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
18. Tie-Yan Liu

    [View author publications](/search?author=Tie-Yan%20Liu)

    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Tie-Yan%20Liu) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Tie-Yan%20Liu%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
19. Arjun Manrai

    [View author publications](/search?author=Arjun%20Manrai)

    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Arjun%20Manrai) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Arjun%20Manrai%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
20. Debora Marks

    [View author publications](/search?author=Debora%20Marks)

    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Debora%20Marks) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Debora%20Marks%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
21. Bharath Ramsundar

    [View author publications](/search?author=Bharath%20Ramsundar)

    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Bharath%20Ramsundar) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Bharath%20Ramsundar%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
22. Le Song

    [View author publications](/search?author=Le%20Song)

    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Le%20Song) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Le%20Song%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
23. Jimeng Sun

    [View author publications](/search?author=Jimeng%20Sun)

    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jimeng%20Sun) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jimeng%20Sun%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
24. Jian Tang

    [View author publications](/search?author=Jian%20Tang)

    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Jian%20Tang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Jian%20Tang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
25. Petar Veličković

    [View author publications](/search?author=Petar%20Veli%C4%8Dkovi%C4%87)

    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Petar%20Veli%C4%8Dkovi%C4%87) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Petar%20Veli%C4%8Dkovi%C4%87%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
26. Max Welling

    [View author publications](/search?author=Max%20Welling)

    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Max%20Welling) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Max%20Welling%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
27. Linfeng Zhang

    [View author publications](/search?author=Linfeng%20Zhang)

    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Linfeng%20Zhang) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Linfeng%20Zhang%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
28. Connor W. Coley

    [View author publications](/search?author=Connor%20W.%20Coley)

    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Connor%20W.%20Coley) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Connor%20W.%20Coley%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
29. Yoshua Bengio

    [View author publications](/search?author=Yoshua%20Bengio)

    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Yoshua%20Bengio) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Yoshua%20Bengio%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)
30. Marinka Zitnik

    [View author publications](/search?author=Marinka%20Zitnik)

    Search author on:[PubMed](https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Marinka%20Zitnik) [Google Scholar](https://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Marinka%20Zitnik%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en)

### Contributions

All authors contributed to the design and writing of the paper, helped shape the research, provided critical feedback, and commented on the paper and its revisions. H.W., T.F., Y.D. and M.Z conceived the study and were responsible for overall direction and planning. W.G., K.H. and Z.L. contributed equally to this work (equal second authorship) and are listed alphabetically.

### Corresponding author

Correspondence to
[Marinka Zitnik](mailto:marinka@hms.harvard.edu).

## Ethics declarations

### Competing interests

The authors declare no competing interests.

## Peer review

### Peer review information

*Nature* thanks Brian Gallagher and Benjamin Nachman for their contribution to the peer review of this work.

## Additional information

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

## Rights and permissions

Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.

[Reprints and permissions](https://s100.copyright.com/AppDispatchServlet?title=Scientific%20discovery%20in%20the%20age%20of%20artificial%20intelligence&author=Hanchen%20Wang%20et%20al&contentID=10.1038%2Fs41586-023-06221-2&copyright=Springer%20Nature%20Limited&publication=0028-0836&publicationDate=2023-08-02&publisherName=SpringerNature&orderBeanReset=true)

## About this article

[![Check for updates. Verify currency and authenticity via CrossMark](data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>)](ai-leaders-insights/技术路径与方法论/深度研究流派/科学突破驱动/images/2510e6307f5db94dc99e7882ecef8623.jpg)

### Cite this article

Wang, H., Fu, T., Du, Y. *et al.* Scientific discovery in the age of artificial intelligence.
*Nature* **620**, 47–60 (2023). https://doi.org/10.1038/s41586-023-06221-2

[Download citation](https://citation-needed.springer.com/v2/references/10.1038/s41586-023-06221-2?format=refman&flavour=citation)

* Received: 30 March 2022
* Accepted: 16 May 2023
* Published: 02 August 2023
* Version of record: 02 August 2023
* Issue date: 03 August 2023
* DOI: https://doi.org/10.1038/s41586-023-06221-2

### Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy shareable link to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative

## This article is cited by

* ### [Redefining academic integrity: patterns of assignment cheating with generative AI tools](https://doi.org/10.1007/s43681-025-00863-w)

  + Yasemin KAHYAOĞLU ERDOĞMUŞ
  + Sebahat Sevgi UYGUR

  *AI and Ethics* (2026)
* ### [The application of organoids in treatment decision-making for digestive system cancers: progress and challenges](https://doi.org/10.1186/s12943-025-02429-0)

  + Yufei Wang
  + Limin Zhang
  + Boon-Cher Goh

  *Molecular Cancer* (2025)
* ### [Sex differences in bladder cancer: understanding biological and clinical implications](https://doi.org/10.1186/s13293-025-00715-6)

  + Prakash Chaudhary
  + Biplab Singha
  + Xue Li

  *Biology of Sex Differences* (2025)
* ### [Digital twins as global learning health and disease models for preventive and personalized medicine](https://doi.org/10.1186/s13073-025-01435-7)

  + Xinxiu Li
  + Joseph Loscalzo
  + Mikael Benson

  *Genome Medicine* (2025)
* ### [Deep learning-assisted screening and diagnosis of scoliosis: segmentation of bare-back images via an attention-enhanced convolutional neural network](https://doi.org/10.1186/s13018-025-05564-y)

  + Xingyu Duan
  + Xiaojun Ma
  + Ningkui Niu

  *Journal of Orthopaedic Surgery and Research* (2025)

---

*爬取时间: 2025-11-28 21:49:30*
