# Artificial Intelligence in Biomedical Sciences: A Scoping Review

**来源**: [https://pmc.ncbi.nlm.nih.gov/articles/PMC12360964/](https://pmc.ncbi.nlm.nih.gov/articles/PMC12360964/)

---

# 
            Artificial Intelligence in Biomedical Sciences: A Scoping Review - PMC
        

原文链接: https://pmc.ncbi.nlm.nih.gov/articles/PMC12360964/

Br J Biomed Sci

. 2025 Aug 5;82:14362. doi: [10.3389/bjbs.2025.14362](https://doi.org/10.3389/bjbs.2025.14362)

# Artificial Intelligence in Biomedical Sciences: A Scoping Review

[Rasha Abu-El-Ruz](https://pubmed.ncbi.nlm.nih.gov/?term=%22Abu-El-Ruz%20R%22%5BAuthor%5D)

### Rasha Abu-El-Ruz

1
Department of Biomedical Sciences, College of Health Sciences, QU Health, Qatar University, Doha, Qatar

Find articles by [Rasha Abu-El-Ruz](https://pubmed.ncbi.nlm.nih.gov/?term=%22Abu-El-Ruz%20R%22%5BAuthor%5D)

1, [Ali Hasan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hasan%20A%22%5BAuthor%5D)

### Ali Hasan

2
School of Arts and Sciences, Lebanese American University, Beirut, Lebanon

Find articles by [Ali Hasan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hasan%20A%22%5BAuthor%5D)

2, [Dima Hijazi](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hijazi%20D%22%5BAuthor%5D)

### Dima Hijazi

3
Department of Biological and Environmental Sciences, College of Art and Science, Qatar University, Doha, Qatar

Find articles by [Dima Hijazi](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hijazi%20D%22%5BAuthor%5D)

3, [Ovelia Masoud](https://pubmed.ncbi.nlm.nih.gov/?term=%22Masoud%20O%22%5BAuthor%5D)

### Ovelia Masoud

1
Department of Biomedical Sciences, College of Health Sciences, QU Health, Qatar University, Doha, Qatar

Find articles by [Ovelia Masoud](https://pubmed.ncbi.nlm.nih.gov/?term=%22Masoud%20O%22%5BAuthor%5D)

1, [Atiyeh M Abdallah](https://pubmed.ncbi.nlm.nih.gov/?term=%22Abdallah%20AM%22%5BAuthor%5D)

### Atiyeh M Abdallah

1
Department of Biomedical Sciences, College of Health Sciences, QU Health, Qatar University, Doha, Qatar

Find articles by [Atiyeh M Abdallah](https://pubmed.ncbi.nlm.nih.gov/?term=%22Abdallah%20AM%22%5BAuthor%5D)

1, [Susu M Zughaier](https://pubmed.ncbi.nlm.nih.gov/?term=%22Zughaier%20SM%22%5BAuthor%5D)

### Susu M Zughaier

4
College of Medicine, QU Health, Qatar University, Doha, Qatar

Find articles by [Susu M Zughaier](https://pubmed.ncbi.nlm.nih.gov/?term=%22Zughaier%20SM%22%5BAuthor%5D)

4, [Maha Al-Asmakh](https://pubmed.ncbi.nlm.nih.gov/?term=%22Al-Asmakh%20M%22%5BAuthor%5D)

### Maha Al-Asmakh

1
Department of Biomedical Sciences, College of Health Sciences, QU Health, Qatar University, Doha, Qatar

5
Biomedical Research Center, Qatar University, Doha, Qatar

Find articles by [Maha Al-Asmakh](https://pubmed.ncbi.nlm.nih.gov/?term=%22Al-Asmakh%20M%22%5BAuthor%5D)

1,5,\*

* Author information
* Article notes
* Copyright and License information

1
Department of Biomedical Sciences, College of Health Sciences, QU Health, Qatar University, Doha, Qatar

2
School of Arts and Sciences, Lebanese American University, Beirut, Lebanon

3
Department of Biological and Environmental Sciences, College of Art and Science, Qatar University, Doha, Qatar

4
College of Medicine, QU Health, Qatar University, Doha, Qatar

5
Biomedical Research Center, Qatar University, Doha, Qatar

✉

\*Correspondence: Maha Al-Asmakh, maha.alasmakh@qu.edu.qa

Received 2025 Jan 17; Accepted 2025 May 28; Collection date 2025.

Copyright © 2025 Abu-El-Ruz, Hasan, Hijazi, Masoud, Abdallah, Zughaier and Al-Asmakh.

This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.

[PMC Copyright notice](/about/copyright/)

PMCID: PMC12360964  PMID: [40837680](https://pubmed.ncbi.nlm.nih.gov/40837680/)

## Abstract

### Background

Artificial intelligence (AI) is increasingly playing important roles in healthcare diagnosis, treatment, monitoring, and prevention of diseases. Despite this widespread implementation of AI in biomedical sciences, it has yet to be characterized.

### Aim

The aim of this scoping review is to explore AI in biomedical sciences. Specific objectives are to synthesize six scopes addressing the characteristics of AI in biomedical sciences and to provide in-depth understanding of its relevance to education.

### Methods

This scoping review has been developed according to Arksey and O’Malley frameworks. PubMed, Embase, and Web of Science databases were searched using broad search terms without restrictions. Citations were imported into EndNote for screening and extraction. Data were categorized and synthesized to define six scopes discussing AI in biomedical sciences.

### Results

A total of 2,249 articles were retrieved for screening and extraction, and 192 articles were included in this review. Six scopes were synthesized from the extracted data: Scope (1): AI in biomedical sciences by decade, highlighting the increasing number of publications on AI in biomedical sciences. Scope (2): AI in biomedical sciences by region, showing that publications on AI in biomedical sciences mainly originate from high-income countries, particularly the USA. Scope (3): AI in biomedical sciences by model, identifying machine learning as the most frequently reported model. Scope (4): AI in biomedical sciences by discipline, with microbiology the discipline most commonly associated with AI in biomedical sciences. Scope (5): AI in biomedical sciences education, which was limited to only six studies, indicating a gap in research on the educational application of AI in biomedical sciences. Scope (6): Opportunities and limitations of AI in biomedical sciences, where major reported opportunities include efficiency, accuracy, universal applicability, and real-world application. Limitations include; model complexity, limited applicability, and algorithm robustness.

### Conclusion

AI has generally been under characterized in the biomedical sciences due to variability in AI models, disciplines, and perspectives of applicability.

**Keywords:** biomedical sciences, artificial intelligence, scoping review, clinical, NAACLS

## Introduction

Artificial Intelligence (AI) algorithms and models have been rapidly developed and trained to enhance various functions in biomedical operations. Biomedical sciences is one important healthcare profession that embraced AI in patient sampling, testing, interpretation, and diagnostic technologies [[1](#B1), [2](#B2)]. AI has been applied both virtually and physically to biomedical sciences, where virtual applications focus on information availability regarding testing and interpreting clinical results to support diagnosis and clinical decision-making, whereas the physical applications focus on robotics and tools that operate laboratory instruments with minimal human intervention [[3](#B3)–[7](#B7)].

Despite the wide application of AI in healthcare and biomedical sciences, its implementation and field translation are not always well characterized. The role of AI in biomedical sciences are being explored to further understand its implications [[8](#B8)]. For instance, Davenport 2019 reported that IBM developed the Watson AI model for precision medicine and cancer diagnosis, but its implementation was challenged by difficulties in training the model and its cost [[9](#B9)]. Abdulkareem 2021, discussed several obstacles to adopting AI in biomedical fields such as infection management, with major limitations including: data accessibility, need for large training data to develop the algorithms, difficulties in model updates, interdisciplinary expertise requirements, model validation, generalizability, high cost, and data privacy concerns [[10](#B10), [11](#B11)].

Despite these challenges, AI is increasingly being considered in clinical laboratory practice, including sample collection, processing, testing, and result interpretation. Many laboratories are now almost fully automated with minimal need for human input, and many of these automations imply AI working side-by-side with smart technologies [[12](#B12)]. In microbiology, AI has enabled the early and accurate detection of microbes, outbreak prediction, microbial growth monitoring, drug resistance profiling, and microbial genomics associated with disease development [[13](#B13)–[15](#B15)]. Similarly, AI-driven image analysis in haematology has helped to identify blood cell abnormalities, accelerating the diagnosis of diseases like leukemia [[16](#B16)]. Clinical chemistry laboratories also leverage AI to analyze plasma and amino acid profiles [[17](#B17)].

However, the application of AI in biomedical education, at both the pre-clinical and clinical levels, remains limited due to questions and concerns about its effectiveness, lack of standardization, and high costs [[18](#B18)]. Ethical concerns, including the risk of bias in AI models based on their training data, pose further challenges. Such biases may compromise educational outcomes if not carefully addressed [[19](#B19)–[24](#B24)]. Academic stakeholders also face challenges in making vital decisions regarding introducing AI in their curricula [[25](#B25)–[28](#B28)]. There were reported concerns of built-in biases and data confidentiality in AI models, this is contingent on the type of data used to train the model [[23](#B23), [24](#B24)].

This scoping review explores AI in biomedical sciences. Specific objectives are to synthesize six scopes addressing the characteristics of AI in biomedical sciences and to provide in-depth understanding of its relevance to education. The review answers two questions: what are the characteristics of AI in biomedical sciences? What are the applications of AI in biomedical sciences education?

The findings of this review will help in the evaluation of existing AI applications for potential use in pre-clinical and clinical education. A preliminary PubMed search revealed no similar reviews, indicating a lack of systematic evidence of AI applications in biomedical science. This study also offers foundational insights for academic stakeholders aiming to develop effective AI-enhanced strategies for biomedical sciences programs.

## Methodology

This scoping review was conducted according to Arksey and O’Malley frameworks [[29](#B29)]. We mapped the key concepts of AI research in biomedical sciences by summarizing and synthesizing the available sources of evidence. This study is a precursor of a forthcoming systematic review, in which we assessed the extent and nature of published research on this topic. We adhered to stages of scoping reviews outlined by Arksey and O’Malley 2005 [[29](#B29)], as well as the recommendations by Levac 2010 [[30](#B30)]. The stages included: developing two questions: what are the characteristics of AI in biomedical sciences? What are the applications of AI in biomedical sciences education? Followed by identifying the relevant studies by creating well-defined search strategies across three databases and evaluating their relevance. Then the selection criteria were carried out based on specific eligibility standards. The Qualitative data charting was performed using a detailed extraction sheet, following the recommendations of Ritchie and Spencer 2002 [[31](#B31)]. Finally, the data collection and synthesis included coding of keyword and thematic analysis, conducted in consultation with experts in Biomedical Sciences, resulting in the identification of six key scopes.

### Data Sources and Search Strategy

We conducted comprehensive searches of three databases: PubMed, Embase, and Web of Science. Our searches, completed on January 9, 2024, utilized broad search terms in titles and abstracts incorporating MeSH and Emtree terms expanded to include all subheadings, titles, and free-text terms ([Supplementary Appendix S1](#s10)). The citations obtained from our search strategy were imported to EndNote citation management software (v20.2.1). All team members contributed to the development of the search strategy, which was created in accordance with the Peer Review of Electronic Search Strategies (PRESS) 2015 checklist ([Supplementary Appendix S2](#s10)) [[32](#B32)].

### Study Selection and Eligibility Criteria

We included any publication of original research on the application of AI in the biomedical sciences. We excluded any publication not relying on primary data including, case reports, case series, editorials, expert opinion, commentaries, reviews, conference abstracts of peer-reviewed publications, and publications not relevant to educational, clinical, and research perspectives of biomedical sciences.

Disciplines within the biomedical sciences were defined according to the National Accrediting Agency for Clinical Laboratory Sciences (NAACLS) standards. Biomedical sciences referred to program studying the clinical disciplines of microbiology, chemistry, serology, haematology and blood banking, histotechnology, laboratory pathology, and laboratory management, all according to specific standards and policies for NAACLS accreditation [[33](#B33)–[35](#B35)]. NAACLS-accredited program graduates are eligible to take the American Society for Clinical Pathology (ASCP) board exam, which encompasses a broad spectrum of laboratory science topics essential for clinical competency [[36](#B36)–[39](#B39)].

### Data Extraction and Synthesis

Citations obtained following entry of our search strategy were imported into EndNote (v20.2.1) for screening. Duplicates were identified and deleted through EndNote library. The first screening entailed the identification of relevant or potentially relevant articles through title and abstract screening and was performed by two researchers. Then, the citations were rescreened, extracted, and double-extracted by three researchers and validated by a fourth reviewer. If a discrepancy was detected, consensus was reached with input from the corresponding author [[40](#B40)]. This scoping review was conducted systematically in accordance with the Preferred Reporting Items for Systematic Reviews and Meta-Analyses Extension for Scoping Reviews (PRISMA-ScR) checklist ([Supplementary Appendix S3](#s10)) [[41](#B41)].

Upon consensus between the research team, we adopted extraction variables relevant to our review. A spreadsheet for the extraction variables was developed according to the recommendation of Ritchie and Spencer 2002 [[31](#B31)], the included variables are: publication variables (author, citation, year of data collection, year of publication, country, and site type), study-related variables (study design, population type, sampling, gender, age group, mean age), and AI-related variables (name of AI, the purpose of AI, type of AI, description of AI aim, field, AI measure, effect size, reported effectiveness, advantages, limitations, conclusions, and real-life implementation).

The definition of AI in this review follows the World Health Organization’s guidance on the ethics and governance of AI for health, which is based on a recommendation by the Council on Artificial Intelligence of the Organization for Economic Co-operation and Development [[42](#B42)]. According to this definition, “*An AI system is a machine-based system that can, for a given set of human-defined objectives, make predictions, recommendations, or decisions influencing real or virtual environments. AI systems are designed to operate with varying levels of autonomy*” [[42](#B42)].

The protocol of this scoping review was registered in the Open Science Framework (OSF) on 4th June 2024 and was updated on 4th October 2024, with registration DOI: <https://doi.org/10.17605/OSF.IO/4F2EZ>.

### Data Synthesis

PRISMA flow chart was generated throughout the process of importing and extracting citations [[43](#B43)]. The extracted data were then thoroughly narratively mapped and summarized, followed by the construction of Six scopes. Scope (1): AI in biomedical sciences by decade. Scope (2): AI in biomedical sciences by region. Scope (3): AI in biomedical sciences by model. Scope (4): AI in biomedical sciences by discipline. Scope (5): AI in biomedical sciences education. Scope (6): Opportunities and limitations of AI in biomedical sciences. The latter scope involved coding keywords to generate categories of opportunities and limitations ([Supplementary Appendix S6](#s10)). The coding and categorizing of scopes were performed by three researchers, and uncertainties were discussed among all researchers until a consensus was achieved [[2](#B2), [44](#B44)]. Any unclear data from the citations was handled by discussion between the researchers and communication with the original authors whenever possible. Data were cleaned by four researchers and involved summarizing and reorganizing the data based on its relevance to this review.

## Results

### PRISMA Flow Chart

We identified 2,449 articles, which were imported into our library: 263 from Embase, 165 from PubMed, and 2,021 from Web of Science. After removing 190 duplicates, a total of 2,259 articles remained for screening. The screening involved evaluating titles and abstracts, followed by full-text review, ultimately yielding 192 articles for data extraction. Articles were excluded for the following reasons: conference abstracts (141), reviews (82), out of scope (1618), full text not found (69), non-English language (1), not AI related (66), and not relevant to biomedical science (90) ([Figure 1](#F1)).

#### FIGURE 1.

[![Flowchart detailing the identification of studies via databases and registers. Initially, 2,449 records were identified from Embase, PubMed, and Web of Science. After removing 190 duplicates, 2,259 records were screened. 1,841 records were excluded due to conference abstracts, reviews, or being out of scope. From 418 reports sought, 70 were excluded due to full-text unavailability or language. 348 reports were assessed for eligibility, excluding 148 not related to AI or biomedical sciences. Finally, 192 studies were included in the review.](ai-leaders-insights/产业应用与实践/科学发现/生物医学突破/images/526a67a5a2c38f5ccd63cb1a93ce313a.jpg)](ai-leaders-insights/产业应用与实践/科学发现/生物医学突破/images/be073235dff1eb049f92d9c5c63917f8.html)

[Open in a new tab](figure/F1/)

PRISMA flow chart.

### Extraction Tables

The extracted key variables from the citations were organized by discipline ([Supplementary Table S1](#s10)). Discipline (1): Microbiology and infectious diseases. Discipline (2): Laboratory pathology, histotechnology, and cytogynecology. Discipline (3): Clinical chemistry. Discipline (4): Genetics and forensics. Discipline (5): Laboratory management. Discipline (6): Haematology and blood bank. Discipline (7): General laboratory sciences (*AI applications that are not limited to specific discipline and utilized across all laboratory sciences, as indicated in the extracted articles*). Discipline (8): Serology and immunology.

Additionally, a matrix was synthesized by mapping the first four scopes with the fifth scope “AI in biomedical sciences education,” in order to further identify the articles of educational perspective across the scopes ([Supplementary Table S2](#s10)).

### Qualitative Thematic Analyses

The following six scopes were synthesized from the key extraction variables. The narrative description provides an overview regarding AI in biomedical sciences. Scope (1): AI in biomedical sciences by decade; Scope (2): AI in biomedical sciences by region; Scope (3): AI in biomedical sciences by model; Scope (4): AI in biomedical sciences by discipline; Scope (5): AI in biomedical science education; Scope (6): Opportunities and limitations of AI in biomedical sciences.

#### Scope (1): AI in Biomedical Sciences by Decade

The earliest article regarding use of AI in biomedical sciences was published in 1996 [[45](#B45)]. The extracted articles were classified by date into three major phases: pioneering phase (1996–2005), expansion phase (2006–2015), and prosperous phase (2016–2023) ([Figure 2](#F2)). In the pioneering phase, four articles attempted to explore AI in biomedical sciences, though none addressed the educational perspective. Two focused on clinical perspective (Liu 2001 and Prank 2005) [[46](#B46), [47](#B47)] while the other two examined the research perspective (Udelhoven 2000 and Giacomini 1996) [[45](#B45), [48](#B48)]. Liu 2001 aimed to develop a system capable of understanding and processing complex biomedical terms in written formats [[46](#B46)], while Prank 2005 predicted genotype from complex biochemical data, comparing linear and non-linear analytical methods to the performance of experienced clinicians [[47](#B47)]. Udelhoven 2000 established a hierarchical classification system for identifying bacteria using infrared spectra [[48](#B48)], and Giacomini 1996 employed AI to classify patients with HIV-1 using neural networks for improved clinical follow-up [[45](#B45)]. The main limitation of the models and algorithms during this decade was their complexity. Despite this, all studies reported high efficiency and accuracy with universal applicability [[45](#B45)]. All the studies in the pioneering phase were conducted in high-income countries including USA, Germany, Italy, followed by Taiwan and others.

##### FIGURE 2.

[![Line graph titled "Articles over years" showing the number of articles from 1995 to 2023. The count is steady below 5 until around 2015, then rises sharply to about 40 in 2023.](ai-leaders-insights/产业应用与实践/科学发现/生物医学突破/images/b62b4d1e327554d9eb094a26179c1245.jpg)](ai-leaders-insights/产业应用与实践/科学发现/生物医学突破/images/57cba7a2be757839679481f28e825cb1.html)

[Open in a new tab](figure/F2/)

The number of articles published over time\*. \*Sharp increase in relevant publications over the last decade, highlighting increased interest in AI use in biomedical sciences.

In the expansion phase, 11 articles were published, two of them addressed the educational perspective (Munshi 2006 and Zheng 2015) [[49](#B49), [50](#B50)]. Munshi 2006 utilized simulation and visualization to focus on computational approaches for understanding molecular and cellular systems, reporting high reliability but limited applicability [[49](#B49)]. Zheng 2015 explored linking entities from unstructured full-texts of biomedical literature to 300 ontologies for automatic knowledge enrichment for scientific literature. Their approach had high accuracy and reliability, although the complexity of the algorithm was a limitation [[50](#B50)]. Five of the remaining articles discussed the research perspective, covering various aspects such as problem-solving, linguistics, handling narrative texts, and analytical schemes ([Supplementary Tables S1, S2](#s10)). The remaining four articles discussed the clinical perspective including clinical data assessment and diagnosis of infectious diseases ([Supplementary Tables S1, S2](#s10)). The majority of the opportunities reported in this decade highlighted high accuracy, reliability, and efficiency. However, the complexity of algorithms and limited model robustness were noted as significant limitations.

In the prosperous phase, 177 articles were published, a significant increase compared to the 11 published in the expansion phase and four in the pioneering phase. The majority, 122 articles, focused on the clinical perspective across various disciplines, with microbiology being the most discussed, followed by histotechnology, general laboratory sciences, haematology, and others ([Supplementary Tables S1, S2](#s10)). These articles primarily addressed the testing and diagnostic aspects of existing models. Fifty-one articles explored the research perspective, discussing the development and training of models to enhance research findings in biomedical sciences ([Supplementary Tables S1, S2](#s10)). Of all the articles in the prosperous phase, only four discussed the educational perspective Ambite 2019, Aparicio 2018, Henderson 2020, and Tota 2021 [[51](#B51)–[54](#B54)]. Ambite 2019 explored a tool that enabled computers to learn from data without programming, designing the ERuDIte framework to foster open-source educational resources on the web. Aparicio 2018 investigated how educators teaching biomedical subjects at the university level used intelligent information access systems such as BioAnnote, CLEiM, and MedCMap. Henderson 2020 explored the use of AI to improve the learning environment by supporting student engagement and activities. Tota 2021 proposed a telepresence robot kit with modules for teaching and conducting remote laboratory experiments.

Articles published during this phase highlighted enhanced efficiency, universal applicability, and real-world clinical relevance. However, algorithm complexity, limited model robustness, and moderate accuracy and reliability were reported as limitations.

#### Scope (2): AI in Biomedical Sciences by Region

One-hundred and thirty-four articles were from high-income countries ([Supplementary Tables S1, S2](#s10)), particularly USA (44 articles), followed by Germany (13 articles), Italy (10 articles), United Kingdome and Taiwan (9 articles each), and Japan (5 articles). Five of the AI models exploring the education perspective were conducted in high-income countries [[49](#B49)–[53](#B53)]. The majority of studies in these countries focused on the clinical perspective (84 articles), with a particular emphasis on microbiology and histotechnology. The opportunities reported in these studies included enhanced efficiency, universal applicability, and real-world clinical relevance. However, limitations such as algorithm complexity, limited model robustness, and moderate accuracy and reliability were also noted.

Fifty-four articles were from middle-income countries, notably China (25 articles), followed by India and Türkiye (7 articles each), Romania (3 articles), Thailand (2 articles), and others. Only one article from this group investigated AI from an educational perspective [[54](#B54)]. Forty articles explored AI from a clinical perspective and 13 articles from a research perspective. The majority explored AI in microbiology (23 articles) and general laboratory sciences (14 articles), with one in forensic medicine [[55](#B55)] and seven in haematology. Similar to studies from high-income countries, those from middle-income countries reported multiple benefits, including enhanced efficiency, universal applicability, and real-world clinical relevance. However, limitations such as algorithm complexity, limited model robustness, and moderate accuracy and reliability were also noted.

Only four articles were published from low-income countries: one article from each of Iraq [[10](#B10)], Bangladesh [[56](#B56)], Ethiopia [[57](#B57)], and Pakistan [[58](#B58)]. Three of the articles investigated AI from a research perspective, while one focused on a clinical perspective. No articles addressed the educational perspective. Three were in the field of microbiology and one in haematology. The articles reported opportunities such as enhanced efficiency, real-world clinical applicability, and high accuracy; however, the robustness of these AI models was limited ([Supplementary Tables S1, S2](#s10)).

#### Scope (3): AI in Biomedical Sciences by Model

Forty-seven articles explored AI in biomedical sciences using deep learning models. The majority (34 articles) examined AI from a clinical perspective, primarily in microbiology as well as laboratory pathology, while 13 articles focused on AI from a research perspective, mainly in general laboratory sciences. None of the deep learning models addressed the educational perspective.

In addition, 81 articles studied AI in biomedical sciences through machine learning models. Of these, 53 articles focused on clinical applications, primarily in microbiology, general laboratory sciences, and clinical chemistry, followed by histotechnology. Twenty-four articles investigated AI from a research perspective, with the general laboratory sciences being the dominant discipline, followed by microbiology and haematology. Three articles, Ambite 2019, Henderson 2020, and Tota 2021 examined the educational perspective around AI in laboratory sciences [[51](#B51), [53](#B53), [54](#B54)].

Twenty-five articles reported on hybrid AI applications in biomedical sciences, combining machine learning with deep learning, natural language processing, convolutional neural networks, or deep neural networks. The majority of this research focused on general laboratory applications, followed by microbiology, histotechnology, haematology, clinical chemistry, and forensic medicine. While most studies explored the clinical implications of hybrid AI models, none addressed their potential educational applications.

Natural language processing models were reported in seven articles, with six focusing on general laboratory disciplines and one on microbiology. Of these, four studied AI from a research perspective, one from a clinical perspective, and two from an educational perspective (Aparicio 2018 and Zheng 2015) [[50](#B50), [52](#B52)]. Neural networks were discussed in four articles, three from clinical perspective and one research, each addressing microbiology, chemistry, haematology, and serology.

Artificial neural networks and automated AI-based diagnosis method were each reported in two articles. Knowledge discovery, semantic analysis, and simulation and visualization were each discussed in one article, with the latter examining AI in biomedical sciences from an educational perspective (Munshi 2006) [[49](#B49)] ([Figure 3](#F3)).

##### FIGURE 3.

[![Bar chart titled "Frequency of AI models across the perspectives of education, research, and clinical." It shows the number of articles for different AI models: Machine Learning (3 education, 22 research, 53 clinical), Deep Learning (13 research, 34 clinical), Hybrid (10 research, 15 clinical), Natural Language Processing (2 education, 4 research, 1 clinical), and Others (1 education, 4 research, 6 clinical). Bars are color-coded as blue for education, orange for research, and gray for clinical.](ai-leaders-insights/产业应用与实践/科学发现/生物医学突破/images/d2228326614f7d8faf6a555a6247b6e8.jpg)](ai-leaders-insights/产业应用与实践/科学发现/生物医学突破/images/4a104d7381e74139793c39da2547a2ee.html)

[Open in a new tab](figure/F3/)

*Bar chart showing the frequency of AI models in the reported articles\**. \*Others are automated AI-based diagnosis, generative adversarial networks, knowledge discovery and semantic analysis, and simulation and visualization.

#### Scope (4): AI in Biomedical Sciences by Discipline

Sixty-three articles focused on microbiology and infectious diseases. The main aims of these studies included pathogen identification, antimicrobial resistance prediction, automated image analysis and digital pathology, vaccine development, predictive modeling for disease outcomes and treatment, and biomedical text mining.

Twenty-four articles focused on laboratory pathology, histotechnology, and cytogynecology. The main aims of these studies included laboratory pathology and molecular analysis for cancer detection whether in body fluids samples or biopsies. Additionally, improving the diagnostic accuracy and consistency in cancer detection and therapy.

Ten articles focused on clinical chemistry, with the main aims including automation in urinalysis, biochemical and lipid analysis, clinical decision support systems, advanced applications in medical imaging, and automated scoring of laboratory assays.

Six articles focused on laboratory management, primarily addressing data management and clinical decision support systems.

Twenty articles examined haematology and blood bank applications, with key aims including automated blood smear analysis, abnormal blood cell identification, blood type identification, bone marrow cell classification, assessment of blood cell integrity in stored samples, screening and risk assessment for hematologic malignancies, blood clot detection, development of hematologic point-of-care testing systems, prediction of hemoglobin variants, and optimization of spectroscopic data analysis.

Fifty-eight articles explored general laboratory applications, those AI applications that are not limited to specific discipline and utilized across all laboratory sciences, as indicated in the extracted articles. The primary aims of those articles covered aspects including; conducting in-depth biomedical literature analyses, enhancing decision-making in intensive care unit laboratories, quality control applications, improving diagnostic accuracy through methods such as enhanced imaging, evaluating the effectiveness of AI use in biomedical laboratories, and proposing machine learning tools for non-expert biomedical technologists.

Four articles addressed serology, focusing on antigen identification and pathogen neutralization, improved immune cell treatment outcomes, classification of immune system diseases, and enhancing the diagnostic capabilities of serological tests through clinical cytometry ([Figure 4](#F4)).

##### FIGURE 4.

[![Bar chart showing the number of publications by discipline in biomedical science. Microbiology has the highest number of articles at 63, predominantly clinical. General Medical Lab shows 31 research articles, while Lab Pathology has a total of 20 articles. Other disciplines like Haematology, Genetics-Forensic, Clinical Chemistry, Lab Management, and Serology have fewer publications, all below 10. Categories include Education (blue), Research (orange), and Clinical (gray).](ai-leaders-insights/产业应用与实践/科学发现/生物医学突破/images/5d9a75432ab2cb21f25784adfc3d0848.jpg)](ai-leaders-insights/产业应用与实践/科学发现/生物医学突破/images/82a532c33e8c99ae339d9e07a4bc8bd7.html)

[Open in a new tab](figure/F4/)

Bar chart showing the number of publications by discipline in biomedical sciences\*. \*Publications from clinical perspective dominated across all biomedical science disciplines, with microbiology, general medical laboratory and haematology leading overall output. Publications from educational perspective were minimal, highlighting a gap in academic-focused literature.

#### Scope (5): AI in Biomedical Science Education: Further Depth

Of the extracted (192 articles), six reported the use of AI in biomedical sciences education [[49](#B49)–[54](#B54)].

The earliest paper investigating AI in education was a population-based, cross-sectional observational study by Munshi 2006. This study was conducted in the USA, employed simulation and visualization AI techniques within the general laboratory sciences discipline. The primary goal was to leverage computational and mathematical approaches to unravel the complexities of molecular and cellular systems. By simulating and visualizing biological processes at multiple scales, the study authors aimed to comprehend the complex biomedical processes, enable the observation and analysis of interactions at these scales, and facilitate hypothesis testing and model validation. While the study demonstrated high reliability, its applicability was limited [[49](#B49)].

A Romanian study by Tota 2021 proposed a computational algorithmic design for developing Telepresence Robotics, a technology to enable remote participation in courses and laboratories, allowing students to interact physically with teaching and laboratory equipment through robotic avatars. The robotic system incorporated AI to facilitate real-time measurements and error correction. The study advocated for a modular Telepresence Robot Kit, easy to assemble and adaptable to various educational settings. Machine learning AI was employed as a key tool in this innovative approach [[54](#B54)].

Another population-based study by Aparicio 2018, conducted in Spain using a qualitative mixed-method design, aimed to gather expert opinions on the use of intelligent systems for processing text to enhance learning activities in biomedical sciences. Interviews were conducted with educators using a questionnaire containing 66 predefined closed- and open-ended questions. The results revealed that teachers highly valued the integration of reliable information sources, bilingualism, and selective annotation of concepts [[52](#B52)]. This study focused on general biomedical laboratory disciplines and utilized a natural language processing AI tool.

A study by Zheng 2015, in the USA, explored a computational supervised entity-linking design. This unsupervised collective inference approach linked entities from unstructured full texts of biomedical literature to ontologies. The method leveraged the rich semantic information and structure in ontologies for similarity computation and entity ranking. Despite using no labeled data, the unsupervised approach outperformed a state-of-the-art supervised method that was trained on a large amount of manually labeled data. The study covered general biomedical laboratory disciplines and utilized a natural language processing AI tool. This method offered the potential to save scientists an enormous amount of time by providing easy access to specific data within a vast pool of information [[50](#B50)].

Another US-based study by Henderson 2020 employed a computational model-based approach to compare various machine learning-based affective models. The research aimed to recognize students' affective states within a game-based learning environment, utilizing competing feature-level and decision-level multimodal data fusion approaches. The results indicated that multimodal affect detectors, driven by posture- and interaction-based data, were effective in identifying emotional states. The AI tool examined in this study was a machine learning AI model [[53](#B53)].

A computational entity-linking study, conducted in the USA by Ambite in 2019, proposed ERuDITe, a platform that organized over 11,000 data science training resources, including courses, tutorials, and talks, specifically for biomedical researchers. Using machine learning, the platform tagged resources with relevant concepts from a newly created data science education ontology and linked people and organizations to public databases like DBpedia and ORCID. This approach provided access to personalized training and connected biomedical researchers to linked educational resources in data science [[51](#B51)].

#### Scope (6): Opportunities and Limitations of AI in Biomedical Sciences

Ambite in 2019 reported high accuracy and reliability for AI software like ERuDIte, with its comprehensive collection of over 11,000 data science training resources, including courses, video tutorials, and conference talks. The metadata for these resources was uniformly described using [Schema.org](http://Schema.org), enhancing consistency and discoverability. However, the study discussed limitations related to algorithm complexity. Despite ERuDIte’s extensive data collection, the task of automatically identifying and organizing such diverse training resources is inherently complex, and the model’s robustness remains limited [[51](#B51)].

Aparicio 2018 highlighted the potential of AI in higher education, particularly in the biomedical field. Their research explored the benefits and challenges of integrating intelligent information access systems into active learning environments. They identified opportunities to enhance teaching practices, boost student engagement, and gain insights into educators' attitudes towards technological integration in biomedical education. While AI offers significant potential, the study also acknowledged limitations such as algorithm complexity, limited data sources, and technical constraints [[52](#B52)].

Tota 2021 proposed a telepresence robot kit containing easily assembled modules adapted for various teaching situations, specifically for the development of remote laboratory experiments. Users could create their own designs and adapt the electronic components and software to their needs through 3D printing and reprogrammable development boards. With components tailored to each situation, both simple laboratory tasks and complex operations, such as transporting and monitoring laboratory samples, could be performed remotely in real time, offering universal applicability. While this study advocated for telepresence in remote laboratory experiments, it acknowledged the complexity of the algorithm and the limited applicability of the AI software. The use of AI for text detection and recognition is challenged by the need for diverse requirements across different scenarios e.g., text in street scenes for robot navigation vs. receipts for optical character recognition (OCR) in financial departments. Also, difficulties in handling lower-quality or degraded data e.g., scanned legacy books in Google Books [[54](#B54)].

The approach presented by Zheng 2015 significantly outperformed state-of-the-art entity-linking methods, even without using labeled data. This would save scientists, particularly those focused on keeping informed about research developments, an enormous amount of time. All methods were highly accurate and efficient. However, the disambiguation algorithm assumed that phrases within the same sentence or paragraph were related, potentially undermining the entity linking (EL) performance [[50](#B50)].

Henderson 2020 demonstrated that combining different types of data creates a more accurate model for understanding and predicting the unique signals of emotions. By merging these data types, the model developed a comprehensive view of learners, leading to more accurate predictions than using just one data type. However, this study proposed an AI model that focused solely on specific datasets, neglecting some important affective states due to limited available observations. Additionally, the model was designed for specific learning environments, limiting its generalizability and robustness [[53](#B53)].

Simulation and visualization, while powerful tools for understanding complex biological processes, have limitations in real-time clinical practice. Implementing interdisciplinary programs that integrate simulation and visualization can also present challenges, such as group heterogeneity and time constraints. However, Munshi 2006 demonstrated that simulation and visualization can offer significant benefits, such as enhanced comprehension of complex biological processes, the ability to observe and analyze interactions at multiple scales, facilitation of hypothesis testing and model validation, and insights into the emergent properties of biological systems [[49](#B49)].

## Discussion

This scoping review synthesized six scopes addressing the characteristics of AI in biomedical sciences and provided in-depth understanding of its relevance to education. We summarized articles reporting on the implications of AI in biomedical sciences. Out of 2,249 publications, only 192 met our eligibility criteria and reported AI in one of the disciplines recognized by the NAACLS accreditation standards for biomedical sciences program and the disciplines of the ASCP board exam for biomedical scientists [[34](#B34), [35](#B35), [37](#B37)–[39](#B39)]. Of the 192 articles, only six directly addressed the educational perspective of their AI tools. The integration of AI in biomedical sciences dates back to 1996, when Giacomini proposed the use of a deep neural network for the clinical classification of HIV-1 patients based on P-24 and CD4 count [[45](#B45)].

There was a sharp increase in the number of articles published during the recent prosperous phase compared with earlier years. The AI tools reported in this phase covered all types of AI approach, including machine learning, deep learning, neural networks, and simulation and visualization, across various biomedical science disciplines. This trend aligns with Bohr 2020, who emphasized the need for healthcare institutions to keep pace with technological advances in healthcare and the growing demand for improved healthcare delivery outcomes [[59](#B59)]. Similarly, high-income countries are advancing rapidly in AI technology due to factors such as greater feasibility of utilization, budget availability, digital data availability, and infrastructure readiness, which are known limitations in low-income countries [[60](#B60)].

Microbiology has emerged as a leading discipline in the adoption of AI in biomedical sciences. Egli 2020 highlighted the significant shift toward digital microbiology, where advanced laboratory technologies powered by machine learning algorithms are revolutionizing the field. This trend is further fueled by the growth of microbiology big data. The increased utilization of AI in microbiology promises significant benefits, including improved testing, diagnosis, and treatment of infectious diseases, while also supporting related sectors such as antimicrobial stewardship, public health surveillance, quality control, and sepsis management [[61](#B61)].

There is limited evidence regarding the application of AI in biomedical sciences from an educational perspective, with only six studies discussing AI in education. Several limitations associated with incorporating AI into education were identified. Pedro 2019 reviewed the complexity of integrating AI into curricula, educational policies, public policies, equity considerations, risk assessment, sustainability, ethical concerns, and infrastructure needs [[62](#B62)]. Furthermore, Kuleto 2021 discussed the costs associated with AI in higher education, including structural costs, IT costs per student, and staffing costs [[63](#B63)]. Da Silva recently proposed autonomous experimentation (AE) systems, also known as self-driving laboratories or materials acceleration platforms, which are capable of running a large number of experiments autonomously. However, one challenge discussed was convincing stakeholders from emerging economies to invest in AE systems [[64](#B64)]. Many healthcare professionals, educators, and politicians have advocated making data available while maintaining anonymity to facilitate its integration into AI systems [[65](#B65)].

Machine learning was the most commonly reported AI approach used in the extracted articles, followed by deep learning. In machine learning models, AI can learn and adapt without manual input, using statistical models and algorithms to interpret the data [[66](#B66)]. Deep learning is a combination of machine learning and artificial neural networks. It is a self-learning software that layers algorithms into neural networks, allowing the system to teach itself [[48](#B48)]. Machine learning can be used for both data description and prediction. Based on the data, it can also provide models that serve as benchmarks for comparing human-led experiments. Artificial neural networks are important as prediction tools and can be used to adjust educational resources according to the individualized learning needs of students [[67](#B67)]. Deep learning-based algorithms can be combined with facial recognition software to assess students' attentiveness in online classes on platforms like Zoom, Google Classroom, and others [[68](#B68)]. Natural learning processing is an AI software model that enables computers to understand and communicate in human language, making interactions more intuitive and convenient [[68](#B68)]. Other AI models, such as convolutional neural networks, simulation, and visualization, can be used in a similar way.

The new era of AI in biomedical education should focus on incorporating AI systems into curricula and examining their application in real-life settings. Some initiatives have already begun, such as at Harvard Medical School, which has developed AI software capable of screening for a wide range of cancers and which can also be used for educational purposes [[69](#B69)]. Carnegie Mellon University is developing an AI-based voice pilot that will allow humans to communicate with robots [[70](#B70)], and the University of Southern California is using AI for teaching, learning, and campus research [[71](#B71)]. Hong Kong is incorporating AI education, AI ethics, and the social impact of AI into its school curriculum [[72](#B72)]. In India, the state of Andhra Pardesh is establishing two boards tasked with incorporating the use of AI in higher education [[73](#B73)]. The UK government has recognized the potential of generative AI tools like ChatGPT to support educators, initiating collaborations with educational stakeholders to explore how these tools can be used to analyze and structure information, ultimately aiding teachers in their work [[74](#B74)].

### Clinical and Research Studies With Potential Implications for Education

Alachram 2021 conducted a study at the University Medical Center, Taiwan, developing a Word2Vec approach using a corpus of 16 million PubMed abstracts. This approach helped to narrow down specific terms by leveraging this large database [[75](#B75)]. Anthony Q 2021 investigated deep learning models in clinical settings for super-resolution (DSLR), an emerging trend in response to the growing need for high-resolution images in machine learning/deep learning applications. The technique has shown promise for biomedical imaging, surveillance, and microscopy, with potential applications in education [[76](#B76)]. Deep learning-based virtual staining techniques, as discussed in Bai 2023, have enabled rapid, cost-effective, and chemical-free histopathology, offering a powerful alternative to traditional histological staining methods that have been in use for over a century [[77](#B77)]. Bonatti 2022 discussed extrusion-based bioprinting (EBB), an easy-to-use hardware system capable of printing a wide variety of materials. The paper advocated for the use of machine learning in the quality control of EBB, which could help publishers disseminate knowledge for educational purposes [[78](#B78)]. Gao 2019 demonstrated that the Edge2Vec model, in the research setting, could exploit machine learning and deep learning to provide powerful analytics and tools for graph analysis across three biomedical domain tasks: biomedical entity classification, compound-gene bioactivity prediction, and biomedical information retrieval [[79](#B79)]. Goncalves 2020 also explored deep learning AI based on convolutional layers and bidirectional gated recurrent units for automatic classification of abstract sentences into their main elements at a research center [[80](#B80)]. Hill 2016 conducted a study in a German University clinic on the use of machine learning to collect, process, analyze, and visualize data in real time, integrating cloud capabilities to ensure accessibility and scalability [[81](#B81)]. The approach supported a wide range of applications in both sports and biomedical disciplines, enhancing the usability and effectiveness of the sensor system. In a Chinese University research center, Li 2022 utilized domain-tuning pre-trained language models to explore methods for integrating imprecise knowledge into prompt-tuning verbalization techniques within the context of biomedical text to stimulate the rich knowledge distribution.

Ultra-high-speed imaging serves as a foundation for modern biomedical science. Liu 2022 conducted a study in a research setting and proposed all-fiber imaging at high speeds, achieved by transforming two-dimensional spatial information into one-dimensional temporal pulsed streams using high intermodal dispersion in multimode fibers. This technique could detect high-quality content-aware images and also images of various types from slightly reduced-quality training data. The imaging technique incorporated deep learning and improved imaging techniques could ultimately lead to better educational outcomes in biomedical science [[82](#B82)]. Luo 2022 used deep learning to train a set of transmissive diffractive surfaces that all-optically reconstructed images of arbitrary objects, even when completely covered by unknown, random phase diffusers [[83](#B83)]. The AI model faced challenges with adaptation, however if these challenges are addressed, it could usher in a new era of imaging systems capable of seeing through at the speed of light without the need for digital computation. This breakthrough would open up numerous new applications in biomedical imaging, astronomy, astrophysics, atmospheric sciences, security, robotics, and beyond. Similarly, Meade 2009 used machine learning in a research setting to optimize the entire analytical scheme and maximize the predictive capacity of spectroscopic data [[84](#B84)]. Nguyen 2022 assessed deep learning and machine learning to overcome the limitations of traditional design approaches in polymer science [[85](#B85)]. Rivenson 2018 provided proof-of-concept of a deep learning-based framework to enhance mobile phone-based microscopy by creating high-resolution, denoised, and color-corrected images through a convolutional neural network via a deep learning approach. This enhancement was achieved by training a deep convolutional neural network using smartphone microscope images and corresponding benchtop microscope images of various specimens [[86](#B86)]. Chiu 2017 used an AI model of limited maturity in a university set-up to create a system with a sparse annotation set to train and evaluate many phenotypes at once, known as “bulk learning” [[87](#B87)]. Liu 2001 used machine learning to develop a system that can understand and process complex medical terms [[46](#B46)].

More recently, Cadamuro 2023 explored ChatGPT’s ability and usefulness for interpreting laboratory test results (185). Similarly, Kumari 2023 studied ChatGPT-3.5, Google Bard, and Microsoft Bing in solving haematology-related cases, comparing their performance using a natural language processing AI tool [[88](#B88)]. Although ChatGPT is not an evidence-based tool, it has been increasingly used to search for information and synthesize knowledge. However, ChatGPT may be an extremely useful educational tool if used in legitimate and regulated ways.

A strength of this review lies in the large number of articles contributing to the synthesized scopes, the diversity of AI models used across biomedical fields, and the use of standardized procedures, structured validation, and the integration of multiple scopes to understand the characteristics of AI in biomedical sciences from various perspectives. However, the limitations of this review include the challenge of dealing with various study designs that cannot be critically appraised in the same manner. The majority of articles included are computational studies focusing on modeling, logarithmic, empirical, or factorial designs. There is, however, limited evidence from population-based studies, such as observational or experimental studies, which would provide more translationally meaningful outcomes. Other challenges include the complexity of the information extracted, requiring an interdisciplinary team, and the fact that we did not appraise the quality of the studies. Additionally, there may be contamination between some sub-categories of the scopes, such as disciplines, AI models, opportunities, and limitations.

## Conclusion

AI in biomedical sciences has been underexplored due to variability in AI models, disciplines, study designs, and perspectives of applicability. While the majority of AI applications in biomedical sciences are focused on clinical and research perspectives, studies exploring AI in biomedical education remain scarce. Furthermore, the quality of publications examining AI in biomedical sciences requires further standardization to ensure consistency and reliability across studies.

## Funding Statement

The author(s) declare that financial support was received for the research and/or publication of this article. Dima Hijazi is on a graduate assistantship scholarship from Qatar University.

## Author Contributions

MA-A: corresponding author, conceived the study, protocol development, study design, study management, discussion, manuscript editing and review. RA-E-R: First author, drafting the manuscript, data extraction, generation of categories, results, discussion, review, figures. AH: Data extraction, validation, cleaning, coding, analyses and review. DH: Data extraction, validation, cleaning, review. OM and AH: Data extraction, validation, cleaning, review. AA: review and editing the final manuscript. SZ: Protocol, layout, review and editing the final manuscript. All authors contributed to the article and approved the submitted version.

## Conflict of Interest

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Generative AI Statement

The author(s) declare that no Generative AI was used in the creation of this manuscript.

## Supplementary Material

The Supplementary Material for this article can be found online at: <https://www.frontierspartnerships.org/articles/10.3389/bjbs.2025.14362/full#supplementary-material>

[DataSheet2.docx](/articles/instance/12360964/bin/DataSheet2.docx) (243.8KB, docx)

[DataSheet1.docx](/articles/instance/12360964/bin/DataSheet1.docx) (98.9KB, docx)

## References

* 1.
  Jiang F, Jiang Y, Zhi H, Dong Y, Li H, Ma S, et al.
  Artificial Intelligence in Healthcare: Past, Present and Future. Stroke Vasc Neurol (2017) 2(4):230–43. 10.1136/svn-2017-000101
   [[DOI](https://doi.org/10.1136/svn-2017-000101)] [[PMC free article](/articles/PMC5829945/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29507784/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stroke%20Vasc%20Neurol&title=Artificial%20Intelligence%20in%20Healthcare:%20Past,%20Present%20and%20Future&author=F%20Jiang&author=Y%20Jiang&author=H%20Zhi&author=Y%20Dong&author=H%20Li&volume=2&issue=4&publication_year=2017&pages=230-43&pmid=29507784&doi=10.1136/svn-2017-000101&)]
* 2.
  Kueper JK, Terry AL, Zwarenstein M, Lizotte DJ. Artificial Intelligence and Primary Care Research: A Scoping Review. Ann Fam Med (2020) 18(3):250–8. 10.1370/afm.2518
   [[DOI](https://doi.org/10.1370/afm.2518)] [[PMC free article](/articles/PMC7213996/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32393561/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Fam%20Med&title=Artificial%20Intelligence%20and%20Primary%20Care%20Research:%20A%20Scoping%20Review&author=JK%20Kueper&author=AL%20Terry&author=M%20Zwarenstein&author=DJ%20Lizotte&volume=18&issue=3&publication_year=2020&pages=250-8&pmid=32393561&doi=10.1370/afm.2518&)]
* 3.
  Hamet P, Tremblay J. Artificial Intelligence in Medicine. Metabolism (2017) 69s:S36–s40. 10.1016/j.metabol.2017.01.011
   [[DOI](https://doi.org/10.1016/j.metabol.2017.01.011)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/28126242/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Metabolism&title=Artificial%20Intelligence%20in%20Medicine&author=P%20Hamet&author=J%20Tremblay&volume=69s&publication_year=2017&pages=S36-s40&pmid=28126242&doi=10.1016/j.metabol.2017.01.011&)]
* 4.
  Felder RA. Laboratory Systems Integration: Robotics and Automation. Ann Biol Clin (Paris) (1991) 49(5):298–300.
   [[PubMed](https://pubmed.ncbi.nlm.nih.gov/1928847/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Biol%20Clin%20(Paris)&title=Laboratory%20Systems%20Integration:%20Robotics%20and%20Automation&author=RA%20Felder&volume=49&issue=5&publication_year=1991&pages=298-300&pmid=1928847&)]
* 5.
  Goto T, Hongo K, Yako T, Hara Y, Okamoto J, Toyoda K, et al.
  The Concept and Feasibility of EXPERT: Intelligent Armrest Using Robotics Technology. Neurosurgery (2013) 72:A39–A42. 10.1227/NEU.0b013e318271ee66
   [[DOI](https://doi.org/10.1227/NEU.0b013e318271ee66)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23254811/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurosurgery&title=The%20Concept%20and%20Feasibility%20of%20EXPERT:%20Intelligent%20Armrest%20Using%20Robotics%20Technology&author=T%20Goto&author=K%20Hongo&author=T%20Yako&author=Y%20Hara&author=J%20Okamoto&volume=72&publication_year=2013&pages=A39-A42&pmid=23254811&doi=10.1227/NEU.0b013e318271ee66&)]
* 6.
  Demir U, Kocaoglu S, Akdogan E. Human Impedance Parameter Estimation Using Artificial Neural Network for Modelling Physiotherapist Motion. Biocybernetics Biomed Eng (2016) 36(2):318–26. 10.1016/j.bbe.2016.01.002
   [[DOI](https://doi.org/10.1016/j.bbe.2016.01.002)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biocybernetics%20Biomed%20Eng&title=Human%20Impedance%20Parameter%20Estimation%20Using%20Artificial%20Neural%20Network%20for%20Modelling%20Physiotherapist%20Motion&author=U%20Demir&author=S%20Kocaoglu&author=E%20Akdogan&volume=36&issue=2&publication_year=2016&pages=318-26&doi=10.1016/j.bbe.2016.01.002&)]
* 7.
  Basok BI. Artificial Intelligence Based Approaches in Medical Laboratories: University of Health Sciences Tepecik Training and Research Hospital Experiences. Turkish J Biochem (2018) 43:18. 10.1515/tjb-2018-0018
   [[DOI](https://doi.org/10.1515/tjb-2018-0018)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Turkish%20J%20Biochem&title=Artificial%20Intelligence%20Based%20Approaches%20in%20Medical%20Laboratories:%20University%20of%20Health%20Sciences%20Tepecik%20Training%20and%20Research%20Hospital%20Experiences&author=BI%20Basok&volume=43&publication_year=2018&pages=18&doi=10.1515/tjb-2018-0018&)]
* 8.
  Howard J. Artificial Intelligence: Implications for the Future of Work. Am J Ind Med (2019) 62(11):917–26. 10.1002/ajim.23037
   [[DOI](https://doi.org/10.1002/ajim.23037)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31436850/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Ind%20Med&title=Artificial%20Intelligence:%20Implications%20for%20the%20Future%20of%20Work&author=J%20Howard&volume=62&issue=11&publication_year=2019&pages=917-26&pmid=31436850&doi=10.1002/ajim.23037&)]
* 9.
  Davenport T, Kalakota R. The Potential for Artificial Intelligence in Healthcare. Future Healthc J (2019) 6(2):94–8. 10.7861/futurehosp.6-2-94
   [[DOI](https://doi.org/10.7861/futurehosp.6-2-94)] [[PMC free article](/articles/PMC6616181/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31363513/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Future%20Healthc%20J&title=The%20Potential%20for%20Artificial%20Intelligence%20in%20Healthcare&author=T%20Davenport&author=R%20Kalakota&volume=6&issue=2&publication_year=2019&pages=94-8&pmid=31363513&doi=10.7861/futurehosp.6-2-94&)]
* 10.
  Abdulkareem KH, Mohammed MA, Salim A, Arif M, Geman O, Gupta D, et al.
  Realizing an Effective COVID-19 Diagnosis System Based on Machine Learning and Iot in Smart Hospital Environment. Ieee Internet Things J (2021) 8(21):15919–28. 10.1109/JIOT.2021.3050775
   [[DOI](https://doi.org/10.1109/JIOT.2021.3050775)] [[PMC free article](/articles/PMC8769008/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35782183/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ieee%20Internet%20Things%20J&title=Realizing%20an%20Effective%20COVID-19%20Diagnosis%20System%20Based%20on%20Machine%20Learning%20and%20Iot%20in%20Smart%20Hospital%20Environment&author=KH%20Abdulkareem&author=MA%20Mohammed&author=A%20Salim&author=M%20Arif&author=O%20Geman&volume=8&issue=21&publication_year=2021&pages=15919-28&pmid=35782183&doi=10.1109/JIOT.2021.3050775&)]
* 11.
  Abdulkareem M, Petersen SE. The Promise of AI in Detection, Diagnosis, and Epidemiology for Combating COVID-19: Beyond the Hype. Front Artif Intelligence (2021) 4:652669. 10.3389/frai.2021.652669
   [[DOI](https://doi.org/10.3389/frai.2021.652669)] [[PMC free article](/articles/PMC8160471/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34056579/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Artif%20Intelligence&title=The%20Promise%20of%20AI%20in%20Detection,%20Diagnosis,%20and%20Epidemiology%20for%20Combating%20COVID-19:%20Beyond%20the%20Hype&author=M%20Abdulkareem&author=SE%20Petersen&volume=4&publication_year=2021&pages=652669&pmid=34056579&doi=10.3389/frai.2021.652669&)]
* 12.
  Hou H, Zhang R, Li J. Artificial Intelligence in the Clinical Laboratory. Clinica Chim Acta (2024) 559:119724. 10.1016/j.cca.2024.119724
   [[DOI](https://doi.org/10.1016/j.cca.2024.119724)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/38734225/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinica%20Chim%20Acta&title=Artificial%20Intelligence%20in%20the%20Clinical%20Laboratory&author=H%20Hou&author=R%20Zhang&author=J%20Li&volume=559&publication_year=2024&pages=119724&pmid=38734225&doi=10.1016/j.cca.2024.119724&)]
* 13.
  Zhang LM, Huang T, Xu FS, Li SJ, Zheng S, Lyu J, et al.
  Prediction of Prognosis in Elderly Patients with Sepsis Based on Machine Learning (Random Survival Forest). Bmc Emerg Med (2022) 22(1):26. 10.1186/s12873-022-00582-z
   [[DOI](https://doi.org/10.1186/s12873-022-00582-z)] [[PMC free article](/articles/PMC8832779/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35148680/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bmc%20Emerg%20Med&title=Prediction%20of%20Prognosis%20in%20Elderly%20Patients%20with%20Sepsis%20Based%20on%20Machine%20Learning%20(Random%20Survival%20Forest)&author=LM%20Zhang&author=T%20Huang&author=FS%20Xu&author=SJ%20Li&author=S%20Zheng&volume=22&issue=1&publication_year=2022&pages=26&pmid=35148680&doi=10.1186/s12873-022-00582-z&)]
* 14.
  Zhang L, Mao R, Lau CT, Chung WC, Chan JCP, Liang F, et al.
  Identification of Useful Genes from Multiple Microarrays for Ulcerative Colitis Diagnosis Based on Machine Learning Methods. Scientific Rep (2022) 12(1):9962. 10.1038/s41598-022-14048-6
   [[DOI](https://doi.org/10.1038/s41598-022-14048-6)] [[PMC free article](/articles/PMC9200771/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35705632/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Scientific%20Rep&title=Identification%20of%20Useful%20Genes%20from%20Multiple%20Microarrays%20for%20Ulcerative%20Colitis%20Diagnosis%20Based%20on%20Machine%20Learning%20Methods&author=L%20Zhang&author=R%20Mao&author=CT%20Lau&author=WC%20Chung&author=JCP%20Chan&volume=12&issue=1&publication_year=2022&pages=9962&pmid=35705632&doi=10.1038/s41598-022-14048-6&)]
* 15.
  Shelke YP, Badge AK, Bankar NJ. Applications of Artificial Intelligence in Microbial Diagnosis. Cureus (2023) 15(11):e49366. 10.7759/cureus.49366
   [[DOI](https://doi.org/10.7759/cureus.49366)] [[PMC free article](/articles/PMC10749263/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/38146579/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cureus&title=Applications%20of%20Artificial%20Intelligence%20in%20Microbial%20Diagnosis&author=YP%20Shelke&author=AK%20Badge&author=NJ%20Bankar&volume=15&issue=11&publication_year=2023&pages=e49366&pmid=38146579&doi=10.7759/cureus.49366&)]
* 16.
  Ghaderzadeh M, Asadi F, Hosseini A, Bashash D, Abolghasemi H, Roshanpour A. Corrigendum to Deep Learning in the Detection and Diagnosis of COVID-19 Using Radiology Modalities: A Systematic Review. J Healthc Eng (2021) 2021:9868517. 10.1155/2021/9868517
   [[DOI](https://doi.org/10.1155/2021/9868517)] [[PMC free article](/articles/PMC8560226/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34733462/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Healthc%20Eng&title=Corrigendum%20to%20Deep%20Learning%20in%20the%20Detection%20and%20Diagnosis%20of%20COVID-19%20Using%20Radiology%20Modalities:%20A%20Systematic%20Review&author=M%20Ghaderzadeh&author=F%20Asadi&author=A%20Hosseini&author=D%20Bashash&author=H%20Abolghasemi&volume=2021&publication_year=2021&pages=9868517&pmid=34733462&doi=10.1155/2021/9868517&)]
* 17.
  Wilkes EH, Emmett E, Beltran L, Woodward GM, Carling RS. A Machine Learning Approach for the Automated Interpretation of Plasma Amino Acid Profiles. Clin Chem (2020) 66(9):1210–8. 10.1093/clinchem/hvaa134
   [[DOI](https://doi.org/10.1093/clinchem/hvaa134)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32870990/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=A%20Machine%20Learning%20Approach%20for%20the%20Automated%20Interpretation%20of%20Plasma%20Amino%20Acid%20Profiles&author=EH%20Wilkes&author=E%20Emmett&author=L%20Beltran&author=GM%20Woodward&author=RS%20Carling&volume=66&issue=9&publication_year=2020&pages=1210-8&pmid=32870990&doi=10.1093/clinchem/hvaa134&)]
* 18.
  Abdulhussain Y, Ghelani H, Henderson H, Sudhir M, Mascarenhas S, Radhakrishnan R, et al.
  The Use and Effectiveness of high-fidelity Simulation in Health Professions Education: Current Update. Simulation-Transactions Soc Model Simulation Int (2022) 98(12):1085–95. 10.1177/00375497221101066
   [[DOI](https://doi.org/10.1177/00375497221101066)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Simulation-Transactions%20Soc%20Model%20Simulation%20Int&title=The%20Use%20and%20Effectiveness%20of%20high-fidelity%20Simulation%20in%20Health%20Professions%20Education:%20Current%20Update&author=Y%20Abdulhussain&author=H%20Ghelani&author=H%20Henderson&author=M%20Sudhir&author=S%20Mascarenhas&volume=98&issue=12&publication_year=2022&pages=1085-95&doi=10.1177/00375497221101066&)]
* 19.
  Hammer RR, Jones TW, Hussain FTN, Bringe K, Harvey RE, Person-Rennell NH, et al.
  Students as Resurrectionists-A Multimodal Humanities Project in Anatomy Putting Ethics and Professionalism in Historical Context. Anatomical Sci Education (2010) 3(5):244–8. 10.1002/ase.174
   [[DOI](https://doi.org/10.1002/ase.174)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20827724/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anatomical%20Sci%20Education&title=Students%20as%20Resurrectionists-A%20Multimodal%20Humanities%20Project%20in%20Anatomy%20Putting%20Ethics%20and%20Professionalism%20in%20Historical%20Context&author=RR%20Hammer&author=TW%20Jones&author=FTN%20Hussain&author=K%20Bringe&author=RE%20Harvey&volume=3&issue=5&publication_year=2010&pages=244-8&pmid=20827724&doi=10.1002/ase.174&)]
* 20.
  Slimi Z, Carballido BV. Navigating the Ethical Challenges of Artificial Intelligence in Higher Education: An Analysis of Seven Global AI Ethics Policies. TEM J (2023) 12(2):590–602. 10.18421/tem122-02
   [[DOI](https://doi.org/10.18421/tem122-02)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=TEM%20J&title=Navigating%20the%20Ethical%20Challenges%20of%20Artificial%20Intelligence%20in%20Higher%20Education:%20An%20Analysis%20of%20Seven%20Global%20AI%20Ethics%20Policies&author=Z%20Slimi&author=BV%20Carballido&volume=12&issue=2&publication_year=2023&pages=590-602&doi=10.18421/tem122-02&)]
* 21.
  Sam AK, Olbrich P. The Need for AI Ethics in Higher Education. In: AI Ethics in Higher Education: Insights from Africa and Beyond. Cham: Springer International Publishing; (2023). p. 3–10. [[Google Scholar](https://scholar.google.com/scholar_lookup?title=AI%20Ethics%20in%20Higher%20Education:%20Insights%20from%20Africa%20and%20Beyond&author=AK%20Sam&author=P%20Olbrich&publication_year=2023&)]
* 22.
  Masters K. Ethical Use of Artificial Intelligence in Health Professions Education: AMEE Guide No. 158. Med Teach (2023) 45(6):574–84. 10.1080/0142159X.2023.2186203
   [[DOI](https://doi.org/10.1080/0142159X.2023.2186203)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/36912253/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Teach&title=Ethical%20Use%20of%20Artificial%20Intelligence%20in%20Health%20Professions%20Education:%20AMEE%20Guide%20No.%20158&author=K%20Masters&volume=45&issue=6&publication_year=2023&pages=574-84&pmid=36912253&doi=10.1080/0142159X.2023.2186203&)]
* 23.
  Baker RS, Hawn A. Algorithmic Bias in Education. Int J Artif Intelligence Education (2022) 32:1052–92. 10.1007/s40593-021-00285-9
   [[DOI](https://doi.org/10.1007/s40593-021-00285-9)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Artif%20Intelligence%20Education&title=Algorithmic%20Bias%20in%20Education&author=RS%20Baker&author=A%20Hawn&volume=32&publication_year=2022&pages=1052-92&doi=10.1007/s40593-021-00285-9&)]
* 24.
  Fesakis G, Prantsoudi S, editors. Raising Artificial Intelligence Bias Awareness in Secondary Education: The Design of an Educational Intervention. Impact of Artificial Intelligence and Robotics; (2021). [[Google Scholar](https://scholar.google.com/scholar_lookup?title=Raising%20Artificial%20Intelligence%20Bias%20Awareness%20in%20Secondary%20Education:%20The%20Design%20of%20an%20Educational%20Intervention&publication_year=2021&)]
* 25.
  Paranjape K, Schinkel M, Hammer RD, Schouten B, Panday RSN, Elbers PWG, et al.
  The Value of Artificial Intelligence in Laboratory Medicine. Am J Clin Pathol (2021) 155(6):823–31. 10.1093/ajcp/aqaa170
   [[DOI](https://doi.org/10.1093/ajcp/aqaa170)] [[PMC free article](/articles/PMC8130876/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33313667/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Clin%20Pathol&title=The%20Value%20of%20Artificial%20Intelligence%20in%20Laboratory%20Medicine&author=K%20Paranjape&author=M%20Schinkel&author=RD%20Hammer&author=B%20Schouten&author=RSN%20Panday&volume=155&issue=6&publication_year=2021&pages=823-31&pmid=33313667&doi=10.1093/ajcp/aqaa170&)]
* 26.
  Naugler C, Church DL. Automation and Artificial Intelligence in the Clinical Laboratory. Crit Rev Clin Lab Sci (2019) 56(2):98–110. 10.1080/10408363.2018.1561640
   [[DOI](https://doi.org/10.1080/10408363.2018.1561640)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30922144/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit%20Rev%20Clin%20Lab%20Sci&title=Automation%20and%20Artificial%20Intelligence%20in%20the%20Clinical%20Laboratory&author=C%20Naugler&author=DL%20Church&volume=56&issue=2&publication_year=2019&pages=98-110&pmid=30922144&doi=10.1080/10408363.2018.1561640&)]
* 27.
  Busch F, Adams LC, Bressem KK. Biomedical Ethical Aspects Towards the Implementation of Artificial Intelligence in Medical Education. Med Sci Educator (2023) 33(4):1007–12. 10.1007/s40670-023-01815-x
   [[DOI](https://doi.org/10.1007/s40670-023-01815-x)] [[PMC free article](/articles/PMC10403458/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/37546190/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Sci%20Educator&title=Biomedical%20Ethical%20Aspects%20Towards%20the%20Implementation%20of%20Artificial%20Intelligence%20in%20Medical%20Education&author=F%20Busch&author=LC%20Adams&author=KK%20Bressem&volume=33&issue=4&publication_year=2023&pages=1007-12&pmid=37546190&doi=10.1007/s40670-023-01815-x&)]
* 28.
  Narayanan S, Ramakrishnan R, Durairaj E, Das A. Artificial Intelligence Revolutionizing the Field of Medical Education. Cureus (2023) 15(11):e49604. 10.7759/cureus.49604
   [[DOI](https://doi.org/10.7759/cureus.49604)] [[PMC free article](/articles/PMC10755136/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/38161821/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cureus&title=Artificial%20Intelligence%20Revolutionizing%20the%20Field%20of%20Medical%20Education&author=S%20Narayanan&author=R%20Ramakrishnan&author=E%20Durairaj&author=A%20Das&volume=15&issue=11&publication_year=2023&pages=e49604&pmid=38161821&doi=10.7759/cureus.49604&)]
* 29.
  Arksey H, O'Malley L. Scoping Studies: Towards a Methodological Framework. Int J Soc Res Methodol (2005) 8(1):19–32. 10.1080/1364557032000119616
   [[DOI](https://doi.org/10.1080/1364557032000119616)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Soc%20Res%20Methodol&title=Scoping%20Studies:%20Towards%20a%20Methodological%20Framework&author=H%20Arksey&author=L%20O'Malley&volume=8&issue=1&publication_year=2005&pages=19-32&doi=10.1080/1364557032000119616&)]
* 30.
  Levac D, Colquhoun H, O'brien KK. Scoping Studies: Advancing the Methodology. Implementation Sci (2010) 5:1–9. 10.1186/1748-5908-5-69
   [[DOI](https://doi.org/10.1186/1748-5908-5-69)] [[PMC free article](/articles/PMC2954944/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20854677/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Implementation%20Sci&title=Scoping%20Studies:%20Advancing%20the%20Methodology&author=D%20Levac&author=H%20Colquhoun&author=KK%20O'brien&volume=5&publication_year=2010&pages=1-9&pmid=20854677&doi=10.1186/1748-5908-5-69&)]
* 31.
  Ritchie J, Spencer L. Qualitative Data Analysis for Applied Policy Research. In: Analyzing Qualitative Data. Routledge; (2002). p. 173–94. [[Google Scholar](https://scholar.google.com/scholar_lookup?title=Analyzing%20Qualitative%20Data&author=J%20Ritchie&author=L%20Spencer&publication_year=2002&)]
* 32.
  McGowan J, Sampson M, Salzwedel DM, Cogo E, Foerster V, Lefebvre C. PRESS Peer Review of Electronic Search Strategies: 2015 Guideline Statement. J Clin Epidemiol (2016) 75:40–6. 10.1016/j.jclinepi.2016.01.021
   [[DOI](https://doi.org/10.1016/j.jclinepi.2016.01.021)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27005575/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Epidemiol&title=PRESS%20Peer%20Review%20of%20Electronic%20Search%20Strategies:%202015%20Guideline%20Statement&author=J%20McGowan&author=M%20Sampson&author=DM%20Salzwedel&author=E%20Cogo&author=V%20Foerster&volume=75&publication_year=2016&pages=40-6&pmid=27005575&doi=10.1016/j.jclinepi.2016.01.021&)]
* 33.
  Hope Kearns E. A Model for International clinical/biomedical Science Programs. University of Central Lancashire; (2004). [[Google Scholar](https://scholar.google.com/scholar_lookup?title=A%20Model%20for%20International%20clinical/biomedical%20Science%20Programs&author=E%20Hope%20Kearns&publication_year=2004&)]
* 34.
  Sharma A. Evaluation of Laboratory Clinical Training Internship of a NAACLS Accredited Program in Qatar: Graduates Perspective (2021). [[Google Scholar](https://scholar.google.com/scholar_lookup?Sharma%20A.%20Evaluation%20of%20Laboratory%20Clinical%20Training%20Internship%20of%20a%20NAACLS%20Accredited%20Program%20in%20Qatar:%20Graduates%20Perspective%20(2021).)]
* 35.
  (NAACLS) NAAfCLS. NAACLS Standards for Accredited Programs. (2024). [[Google Scholar](https://scholar.google.com/scholar_lookup?(NAACLS)%20NAAfCLS.%20NAACLS%20Standards%20for%20Accredited%20Programs.%20(2024).)]
* 36.
  Emancipator K. Critical Values: ASCP Practice Parameter. Am J Clin Pathol (1997) 108(3):247–53. 10.1093/ajcp/108.3.247
   [[DOI](https://doi.org/10.1093/ajcp/108.3.247)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/9291451/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Clin%20Pathol&title=Critical%20Values:%20ASCP%20Practice%20Parameter&author=K%20Emancipator&volume=108&issue=3&publication_year=1997&pages=247-53&pmid=9291451&doi=10.1093/ajcp/108.3.247&)]
* 37.
  Al Sudairi MAT, Vasista TG. Achieving Process Standardization in Digital Society With'Ascp Model. J Supply Chain Customer Relationship Management (2013) 2013:1–12. 10.5171/2013.759359
   [[DOI](https://doi.org/10.5171/2013.759359)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Supply%20Chain%20Customer%20Relationship%20Management&title=Achieving%20Process%20Standardization%20in%20Digital%20Society%20With'Ascp%20Model&author=MAT%20Al%20Sudairi&author=TG%20Vasista&volume=2013&publication_year=2013&pages=1-12&doi=10.5171/2013.759359&)]
* 38.
  Waibel E, Garcia E, Kelly M, Soles R, Hilborne L. Systematic Review of non-ASCP Choosing Wisely Recommendations Relevant to Pathology and Laboratory Medicine. Am J Clin Pathol (2018) 149(3):267–74. 10.1093/ajcp/aqx159
   [[DOI](https://doi.org/10.1093/ajcp/aqx159)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29425260/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Clin%20Pathol&title=Systematic%20Review%20of%20non-ASCP%20Choosing%20Wisely%20Recommendations%20Relevant%20to%20Pathology%20and%20Laboratory%20Medicine&author=E%20Waibel&author=E%20Garcia&author=M%20Kelly&author=R%20Soles&author=L%20Hilborne&volume=149&issue=3&publication_year=2018&pages=267-74&pmid=29425260&doi=10.1093/ajcp/aqx159&)]
* 39.
  Beck L. ASCP Bylaws 2020- American Society for Clinical Pathology. (2019). [[Google Scholar](https://scholar.google.com/scholar_lookup?Beck%20L.%20ASCP%20Bylaws%202020-%20American%20Society%20for%20Clinical%20Pathology.%20(2019).)]
* 40.
  Smolak A, Chemaitelly H, Hermez JG, Low N, Abu-Raddad LJ. Epidemiology of *Chlamydia trachomatis* in the Middle East and North Africa: A Systematic Review, meta-analysis, and meta-regression. Lancet Glob Health (2019) 7(9):e1197–e225. 10.1016/S2214-109X(19)30279-7
   [[DOI](https://doi.org/10.1016/S2214-109X(19)30279-7)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31402004/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Glob%20Health&title=Epidemiology%20of%20Chlamydia%20trachomatis%20in%20the%20Middle%20East%20and%20North%20Africa:%20A%20Systematic%20Review,%20meta-analysis,%20and%20meta-regression&author=A%20Smolak&author=H%20Chemaitelly&author=JG%20Hermez&author=N%20Low&author=LJ%20Abu-Raddad&volume=7&issue=9&publication_year=2019&pages=e1197-e225&pmid=31402004&doi=10.1016/S2214-109X(19)30279-7&)]
* 41.
  Tricco AC, Lillie E, Zarin W, O'Brien KK, Colquhoun H, Levac D, et al.
  PRISMA Extension for Scoping Reviews (PRISMA-ScR): Checklist and Explanation. Ann Intern Med (2018) 169(7):467–73. 10.7326/M18-0850
   [[DOI](https://doi.org/10.7326/M18-0850)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30178033/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=PRISMA%20Extension%20for%20Scoping%20Reviews%20(PRISMA-ScR):%20Checklist%20and%20Explanation&author=AC%20Tricco&author=E%20Lillie&author=W%20Zarin&author=KK%20O'Brien&author=H%20Colquhoun&volume=169&issue=7&publication_year=2018&pages=467-73&pmid=30178033&doi=10.7326/M18-0850&)]
* 42.
  WHO. Recommendation of the Council on Artificial Intelligence (OECD Legal Instruments. OECD/LEGAL/O449). (2023). [[Google Scholar](https://scholar.google.com/scholar_lookup?WHO.%20Recommendation%20of%20the%20Council%20on%20Artificial%20Intelligence%20(OECD%20Legal%20Instruments.%20OECD/LEGAL/O449).%20(2023).)]
* 43.
  Moher D, Shamseer L, Clarke M, Ghersi D, Liberati A, Petticrew M, et al.
  Preferred Reporting Items for Systematic Review and meta-analysis Protocols (PRISMA-P) 2015 Statement. Syst Rev (2015) 4(1):1. 10.1186/2046-4053-4-1
   [[DOI](https://doi.org/10.1186/2046-4053-4-1)] [[PMC free article](/articles/PMC4320440/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25554246/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Syst%20Rev&title=Preferred%20Reporting%20Items%20for%20Systematic%20Review%20and%20meta-analysis%20Protocols%20(PRISMA-P)%202015%20Statement&author=D%20Moher&author=L%20Shamseer&author=M%20Clarke&author=D%20Ghersi&author=A%20Liberati&volume=4&issue=1&publication_year=2015&pages=1&pmid=25554246&doi=10.1186/2046-4053-4-1&)]
* 44.
  Scardoni A, Balzarini F, Signorelli C, Cabitza F, Odone A. Artificial Intelligence-based Tools to Control Healthcare Associated Infections: A Systematic Review of the Literature. J Infect Public Health (2020) 13(8):1061–77. 10.1016/j.jiph.2020.06.006
   [[DOI](https://doi.org/10.1016/j.jiph.2020.06.006)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32561275/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Public%20Health&title=Artificial%20Intelligence-based%20Tools%20to%20Control%20Healthcare%20Associated%20Infections:%20A%20Systematic%20Review%20of%20the%20Literature&author=A%20Scardoni&author=F%20Balzarini&author=C%20Signorelli&author=F%20Cabitza&author=A%20Odone&volume=13&issue=8&publication_year=2020&pages=1061-77&pmid=32561275&doi=10.1016/j.jiph.2020.06.006&)]
* 45.
  Giacomini M, Ruggiero C, Maillard M, Lillo FB, Varnier OE. Objective Evaluation of Two Markers of HIV-1 Infection (P24 Antigen Concentration and CD4+ Cell Counts) by a Self Organizing Neural Network. Med Inform (1996) 21(3):215–28. 10.3109/14639239609025359
   [[DOI](https://doi.org/10.3109/14639239609025359)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/9062884/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Inform&title=Objective%20Evaluation%20of%20Two%20Markers%20of%20HIV-1%20Infection%20(P24%20Antigen%20Concentration%20and%20CD4+%20Cell%20Counts)%20by%20a%20Self%20Organizing%20Neural%20Network&author=M%20Giacomini&author=C%20Ruggiero&author=M%20Maillard&author=FB%20Lillo&author=OE%20Varnier&volume=21&issue=3&publication_year=1996&pages=215-28&pmid=9062884&doi=10.3109/14639239609025359&)]
* 46.
  Liu HF, Lussier YA, Friedman C. Disambiguating Ambiguous Biomedical Terms in Biomedical Narrative Text: An Unsupervised Method. J Biomed Inform (2001) 34(4):249–61. 10.1006/jbin.2001.1023
   [[DOI](https://doi.org/10.1006/jbin.2001.1023)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11977807/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biomed%20Inform&title=Disambiguating%20Ambiguous%20Biomedical%20Terms%20in%20Biomedical%20Narrative%20Text:%20An%20Unsupervised%20Method&author=HF%20Liu&author=YA%20Lussier&author=C%20Friedman&volume=34&issue=4&publication_year=2001&pages=249-61&pmid=11977807&doi=10.1006/jbin.2001.1023&)]
* 47.
  Prank K, Schulze E, Eckert O, Nattkemper TW, Bettendorf M, Maser-Gluth C, et al.
  Machine Learning Approaches for phenotype-genotype Mapping: Predicting Heterozygous Mutations in the CYP21B Gene from Steroid Profiles. Eur J Endocrinol (2005) 153(2):301–5. 10.1530/eje.1.01957
   [[DOI](https://doi.org/10.1530/eje.1.01957)] [[PMC free article](/articles/PMC2909743/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16061837/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Endocrinol&title=Machine%20Learning%20Approaches%20for%20phenotype-genotype%20Mapping:%20Predicting%20Heterozygous%20Mutations%20in%20the%20CYP21B%20Gene%20from%20Steroid%20Profiles&author=K%20Prank&author=E%20Schulze&author=O%20Eckert&author=TW%20Nattkemper&author=M%20Bettendorf&volume=153&issue=2&publication_year=2005&pages=301-5&pmid=16061837&doi=10.1530/eje.1.01957&)]
* 48.
  Udelhoven T, Naumann D, Schmitt J. Development of a Hierarchical Classification System with Artificial Neural Networks and FT-IR Spectra for the Identification of Bacteria. Appl Spectrosc (2000) 54(10):1471–9. 10.1366/0003702001948619
   [[DOI](https://doi.org/10.1366/0003702001948619)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Spectrosc&title=Development%20of%20a%20Hierarchical%20Classification%20System%20with%20Artificial%20Neural%20Networks%20and%20FT-IR%20Spectra%20for%20the%20Identification%20of%20Bacteria&author=T%20Udelhoven&author=D%20Naumann&author=J%20Schmitt&volume=54&issue=10&publication_year=2000&pages=1471-9&doi=10.1366/0003702001948619&)]
* 49.
  Munshi R, Coalson RD, Ermentrout GB, Madura JD, Meirovitch H, Stiles JR, et al.
  An Introduction to Simulation and Visualization of Biological Systems at Multiple Scales: A Summer Training Program for Interdisciplinary Research. Biotechnol Prog (2006) 22(1):179–85. 10.1021/bp0501773
   [[DOI](https://doi.org/10.1021/bp0501773)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16454509/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biotechnol%20Prog&title=An%20Introduction%20to%20Simulation%20and%20Visualization%20of%20Biological%20Systems%20at%20Multiple%20Scales:%20A%20Summer%20Training%20Program%20for%20Interdisciplinary%20Research&author=R%20Munshi&author=RD%20Coalson&author=GB%20Ermentrout&author=JD%20Madura&author=H%20Meirovitch&volume=22&issue=1&publication_year=2006&pages=179-85&pmid=16454509&doi=10.1021/bp0501773&)]
* 50.
  Zheng JG, Howsmon D, Zhang BL, Hahn J, McGuinness D, Hendler J, et al.
  Entity Linking for Biomedical Literature. Bmc Med Inform Decis Making (2015) 15:S4. 10.1186/1472-6947-15-S1-S4
   [[DOI](https://doi.org/10.1186/1472-6947-15-S1-S4)] [[PMC free article](/articles/PMC4460707/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26045232/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bmc%20Med%20Inform%20Decis%20Making&title=Entity%20Linking%20for%20Biomedical%20Literature&author=JG%20Zheng&author=D%20Howsmon&author=BL%20Zhang&author=J%20Hahn&author=D%20McGuinness&volume=15&publication_year=2015&pages=S4&pmid=26045232&doi=10.1186/1472-6947-15-S1-S4&)]
* 51.
  Ambite JL, Gordon J, Fierro L, Burns G, Mathew J. Linking Educational Resources on Data Science. Honolulu: Educational Advances in Artificial Intelligence; (2019). [[Google Scholar](https://scholar.google.com/scholar_lookup?title=Linking%20Educational%20Resources%20on%20Data%20Science&author=JL%20Ambite&author=J%20Gordon&author=L%20Fierro&author=G%20Burns&author=J%20Mathew&publication_year=2019&)]
* 52.
  Aparicio F, Morales-Botello ML, Rubio M, Hernando A, Muñoz R, López-Fernández H, et al.
  Perceptions of the Use of Intelligent Information Access Systems in University Level Active Learning Activities Among Teachers of Biomedical Subjects. Int J Med Inform (2018) 112:21–33. 10.1016/j.ijmedinf.2017.12.016
   [[DOI](https://doi.org/10.1016/j.ijmedinf.2017.12.016)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29500018/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Med%20Inform&title=Perceptions%20of%20the%20Use%20of%20Intelligent%20Information%20Access%20Systems%20in%20University%20Level%20Active%20Learning%20Activities%20Among%20Teachers%20of%20Biomedical%20Subjects&author=F%20Aparicio&author=ML%20Morales-Botello&author=M%20Rubio&author=A%20Hernando&author=R%20Mu%C3%B1oz&volume=112&publication_year=2018&pages=21-33&pmid=29500018&doi=10.1016/j.ijmedinf.2017.12.016&)]
* 53.
  Henderson N, Rowe J, Paquette L, Baker RS, Lester J. editors. Improving Affect Detection in Game-based Learning with Multimodal Data Fusion. Artificial Intelligence in Education AIED; (2020). [[Google Scholar](https://scholar.google.com/scholar_lookup?title=Improving%20Affect%20Detection%20in%20Game-based%20Learning%20with%20Multimodal%20Data%20Fusion&publication_year=2020&)]
* 54.
  Tota P, Vaida MF. Modular Telepresence Robot for Distance Medical Education. Health and Bioengineering (EHB). Grigore T Popa Univ Med and Pharmacy; (2021). [[Google Scholar](https://scholar.google.com/scholar_lookup?title=Modular%20Telepresence%20Robot%20for%20Distance%20Medical%20Education.%20Health%20and%20Bioengineering%20(EHB)&author=P%20Tota&author=MF%20Vaida&publication_year=2021&)]
* 55.
  Manaka T, van Zyl T, Kar D. editors. Improving cause-of-death Classification from Verbal Autopsy Reports. South Africa SACAIR; (2022). [[Google Scholar](https://scholar.google.com/scholar_lookup?title=Improving%20cause-of-death%20Classification%20from%20Verbal%20Autopsy%20Reports&publication_year=2022&)]
* 56.
  Alam MM, Islam MT. Machine Learning Approach of Automatic Identification and Counting of Blood Cells. Healthc Technology Lett (2019) 6(4):103–8. 10.1049/htl.2018.5098
   [[DOI](https://doi.org/10.1049/htl.2018.5098)] [[PMC free article](/articles/PMC6718065/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31531224/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Healthc%20Technology%20Lett&title=Machine%20Learning%20Approach%20of%20Automatic%20Identification%20and%20Counting%20of%20Blood%20Cells&author=MM%20Alam&author=MT%20Islam&volume=6&issue=4&publication_year=2019&pages=103-8&pmid=31531224&doi=10.1049/htl.2018.5098&)]
* 57.
  Shewajo FA, Fante KA. Tile-Based Microscopic Image Processing for Malaria Screening Using a Deep Learning Approach. Bmc Med Imaging (2023) 23(1):39. 10.1186/s12880-023-00993-9
   [[DOI](https://doi.org/10.1186/s12880-023-00993-9)] [[PMC free article](/articles/PMC10035268/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/36949382/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bmc%20Med%20Imaging&title=Tile-Based%20Microscopic%20Image%20Processing%20for%20Malaria%20Screening%20Using%20a%20Deep%20Learning%20Approach&author=FA%20Shewajo&author=KA%20Fante&volume=23&issue=1&publication_year=2023&pages=39&pmid=36949382&doi=10.1186/s12880-023-00993-9&)]
* 58.
  Umer M, Sadiq S, Ahmad M, Ullah S, Choi GS, Mehmood A. A Novel Stacked CNN for Malarial Parasite Detection in Thin Blood Smear Images. Ieee Access (2020) 8:93782–92. 10.1109/access.2020.2994810
   [[DOI](https://doi.org/10.1109/access.2020.2994810)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ieee%20Access&title=A%20Novel%20Stacked%20CNN%20for%20Malarial%20Parasite%20Detection%20in%20Thin%20Blood%20Smear%20Images&author=M%20Umer&author=S%20Sadiq&author=M%20Ahmad&author=S%20Ullah&author=GS%20Choi&volume=8&publication_year=2020&pages=93782-92&doi=10.1109/access.2020.2994810&)]
* 59.
  Bohr A, Memarzadeh K. The Rise of Artificial Intelligence in Healthcare Applications. In: Bohr A, Memarzadeh K, editors. Artificial Intelligence in Healthcare (2020). [[Google Scholar](https://scholar.google.com/scholar_lookup?title=Artificial%20Intelligence%20in%20Healthcare&author=A%20Bohr&author=K%20Memarzadeh&publication_year=2020&)]
* 60.
  Yang J, Dung NT, Thach PN, Phong NT, Phu VD, Phu KD, et al.
  Generalizability Assessment of AI Models Across Hospitals in a low-middle and High Income Country. Nat Commun (2024) 15(1):8270. 10.1038/s41467-024-52618-6
   [[DOI](https://doi.org/10.1038/s41467-024-52618-6)] [[PMC free article](/articles/PMC11436917/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/39333515/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Generalizability%20Assessment%20of%20AI%20Models%20Across%20Hospitals%20in%20a%20low-middle%20and%20High%20Income%20Country&author=J%20Yang&author=NT%20Dung&author=PN%20Thach&author=NT%20Phong&author=VD%20Phu&volume=15&issue=1&publication_year=2024&pages=8270&pmid=39333515&doi=10.1038/s41467-024-52618-6&)]
* 61.
  Egli A, Schrenzel J, Greub G. Digital Microbiology. Clin Microbiol Infect (2020) 26(10):1324–31. 10.1016/j.cmi.2020.06.023
   [[DOI](https://doi.org/10.1016/j.cmi.2020.06.023)] [[PMC free article](/articles/PMC7320868/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32603804/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Microbiol%20Infect&title=Digital%20Microbiology&author=A%20Egli&author=J%20Schrenzel&author=G%20Greub&volume=26&issue=10&publication_year=2020&pages=1324-31&pmid=32603804&doi=10.1016/j.cmi.2020.06.023&)]
* 62.
  Pedro F, Subosa M, Rivas A, Valverde P. Artificial Intelligence in Education: Challenges and Opportunities for Sustainable Development. (2019). [[Google Scholar](https://scholar.google.com/scholar_lookup?Pedro%20F,%20Subosa%20M,%20Rivas%20A,%20Valverde%20P.%20Artificial%20Intelligence%20in%20Education:%20Challenges%20and%20Opportunities%20for%20Sustainable%20Development.%20(2019).)]
* 63.
  Kuleto V, Ilić M, Dumangiu M, Ranković M, Martins OMD, Păun D, et al.
  Exploring Opportunities and Challenges of Artificial Intelligence and Machine Learning in Higher Education Institutions. Sustainability (2021) 13(18):10424. 10.3390/su131810424
   [[DOI](https://doi.org/10.3390/su131810424)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sustainability&title=Exploring%20Opportunities%20and%20Challenges%20of%20Artificial%20Intelligence%20and%20Machine%20Learning%20in%20Higher%20Education%20Institutions&author=V%20Kuleto&author=M%20Ili%C4%87&author=M%20Dumangiu&author=M%20Rankovi%C4%87&author=OMD%20Martins&volume=13&issue=18&publication_year=2021&pages=10424&doi=10.3390/su131810424&)]
* 64.
  da Silva RGL. The Advancement of Artificial Intelligence in Biomedical Research and Health Innovation: Challenges and Opportunities in Emerging Economies. Globalization and Health (2024) 20(1):44. 10.1186/s12992-024-01049-5
   [[DOI](https://doi.org/10.1186/s12992-024-01049-5)] [[PMC free article](/articles/PMC11107016/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/38773458/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Globalization%20and%20Health&title=The%20Advancement%20of%20Artificial%20Intelligence%20in%20Biomedical%20Research%20and%20Health%20Innovation:%20Challenges%20and%20Opportunities%20in%20Emerging%20Economies&author=RGL%20da%20Silva&volume=20&issue=1&publication_year=2024&pages=44&pmid=38773458&doi=10.1186/s12992-024-01049-5&)]
* 65.
  Murdoch B. Privacy and Artificial Intelligence: Challenges for Protecting Health Information in a New Era. BMC Med Ethics (2021) 22(1):122. 10.1186/s12910-021-00687-3
   [[DOI](https://doi.org/10.1186/s12910-021-00687-3)] [[PMC free article](/articles/PMC8442400/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34525993/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med%20Ethics&title=Privacy%20and%20Artificial%20Intelligence:%20Challenges%20for%20Protecting%20Health%20Information%20in%20a%20New%20Era&author=B%20Murdoch&volume=22&issue=1&publication_year=2021&pages=122&pmid=34525993&doi=10.1186/s12910-021-00687-3&)]
* 66.
  Brown S, Sprague C. Health Care Providers' Perceptions of Barriers to Perinatal Mental Healthcare in South Africa. BMC Public Health (2021) 21:1905. 10.1186/s12889-021-11954-8
   [[DOI](https://doi.org/10.1186/s12889-021-11954-8)] [[PMC free article](/articles/PMC8528557/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34670531/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Public%20Health&title=Health%20Care%20Providers'%20Perceptions%20of%20Barriers%20to%20Perinatal%20Mental%20Healthcare%20in%20South%20Africa&author=S%20Brown&author=C%20Sprague&volume=21&publication_year=2021&pages=1905&pmid=34670531&doi=10.1186/s12889-021-11954-8&)]
* 67.
  Valko N, Osadchyi V. Education Individualization by Means of Artificial Neural Networks. E3s Web Conf (2020) 166:10021. 10.1051/e3sconf/202016610021
   [[DOI](https://doi.org/10.1051/e3sconf/202016610021)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=E3s%20Web%20Conf&title=Education%20Individualization%20by%20Means%20of%20Artificial%20Neural%20Networks&author=N%20Valko&author=V%20Osadchyi&volume=166&publication_year=2020&pages=10021&doi=10.1051/e3sconf/202016610021&)]
* 68.
  Bhardwaj P, Gupta PK, Panwar H, Siddiqui MK, Morales-Menendez R, Bhaik A. Application of Deep Learning on Student Engagement in e-learning Environments. Comput and Electr Eng (2021) 93:107277. 10.1016/j.compeleceng.2021.107277
   [[DOI](https://doi.org/10.1016/j.compeleceng.2021.107277)] [[PMC free article](/articles/PMC9757737/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/36567679/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Comput%20and%20Electr%20Eng&title=Application%20of%20Deep%20Learning%20on%20Student%20Engagement%20in%20e-learning%20Environments&author=P%20Bhardwaj&author=PK%20Gupta&author=H%20Panwar&author=MK%20Siddiqui&author=R%20Morales-Menendez&volume=93&publication_year=2021&pages=107277&pmid=36567679&doi=10.1016/j.compeleceng.2021.107277&)]
* 69.
  Pesheva E. A New Artificial Intelligence Tool for Cancer (2024). Available online at: <https://hms.harvard.edu/news/new-artificial-intelligence-tool-cancer>. (Accessed: September 4, 2024).
* 70.
  Lindahl M. Voicepilot Framework Enhances Communication Between Humans and Physically Assistive Robots (2024). Available online at: <https://www.cmu.edu/news/stories/archives/2024/august/voicepilot-framework-enhances-communication-between-humans-and-physically-assistive-robots>. (Accessed: August 27, 2024).
* 71.
  Services UIT. Artificial Intelligence (AI) (2024). Available online at: <https://itservices.usc.edu/ai/>
  (Accessed: October 5, 2024).
* 72.
  Press OU. AI in Education: Where We Are and What Happens Next (2024). Available online at: <https://corp.oup.com/feature/ai-in-education-where-we-are-and-what-happens-next/>. (Accessed: October 18, 2023).
* 73.
  INDIAai. Andhra Pradesh Calls for AI Adoption in Curriculum (2024). Available online at: <https://indiaai.gov.in/news/andhra-pradesh-calls-for-ai-adoption-in-curriculum>. (Accessed: July 17, 2023).
* 74.
  GOV.UK. Policy Paper Generative Artificial Intelligence (AI) in Education (2024). Available online at: <https://www.gov.uk/government/publications/generative-artificial-intelligence-in-education/generative-artificial-intelligence-ai-in-education>. (Accessed: October 26, 2023).
* 75.
  Alachram H, Chereda H, Beissbarth T, Wingender E, Stegmaier P. Text Mining-based Word Representations for Biomedical Data Analysis and protein-protein Interaction Networks in Machine Learning Tasks. Plos One. (2021) 16(10):e0258623. 10.1371/journal.pone.0258623
   [[DOI](https://doi.org/10.1371/journal.pone.0258623)] [[PMC free article](/articles/PMC8519453/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34653224/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plos%20One.&title=Text%20Mining-based%20Word%20Representations%20for%20Biomedical%20Data%20Analysis%20and%20protein-protein%20Interaction%20Networks%20in%20Machine%20Learning%20Tasks&author=H%20Alachram&author=H%20Chereda&author=T%20Beissbarth&author=E%20Wingender&author=P%20Stegmaier&volume=16&issue=10&publication_year=2021&pages=e0258623&pmid=34653224&doi=10.1371/journal.pone.0258623&)]
* 76.
  Anthony Q, Xu L, Subramoni H, Panda DKD. Scaling Single-Image Super-resolution Training on Modern HPC Clusters: Early Experiences. Parallel Distributed Process (2021) 21. [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Parallel%20Distributed%20Process&title=Scaling%20Single-Image%20Super-resolution%20Training%20on%20Modern%20HPC%20Clusters:%20Early%20Experiences&author=Q%20Anthony&author=L%20Xu&author=H%20Subramoni&author=DKD%20Panda&publication_year=2021&)]
* 77.
  Bai BJ, Yang XL, Li YZ, Zhang YJ, Pillar N, Ozcan A. Deep learning-enabled Virtual Histological Staining of Biological Samples. Light-Science and Appl (2023) 12(1):57. 10.1038/s41377-023-01104-7
   [[DOI](https://doi.org/10.1038/s41377-023-01104-7)] [[PMC free article](/articles/PMC9981740/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/36864032/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Light-Science%20and%20Appl&title=Deep%20learning-enabled%20Virtual%20Histological%20Staining%20of%20Biological%20Samples&author=BJ%20Bai&author=XL%20Yang&author=YZ%20Li&author=YJ%20Zhang&author=N%20Pillar&volume=12&issue=1&publication_year=2023&pages=57&pmid=36864032&doi=10.1038/s41377-023-01104-7&)]
* 78.
  Bonatti AF, Vozzi G, Chua CK, De Maria C. A Deep Learning Quality Control Loop of the Extrusion-based Bioprinting Process. Int J Bioprinting (2022) 8(4):620–0. 10.18063/ijb.v8i4.620
   [[DOI](https://doi.org/10.18063/ijb.v8i4.620)] [[PMC free article](/articles/PMC9668573/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/36404777/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Bioprinting&title=A%20Deep%20Learning%20Quality%20Control%20Loop%20of%20the%20Extrusion-based%20Bioprinting%20Process&author=AF%20Bonatti&author=G%20Vozzi&author=CK%20Chua&author=C%20De%20Maria&volume=8&issue=4&publication_year=2022&pages=620-0&pmid=36404777&doi=10.18063/ijb.v8i4.620&)]
* 79.
  Gao Z, Fu G, Ouyang CP, Tsutsui S, Liu XZ, Yang J, et al.
  edge2vec: Representation Learning Using Edge Semantics for Biomedical Knowledge Discovery. Bmc Bioinformatics (2019) 20:306. 10.1186/s12859-019-2914-2
   [[DOI](https://doi.org/10.1186/s12859-019-2914-2)] [[PMC free article](/articles/PMC6593489/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31238875/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bmc%20Bioinformatics&title=edge2vec:%20Representation%20Learning%20Using%20Edge%20Semantics%20for%20Biomedical%20Knowledge%20Discovery&author=Z%20Gao&author=G%20Fu&author=CP%20Ouyang&author=S%20Tsutsui&author=XZ%20Liu&volume=20&publication_year=2019&pages=306&pmid=31238875&doi=10.1186/s12859-019-2914-2&)]
* 80.
  Goncalves S, Cortez P, Moro S. A Deep Learning Classifier for Sentence Classification in Biomedical and Computer Science Abstracts. Neural Comput and Appl (2020) 32(11):6793–807. 10.1007/s00521-019-04334-2
   [[DOI](https://doi.org/10.1007/s00521-019-04334-2)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neural%20Comput%20and%20Appl&title=A%20Deep%20Learning%20Classifier%20for%20Sentence%20Classification%20in%20Biomedical%20and%20Computer%20Science%20Abstracts&author=S%20Goncalves&author=P%20Cortez&author=S%20Moro&volume=32&issue=11&publication_year=2020&pages=6793-807&doi=10.1007/s00521-019-04334-2&)]
* 81.
  Hill M, Hoena B, Kilian W, Odenwald S. editors. Wearable, Modular and Intelligent Sensor Laboratory. International Sports Engineering Association ISEA; (2016). [[Google Scholar](https://scholar.google.com/scholar_lookup?title=Wearable,%20Modular%20and%20Intelligent%20Sensor%20Laboratory&publication_year=2016&)]
* 82.
  Liu ZT, Wang LL, Meng Y, He TT, He SF, Yang YS, et al.
  All-Fiber high-speed Image Detection Enabled by Deep Learning. Nat Commun (2022) 13(1):1433. 10.1038/s41467-022-29178-8
   [[DOI](https://doi.org/10.1038/s41467-022-29178-8)] [[PMC free article](/articles/PMC8930987/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35301332/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=All-Fiber%20high-speed%20Image%20Detection%20Enabled%20by%20Deep%20Learning&author=ZT%20Liu&author=LL%20Wang&author=Y%20Meng&author=TT%20He&author=SF%20He&volume=13&issue=1&publication_year=2022&pages=1433&pmid=35301332&doi=10.1038/s41467-022-29178-8&)]
* 83.
  Hill M, Hoena B, Kilian W, Odenwald S. Wearable, Modular and Intelligent Sensor Laboratory. Proced Eng (2016) 147:671–6. 10.1016/j.proeng.2016.06.270
   [[DOI](https://doi.org/10.1016/j.proeng.2016.06.270)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proced%20Eng&title=Wearable,%20Modular%20and%20Intelligent%20Sensor%20Laboratory&author=M%20Hill&author=B%20Hoena&author=W%20Kilian&author=S%20Odenwald&volume=147&publication_year=2016&pages=671-6&doi=10.1016/j.proeng.2016.06.270&)]
* 84.
  Meade AD, Clarke C, Bonnier F, Poon K, Garcia A, Knief P. editors. Functional and Pathological Analysis of Biological Systems Using Vibrational Spectroscopy with Chemometric and Heuristic Approaches. Hyperspectral Image and Signal Processing - Evolution in Remote Sensing (2009). [[Google Scholar](https://scholar.google.com/scholar_lookup?title=Hyperspectral%20Image%20and%20Signal%20Processing%20-%20Evolution%20in%20Remote%20Sensing&publication_year=2009&)]
* 85.
  Nguyen D, Tao L, Li Y. Integration of Machine Learning and Coarse-Grained Molecular Simulations for Polymer Materials: Physical Understandings and Molecular Design. Front Chem (2022) 9:820417. 10.3389/fchem.2021.820417
   [[DOI](https://doi.org/10.3389/fchem.2021.820417)] [[PMC free article](/articles/PMC8819075/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35141207/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Chem&title=Integration%20of%20Machine%20Learning%20and%20Coarse-Grained%20Molecular%20Simulations%20for%20Polymer%20Materials:%20Physical%20Understandings%20and%20Molecular%20Design&author=D%20Nguyen&author=L%20Tao&author=Y%20Li&volume=9&publication_year=2022&pages=820417&pmid=35141207&doi=10.3389/fchem.2021.820417&)]
* 86.
  Rivenson Y, Koydemir HC, Wang HD, Wei ZS, Ren ZS, Günaydin H, et al.
  Deep Learning Enhanced Mobile-Phone Microscopy. Acs Photon (2018) 5(6):2354–64. 10.3389/fchem.2021.820417
   [[DOI](https://doi.org/10.3389/fchem.2021.820417)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acs%20Photon&title=Deep%20Learning%20Enhanced%20Mobile-Phone%20Microscopy&author=Y%20Rivenson&author=HC%20Koydemir&author=HD%20Wang&author=ZS%20Wei&author=ZS%20Ren&volume=5&issue=6&publication_year=2018&pages=2354-64&doi=10.3389/fchem.2021.820417&)]
* 87.
  Chiu PH, Hripcsak G. EHR-based Phenotyping: Bulk Learning and Evaluation. J Biomed Inform (2017) 70:35–51. 10.1016/j.jbi.2017.04.009
   [[DOI](https://doi.org/10.1016/j.jbi.2017.04.009)] [[PMC free article](/articles/PMC5934756/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/28410982/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biomed%20Inform&title=EHR-based%20Phenotyping:%20Bulk%20Learning%20and%20Evaluation&author=PH%20Chiu&author=G%20Hripcsak&volume=70&publication_year=2017&pages=35-51&pmid=28410982&doi=10.1016/j.jbi.2017.04.009&)]
* 88.
  Kumari A, Kumari A, Singh A, Singh SK, Juhi A, Dhanvijay AKD, et al.
  Large Language Models in Hematology Case Solving: A Comparative Study of ChatGPT-3.5, Google Bard, and Microsoft Bing. Cureus J Med Sci (2023) 15(8). 10.1016/j.jbi.2017.04.008
   [[DOI](https://doi.org/10.1016/j.jbi.2017.04.008)] [[PMC free article](/articles/PMC10511207/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/37736448/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cureus%20J%20Med%20Sci&title=Large%20Language%20Models%20in%20Hematology%20Case%20Solving:%20A%20Comparative%20Study%20of%20ChatGPT-3.5,%20Google%20Bard,%20and%20Microsoft%20Bing&author=A%20Kumari&author=A%20Kumari&author=A%20Singh&author=SK%20Singh&author=A%20Juhi&volume=15&issue=8&publication_year=2023&pmid=37736448&doi=10.1016/j.jbi.2017.04.008&)]

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

[DataSheet2.docx](/articles/instance/12360964/bin/DataSheet2.docx) (243.8KB, docx)

[DataSheet1.docx](/articles/instance/12360964/bin/DataSheet1.docx) (98.9KB, docx)

---

*爬取时间: 2025-11-28 21:53:07*
